<html>
<head><meta http-equiv="content-type" content="text/html; charset=UTF-8">
<title>C-I-TASSER CIT260</title></head>

<script type="text/javascript" src="/jmol/JSmol.min.js"></script>
<script type="text/javascript" src="/jmol/Jmol2.js"></script>

<body>
<font face="Times New Roman">
[<a href="/C-I-TASSER">Back to server</a>]
<br>
<h1 align="center">C-I-TASSER results for CIT260</h1>
<h4 align="center">[Click <a href="result.zip" download>result.zip</a>
    to download all results on this page]</h4>
<font size="4" face="Arial">

<div style="background:#6599FF;width:500px;"><b><font face="Arial" 
color="#FFFFFF" size=4>&nbsp;Input Sequence in FASTA format</font></b></div>
<ul>
    <table style="font-family:Monospace;font-size:14px;background:#F2F2F2;">
    <tr><td colspan="2">&gt;NX_B3GLJ2 (98 residues)<br>
    MNKHFLFLFLLYCLIVAVTSLQCITCHLRTRTDRCRRGFGVCTAQKGEACMLLRIYQRNT<br>
LQISYMVCQKFCRDMTFDLRNRTYVHTCCNYNYCNFKL<br>
    </td></tr>
    </table><br>
</ul>

<div style="background:#6599FF;width:500px;"><b><font face="Arial" 
color="#FFFFFF" size=4>&nbsp;Predicted Secondary Structure</font></b></div>
<ul>
    <table border="0" style="font-family:Monospace;font-size:14px;background:#F2F2F2;">
    <tr><td valign="bottom"><font="Arial"><b>Sequence</b></font></td><td>
    &nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;20&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;40&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;60&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;80<br>
&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;|&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;|&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;|&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;|<br>
    MNKHFLFLFLLYCLIVAVTSLQCITCHLRTRTDRCRRGFGVCTAQKGEACMLLRIYQRNTLQISYMVCQKFCRDMTFDLRNRTYVHTCCNYNYCNFKL</td></tr>
    <tr><td><font="Arial"><span title="Predicted Secondary Structure">
    <b>Prediction</b></span></font></td><td>CCC<font color="blue">S</font><font color="red">HHHHHHHHHHHHHHH</font><font color="blue">SSSSSSSSSS</font>CCCCC<font color="blue">SS</font>CCCC<font color="blue">SSSS</font>CCCCC<font color="blue">SSSSSSSS</font>CC<font color="blue">SSSSSS</font><font color="red">HHHHHH</font>CCCCC<font color="blue">SS</font>CCC<font color="blue">SSSSSS</font>CCCCCCCCCCC</td></tr>
    <tr><td><font="Arial"><span title="Confidence Score of Prediction 
    (0 being least confident, 9 being most confident)">
    <b>Confidence</b></span></font></td><td>98311699999999985324554566677288862346337983564011133256199799980556654620013207679999717997666679</td></tr>
    <tr><td><font="Arial"></td><td><font color=red>H</font>:Helix; 
    <font color=blue>S</font>:Strand; C:Coil</td><tr>
    </table>
</ul>

<div style="background:#6599FF;width:500px;"><b><font face="Arial" 
color="#FFFFFF" size=4>&nbsp;Predicted Solvent Accessibility</font></b></div>
<ul>
    <table border="0" style="font-family:Monospace;font-size:14px;background:#F2F2F2;">
    <tr><td valign="bottom"><font="Arial"><b>Sequence</b></font></td><td>
    &nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;20&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;40&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;60&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;80<br>
&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;|&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;|&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;|&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;|<br>
    MNKHFLFLFLLYCLIVAVTSLQCITCHLRTRTDRCRRGFGVCTAQKGEACMLLRIYQRNTLQISYMVCQKFCRDMTFDLRNRTYVHTCCNYNYCNFKL</td></tr>
    <tr><td><font="Arial"><span title="Solvent Accessibility Prediction">
    <b>Prediction</b></span></font></td><td>85330023133313133144140310103257641442333031454332212312564322111320453066154444231102003646315575</td></tr>
    <tr><td><font="Arial"></td><td>Values range from <b>0</b> (buried residue) 
    to <b>8</b> (highly exposed residue)</td><tr>
    </table>
</ul>

<div style="background:#6599FF;width:500px;"><b><font face="Arial" 
color="#FFFFFF" size=4>&nbsp;Predicted contact map used in C-I-TASSER simulation</font></b></div>
<ul>
    <table border="0" style="font-family:Monospace;font-size:14px;background:#F2F2F2;">
    <tr><td><a href=contact.png><img src=contact.png width=1000></a></td></tr>
    <tr><td>
    <table cellspacing="2" border="0" width="1000px" style="font-family:Arial;font-size:12px;">
    <tr><td>C-I-TASSER simulation is guided by consensus contact map
    (upper left triangle) derived based on confidence scores of
    <a href=/TripletRes>TripletRes</a> (lower right triangle),
    <a href=/ResTriplet>ResTriplet</a>, <a href=/ResPRE>ResPRE</a>, and 
    <a href=/NeBcon>NeBcon</a>.
    In the contact map, the axes mark the residue indexes along the sequence,
    while each dot represents a residue pair with predicted contact.
    No dot is close to the diagonal, because C-I-TASSER does not consider
    contacts for residue pairs separated by &lt;6 residues.
    </td></tr>
    </table>
    </td></tr>
    </table>
</ul>

<div style="background:#6599FF;width:500px;"><b><font face="Arial" 
color="#FFFFFF" size=4>&nbsp;Top 10 threading templates used by C-I-TASSER</font></b></div>
<ul>
    <table border="0" style="font-family:Monospace;font-size:14px;background:#F2F2F2;">
    <tr><th align="left" valign="top"><font face="Arial">Rank</font></th> <th align="center" valign="top"><font face="Arial">PDB<br/>hit</font></th><th align="center" valign="top"><font face="Arial">ID1</font></th><th align="center" valign="top"><font face="Arial">ID2</font></th><th valign="top"><font face="Arial">Cov</font></th><th valign="top"><font face="Arial">Norm.<br/>Zscore</font></th><th valign="top"><font face="Arial">Download</br>alignment</font></th><td>&nbsp;</td><td BGCOLOR="#DEDEDE" style="font-family:Monospace;font-size:13px;">&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;20&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;40&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;60&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;80<br>
&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;|&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;|&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;|&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;|</td></tr>
    <tr><td colspan="7"></td><th align="right" valign ="bottom" ><font face="Arial">SS<br/>Seq</font></th><td BGCOLOR="#D7D7D7" style="font-family:Monospace;font-size:13px;">CCC<font color="blue">S</font><font color="red">HHHHHHHHHHHHHHH</font><font color="blue">SSSSSSSSSS</font>CCCCC<font color="blue">SS</font>CCCC<font color="blue">SSSS</font>CCCCC<font color="blue">SSSSSSSS</font>CC<font color="blue">SSSSSS</font><font color="red">HHHHHH</font>CCCCC<font color="blue">SS</font>CCC<font color="blue">SSSSSS</font>CCCCCCCCCCC<br/>MNKHFLFLFLLYCLIVAVTSLQCITCHLRTRTDRCRRGFGVCTAQKGEACMLLRIYQRNTLQISYMVCQKFCRDMTFDLRNRTYVHTCCNYNYCNFKL</td></tr>

    <tr><td align="center"><font face="Arial">1</font></td><td><a href="https://www.rcsb.org/pdb/explore/explore.do?structureId=6e7k" target=_blank>6e7kC</a></td><td align="center"><font face="Arial"> 0.26</font></td><td><font face="Arial"> 0.20</font></td><td><font face="Arial"> 6.29</font></td><td align="center"><font face="Arial"> <b>1.64</b></font></td><td align="center"><font face="Monospace"><a href="threading1.pdb.gz" download>SPARKS-K</a></font></td><td>&nbsp;</td><td BGCOLOR="#ECECEG" style="font-family:Monospace;font-size:13px;">-------------------L<font color=#80a0f0>L</font>R<font color=#f08080>C</font>Y<font color=#15c015>T</font><font color=#f08080>C</font>KSLP<font color=#f01505>R</font>DE<font color=#f01505>R</font><font color=#f08080>C</font>NL-TQN<font color=#f08080>C</font>S--H<font color=#f09048>G</font>QT<font color=#f08080>C</font>TT<font color=#80a0f0>L</font>IAHGSGL<font color=#80a0f0>L</font>TTHSTW<font color=#f08080>C</font>TDS<font color=#f08080>C</font>QPI<font color=#15c015>T</font>KTVEGTQVTM<font color=#15c015>T</font><font color=#f08080>C</font><font color=#f08080>C</font>QSSL<font color=#f08080>C</font><font color=#15c015>N</font>VPP</td></tr>
    <tr><td align="center"><font face="Arial">2</font></td><td><a href="https://www.rcsb.org/pdb/explore/explore.do?structureId=6e7k" target=_blank>6e7kC</a></td><td align="center"><font face="Arial"> 0.26</font></td><td><font face="Arial"> 0.19</font></td><td><font face="Arial"> 5.99</font></td><td align="center"><font face="Arial"> 0.81</font></td><td align="center"><font face="Monospace"><a href="threading2.pdb.gz" download>FFAS-3D</a></font></td><td>&nbsp;</td><td BGCOLOR="#ECECEG" style="font-family:Monospace;font-size:13px;">--------------------<font color=#80a0f0>L</font>R<font color=#f08080>C</font>Y<font color=#15c015>T</font><font color=#f08080>C</font>KSLP<font color=#f01505>R</font>DE<font color=#f01505>R</font><font color=#f08080>C</font>NL---TQNCSH<font color=#f09048>G</font>QT<font color=#f08080>C</font>TT<font color=#80a0f0>L</font>IAHGSGL<font color=#80a0f0>L</font>TTHSTW<font color=#f08080>C</font>TDS<font color=#f08080>C</font>QPI<font color=#15c015>T</font>KTVEGTQVTM<font color=#15c015>T</font><font color=#f08080>C</font><font color=#f08080>C</font>QSSL<font color=#f08080>C</font><font color=#15c015>N</font>VP-</td></tr>
    <tr><td align="center"><font face="Arial">3</font></td><td><a href="https://www.rcsb.org/pdb/explore/explore.do?structureId=6e7k" target=_blank>6e7kC</a></td><td align="center"><font face="Arial"> 0.25</font></td><td><font face="Arial"> 0.19</font></td><td><font face="Arial"> 6.01</font></td><td align="center"><font face="Arial"> <b>1.13</b></font></td><td align="center"><font face="Monospace"><a href="threading3.pdb.gz" download>HHpred</a></font></td><td>&nbsp;</td><td BGCOLOR="#ECECEG" style="font-family:Monospace;font-size:13px;">-------------------L<font color=#80a0f0>L</font>R<font color=#f08080>C</font>Y<font color=#15c015>T</font><font color=#f08080>C</font>KSLP<font color=#f01505>R</font>DE<font color=#f01505>R</font><font color=#f08080>C</font>N-LTQN<font color=#f08080>C</font>SHG--QT<font color=#f08080>C</font>TT<font color=#80a0f0>L</font>IAHGSGL<font color=#80a0f0>L</font>TTHSTW<font color=#f08080>C</font>TDS<font color=#f08080>C</font>QPI<font color=#15c015>T</font>KTVEGTQVTM<font color=#15c015>T</font><font color=#f08080>C</font><font color=#f08080>C</font>QSSL<font color=#f08080>C</font><font color=#15c015>N</font>VPP</td></tr>
    <tr><td align="center"><font face="Arial">4</font></td><td><a href="https://www.rcsb.org/pdb/explore/explore.do?structureId=6e7k" target=_blank>6e7kC</a></td><td align="center"><font face="Arial"> 0.25</font></td><td><font face="Arial"> 0.19</font></td><td><font face="Arial"> 6.01</font></td><td align="center"><font face="Arial"> 0.93</font></td><td align="center"><font face="Monospace"><a href="threading4.pdb.gz" download>MUSTER</a></font></td><td>&nbsp;</td><td BGCOLOR="#ECECEG" style="font-family:Monospace;font-size:13px;">-------------------L<font color=#80a0f0>L</font>R<font color=#f08080>C</font>Y<font color=#15c015>T</font><font color=#f08080>C</font>KSLP<font color=#f01505>R</font>DE<font color=#f01505>R</font><font color=#f08080>C</font>---NLTQNCSH<font color=#f09048>G</font>QT<font color=#f08080>C</font>TT<font color=#80a0f0>L</font>IAHGNGL<font color=#80a0f0>L</font>TTHSTW<font color=#f08080>C</font>TDS<font color=#f08080>C</font>QPI<font color=#15c015>T</font>KTVEGTQVTM<font color=#15c015>T</font><font color=#f08080>C</font><font color=#f08080>C</font>QSSL<font color=#f08080>C</font><font color=#15c015>N</font>VPP</td></tr>
    <tr><td align="center"><font face="Arial">5</font></td><td><a href="https://www.rcsb.org/pdb/explore/explore.do?structureId=2l03" target=_blank>2l03A</a></td><td align="center"><font face="Arial"> 0.27</font></td><td><font face="Arial"> 0.20</font></td><td><font face="Arial"> 6.26</font></td><td align="center"><font face="Arial"> 0.93</font></td><td align="center"><font face="Monospace"><a href="threading5.pdb.gz" download>CNFpred</a></font></td><td>&nbsp;</td><td BGCOLOR="#ECECEG" style="font-family:Monospace;font-size:13px;">-------------------M<font color=#80a0f0>L</font>D<font color=#f08080>C</font>HV<font color=#f08080>C</font>AYNG--<font color=#c048c0>D</font>N<font color=#f08080>C</font>F-NPMR<font color=#f08080>C</font>P<font color=#80a0f0>A</font>M-VAY<font color=#f08080>C</font><font color=#80a0f0>M</font>TT<font color=#f01505>R</font>T<font color=#15a4a4>Y</font>YTP<font color=#15c015>T</font>RMKVSKS<font color=#f08080>C</font>VPR<font color=#f08080>C</font>FETVY<font color=#c048c0>D</font>GYKHASTTS<font color=#f08080>C</font><font color=#f08080>C</font>Q<font color=#15a4a4>Y</font>DL<font color=#f08080>C</font><font color=#15c015>N</font>G--</td></tr>
    <tr><td align="center"><font face="Arial">6</font></td><td><a href="https://www.rcsb.org/pdb/explore/explore.do?structureId=2muo" target=_blank>2muoA</a></td><td align="center"><font face="Arial"> 0.25</font></td><td><font face="Arial"> 0.19</font></td><td><font face="Arial"> 6.02</font></td><td align="center"><font face="Arial"> <b>1.10</b></font></td><td align="center"><font face="Monospace"><a href="threading6.pdb.gz" download>HHsearch-2</a></font></td><td>&nbsp;</td><td BGCOLOR="#ECECEG" style="font-family:Monospace;font-size:13px;">-------------------M<font color=#80a0f0>L</font>K<font color=#f08080>C</font>Y<font color=#15c015>T</font><font color=#f08080>C</font>KEPMTSAS<font color=#f08080>C</font><font color=#f01505>R</font>T-ITR<font color=#f08080>C</font>KPE-DT<font color=#80a0f0>A</font><font color=#f08080>C</font><font color=#80a0f0>M</font>TTLVTVE<font color=#15c015>N</font>QSPVVTRS<font color=#f08080>C</font>SSS<font color=#f08080>C</font>VATDP<font color=#c048c0>D</font>SIGAAHLIF<font color=#f08080>C</font><font color=#f08080>C</font>FRDL<font color=#f08080>C</font><font color=#15c015>N</font>SE<font color=#80a0f0>L</font></td></tr>
    <tr><td align="center"><font face="Arial">7</font></td><td><a href="https://www.rcsb.org/pdb/explore/explore.do?structureId=2muo" target=_blank>2muoA</a></td><td align="center"><font face="Arial"> 0.22</font></td><td><font face="Arial"> 0.17</font></td><td><font face="Arial"> 5.46</font></td><td align="center"><font face="Arial"> <b>2.36</b></font></td><td align="center"><font face="Monospace"><a href="threading7.pdb.gz" download>Neff-PPAS</a></font></td><td>&nbsp;</td><td BGCOLOR="#ECECEG" style="font-family:Monospace;font-size:13px;">--------------------<font color=#80a0f0>L</font>K<font color=#f08080>C</font>Y<font color=#15c015>T</font><font color=#f08080>C</font>KEPMTSAS<font color=#f08080>C</font><font color=#f01505>R</font>TITRC--KPEDT<font color=#80a0f0>A</font><font color=#f08080>C</font><font color=#80a0f0>M</font>TTLVTVEAQSPVVTRS<font color=#f08080>C</font>SSS<font color=#f08080>C</font>VATDP<font color=#c048c0>D</font>SIGAAHLIF<font color=#f08080>C</font><font color=#f08080>C</font>FRDL<font color=#f08080>C</font><font color=#15c015>N</font>SE<font color=#80a0f0>L</font></td></tr>
    <tr><td align="center"><font face="Arial">8</font></td><td><a href="https://www.rcsb.org/pdb/explore/explore.do?structureId=6e7k" target=_blank>6e7kC</a></td><td align="center"><font face="Arial"> 0.25</font></td><td><font face="Arial"> 0.19</font></td><td><font face="Arial"> 6.00</font></td><td align="center"><font face="Arial"> <b>1.63</b></font></td><td align="center"><font face="Monospace"><a href="threading8.pdb.gz" download>HHsearch</a></font></td><td>&nbsp;</td><td BGCOLOR="#ECECEG" style="font-family:Monospace;font-size:13px;">-------------------L<font color=#80a0f0>L</font>R<font color=#f08080>C</font>Y<font color=#15c015>T</font><font color=#f08080>C</font>KSLP<font color=#f01505>R</font>DE<font color=#f01505>R</font><font color=#f08080>C</font>NL-TQN<font color=#f08080>C</font>SH-G-QT<font color=#f08080>C</font>TT<font color=#80a0f0>L</font>IAHGSGL<font color=#80a0f0>L</font>TTHSTW<font color=#f08080>C</font>TDS<font color=#f08080>C</font>QPI<font color=#15c015>T</font>KTVEGTQVTM<font color=#15c015>T</font><font color=#f08080>C</font><font color=#f08080>C</font>QSSL<font color=#f08080>C</font><font color=#15c015>N</font>VP-</td></tr>
    <tr><td align="center"><font face="Arial">9</font></td><td><a href="https://www.rcsb.org/pdb/explore/explore.do?structureId=6e7k" target=_blank>6e7kC</a></td><td align="center"><font face="Arial"> 0.25</font></td><td><font face="Arial"> 0.19</font></td><td><font face="Arial"> 6.01</font></td><td align="center"><font face="Arial"> 0.62</font></td><td align="center"><font face="Monospace"><a href="threading9.pdb.gz" download>PROSPECTOR2</a></font></td><td>&nbsp;</td><td BGCOLOR="#ECECEG" style="font-family:Monospace;font-size:13px;">L-------------------<font color=#80a0f0>L</font>R<font color=#f08080>C</font>Y<font color=#15c015>T</font><font color=#f08080>C</font>KSLP<font color=#f01505>R</font>DE<font color=#f01505>R</font><font color=#f08080>C</font>---NLTQNCSH<font color=#f09048>G</font>QT<font color=#f08080>C</font>TT<font color=#80a0f0>L</font>IAHESGL<font color=#80a0f0>L</font>TTHSTW<font color=#f08080>C</font>TDS<font color=#f08080>C</font>QPI<font color=#15c015>T</font>KTVEGTQVTM<font color=#15c015>T</font><font color=#f08080>C</font><font color=#f08080>C</font>QSSL<font color=#f08080>C</font><font color=#15c015>N</font>QSS</td></tr>
    <tr><td align="center"><font face="Arial">10</font></td><td><a href="https://www.rcsb.org/pdb/explore/explore.do?structureId=2muo" target=_blank>2muoA</a></td><td align="center"><font face="Arial"> 0.25</font></td><td><font face="Arial"> 0.18</font></td><td><font face="Arial"> 5.70</font></td><td align="center"><font face="Arial"> 0.37</font></td><td align="center"><font face="Monospace"><a href="threading10.pdb.gz" download>SAM</a></font></td><td>&nbsp;</td><td BGCOLOR="#ECECEG" style="font-family:Monospace;font-size:13px;">--------------------<font color=#80a0f0>L</font>K<font color=#f08080>C</font>Y<font color=#15c015>T</font><font color=#f08080>C</font>KEPMTSAS<font color=#f08080>C</font><font color=#f01505>R</font>T-ITR<font color=#f08080>C</font>KPED-T<font color=#80a0f0>A</font><font color=#f08080>C</font><font color=#80a0f0>M</font>TTLVTVF<font color=#15c015>N</font>QSPVVTRS<font color=#f08080>C</font>SSS<font color=#f08080>C</font>VATDP<font color=#c048c0>D</font>SIGAAHLIF<font color=#f08080>C</font><font color=#f08080>C</font>FRDL<font color=#f08080>C</font><font color=#15c015>N</font>---</td></tr>


    <tr><td colspan="9" align="left">
        <table cellspacing="2" border="0" width="1000px" style="font-family:Arial;font-size:12px;">
        <tr><td valign="top">(a)</td><td align="justify"><i>ID1</i> is the number of template residues identical to query divided by number of aligned residues.</td></tr>
        <tr><td valign="top">(b)</td><td align="justify"><i>ID2</i> is the number of template residues identical to query divided by query sequence length.</td></tr>
        <tr><td valign="top">(c)</td><td align="justify"><i>Cov</i> is equal the number of aligned template residues divided by query sequence length.</td></tr>
        <tr><td valign="top">(d)</td><td align="justify"><i>Norm. Zscore</i> is the normalized Z-score of the threading alignments. A Normalized Z-score &gt;1 means a good alignment and is highlighted in <b>bold</b>.</td></tr>
        <tr><td valign="top">(e)</td><td align="justify"><i>Download alignment</i> lists the threading program</a> used to identify the template, and provide the 3D structure of aligned regions of threading templates (threading[1-10].pdb.gz).</td></tr>
        <tr><td valign="top">(f)</td><td align="justify">Template residues identical to query sequence are highlighted in <a href=/C-I-TASSER/color.html target=_blank>color</a>.</td></tr>
        </table>
    </table>
</ul>


<div style="background:#6599FF;width:500px;"><b><font face="Arial" 
color="#FFFFFF" size=4>&nbsp;Top 5 final models from C-I-TASSER</font></b></div>
<font size=2 face="Times New Roman"><br>
</font>
<ul>
    <table border="0" style="font-size:14px;background:#F2F2F2;">

    <td align ="left">
    <script type="text/javascript">
    jmolInitialize("/jmol/");
    jmolSetAppletColor("#000000");
    jmolApplet(500,"load model1.pdb.gz; set frank off; calculate structure rama;  select all; color group; spacefill off; wireframe off; cartoons;set ambient 40;spin off;");
    jmolButton("Reset","Reset");
    jmolCheckbox("set spin x 10; set spin y 10; spin on",
        "spin off", "Spin", false);
    jmolCheckbox("set antialiasDisplay true",
        "set antialiasDisplay false", "High quality", false);
    jmolCheckbox("color background white",
        "color background black", "White background", false);
    jmolButton("write PNGJ model.png","Save image");
    </script>
    </td>

    <td valign="top">
    <table border="0" style="font-family:Arial;font-size:13px;">
    <tr bgcolor="#DEDEDE" align="center"><td>Click<br>to view</td>
    <td>Rank<sup>a</sup></td><td>Download</td><td>C-score<sup>b</sup></td>
    <td>Estimated TM-score<sup>c</sup></td><td>Estimated RMSD<sup>c</sup></td></tr>

    <tr align="center"><td><script type="text/javascript">
    jmolRadio("load model1.pdb.gz; set frank off; calculate structure rama;  select all; color group; spacefill off; wireframe off; cartoons;set ambient 40;","",true);
    </script></td>
    <td>1</td>
    <td><a href="model1.pdb.gz" download>model1.pdb.gz</a></td>
    <td>-1.38</td>
    <td>0.54&plusmn;0.15</td>
    <td>6.7&plusmn;4.0&Aring;</td></tr>

    
    <tr align="center"><td><script type="text/javascript">
    jmolRadio("load model2.pdb.gz; set frank off; calculate structure rama;  select all; color group; spacefill off; wireframe off; cartoons;set ambient 40;","",false);
    </script></td>
    <td>2</td>
    <td><a href="model2.pdb.gz" download>model2.pdb.gz</a></td>
    <td>-1.59</td></tr>

    <tr align="center"><td><script type="text/javascript">
    jmolRadio("load model3.pdb.gz; set frank off; calculate structure rama;  select all; color group; spacefill off; wireframe off; cartoons;set ambient 40;","",false);
    </script></td>
    <td>3</td>
    <td><a href="model3.pdb.gz" download>model3.pdb.gz</a></td>
    <td>-1.59</td></tr>

    <tr align="center"><td><script type="text/javascript">
    jmolRadio("load model4.pdb.gz; set frank off; calculate structure rama;  select all; color group; spacefill off; wireframe off; cartoons;set ambient 40;","",false);
    </script></td>
    <td>4</td>
    <td><a href="model4.pdb.gz" download>model4.pdb.gz</a></td>
    <td>-1.82</td></tr>

    <tr align="center"><td><script type="text/javascript">
    jmolRadio("load model5.pdb.gz; set frank off; calculate structure rama;  select all; color group; spacefill off; wireframe off; cartoons;set ambient 40;","",false);
    </script></td>
    <td>5</td>
    <td><a href="model5.pdb.gz" download>model5.pdb.gz</a></td>
    <td>-4.73</td></tr>

    </table>
    <table cellspacing="2" border="0" width="600px" style="font-family:Arial;font-size:12px;"><tr></tr><tr></tr>
    <tr><td valign="top">(a)</td><td>
    C-I-TASSER simulations generate a large ensemble of structural
    conformations, i.e. decoys. These decoys are clustered by
    <a href=/SPICKER target=_blank>SPICKER</a> based on pairwise structure similarity to
    report up to five final models from the five largest clusters. Models are 
    ranked in descending order of cluster size. If the simulations converge
    well, it is possible to have less than 5 models generated, which is
    usually an indication of good model quality.
    </td></tr>
    <tr><td valign="top">(b)</td><td>
    The model confidence is quantitatified by C-score, calculated based on
    significance of threading template alignments and convergence of
    C-I-TASSER simulations. C-score is typically in the range of [-5, 2],
    with higher C-score signifies higher model confidence.</td></tr>
    <tr><td valign="top">(c)</td><td>
    Model <a href=/TM-score target=_blank>TM-score</a> and RMSD are estimated based on
    C-score and protein length for the first model.
    </td></tr>
    </table>
    </td>
    </table>
</ul>


<div style="background:#6599FF;width:500px;"><b><font face="Arial" 
color="#FFFFFF" size=4>&nbsp;Proteins with similar structure</font></b></div>
<ul>
    <table border="0" style="background:#F2F2F2";align="left">
        <tr><td align ="center"><script type="text/javascript">
        jmolInitialize("/jmol/");
        jmolSetAppletColor("#000000");
        jmolApplet(500, "load model1_6e7kC.pdb.gz;set frank off;select all;spacefill off;wireframe off;select :T;trace 0.35;color trace mediumpurple;select :Q;cartoons;color group;set ambient 40;");
        jmolButton("Reset","Reset");
        jmolCheckbox("set spin x 10; set spin y 10; spin on",
            "spin off", "Spin", false);
        jmolCheckbox("set antialiasDisplay true",
            "set antialiasDisplay false", "High quality", false);
        jmolCheckbox("color background white",
            "color background black", "White background", false);
        jmolButton("write PNGJ model.png","Save image");
        </script></td>
        <td valign="top"><b>Top 10 structural analogs in PDB (as identified by
        <a href="https://zhanglab.ccmb.med.umich.edu/TM-align" target="_blank">
        TM-align</a>)</b><br/><br/>
        <table border="0" style="font-family:Arial;font-size:13px;">
        <tr><th>Click<br>to view</th><th>Rank</th><th>PDB<br/>Hit</th><th>TM-score</th><th>RMSD<sup>a</sup></th><th>IDEN<sup>a</sup></th><th>Cov.</th><th>Download<br/>Alignment</th></tr>
        
        <tr><td align="center"><script type="text/javascript">jmolRadio("load model1_6e7kC.pdb.gz; set frank off; select all; spacefill off; wireframe off; select :T; trace 0.35; color trace mediumpurple; select :Q; cartoons; color group; set ambient 40; set spin x 5; set spin y 5; spin off;", "", false);</script></td><td valign="middle" align="center">1</td><td valign="middle"><a href="https://www.rcsb.org/pdb/explore/explore.do?structureId=6e7k" target="_blank"><font face="Monospace">6e7kC</font></a></td><td align="center">0.68</td><td valign="middle" align="center">1.66</td><td valign="middle" align="center">0.237</td><td valign="middle">0.775</td><td><a href="model1_6e7kC.pdb.gz" download>model1_6e7kC.pdb.gz</a></td></tr>
        <tr><td align="center"><script type="text/javascript">jmolRadio("load model1_1ks6A.pdb.gz; set frank off; select all; spacefill off; wireframe off; select :T; trace 0.35; color trace mediumpurple; select :Q; cartoons; color group; set ambient 40; set spin x 5; set spin y 5; spin off;", "", false);</script></td><td valign="middle" align="center">2</td><td valign="middle"><a href="https://www.rcsb.org/pdb/explore/explore.do?structureId=1ks6" target="_blank"><font face="Monospace">1ks6A</font></a></td><td align="center">0.61</td><td valign="middle" align="center">2.44</td><td valign="middle" align="center">0.154</td><td valign="middle">0.786</td><td><a href="model1_1ks6A.pdb.gz" download>model1_1ks6A.pdb.gz</a></td></tr>
        <tr><td align="center"><script type="text/javascript">jmolRadio("load model1_6iomA.pdb.gz; set frank off; select all; spacefill off; wireframe off; select :T; trace 0.35; color trace mediumpurple; select :Q; cartoons; color group; set ambient 40; set spin x 5; set spin y 5; spin off;", "", false);</script></td><td valign="middle" align="center">3</td><td valign="middle"><a href="https://www.rcsb.org/pdb/explore/explore.do?structureId=6iom" target="_blank"><font face="Monospace">6iomA</font></a></td><td align="center">0.61</td><td valign="middle" align="center">3.06</td><td valign="middle" align="center">0.161</td><td valign="middle">0.888</td><td><a href="model1_6iomA.pdb.gz" download>model1_6iomA.pdb.gz</a></td></tr>
        <tr><td align="center"><script type="text/javascript">jmolRadio("load model1_1ploA.pdb.gz; set frank off; select all; spacefill off; wireframe off; select :T; trace 0.35; color trace mediumpurple; select :Q; cartoons; color group; set ambient 40; set spin x 5; set spin y 5; spin off;", "", false);</script></td><td valign="middle" align="center">4</td><td valign="middle"><a href="https://www.rcsb.org/pdb/explore/explore.do?structureId=1plo" target="_blank"><font face="Monospace">1ploA</font></a></td><td align="center">0.60</td><td valign="middle" align="center">2.62</td><td valign="middle" align="center">0.116</td><td valign="middle">0.796</td><td><a href="model1_1ploA.pdb.gz" download>model1_1ploA.pdb.gz</a></td></tr>
        <tr><td align="center"><script type="text/javascript">jmolRadio("load model1_3laqV1.pdb.gz; set frank off; select all; spacefill off; wireframe off; select :T; trace 0.35; color trace mediumpurple; select :Q; cartoons; color group; set ambient 40; set spin x 5; set spin y 5; spin off;", "", false);</script></td><td valign="middle" align="center">5</td><td valign="middle"><a href="https://www.rcsb.org/pdb/explore/explore.do?structureId=3laq" target="_blank"><font face="Monospace">3laqV1</font></a></td><td align="center">0.60</td><td valign="middle" align="center">2.28</td><td valign="middle" align="center">0.208</td><td valign="middle">0.735</td><td><a href="model1_3laqV1.pdb.gz" download>model1_3laqV1.pdb.gz</a></td></tr>
        <tr><td align="center"><script type="text/javascript">jmolRadio("load model1_6gbiA.pdb.gz; set frank off; select all; spacefill off; wireframe off; select :T; trace 0.35; color trace mediumpurple; select :Q; cartoons; color group; set ambient 40; set spin x 5; set spin y 5; spin off;", "", false);</script></td><td valign="middle" align="center">6</td><td valign="middle"><a href="https://www.rcsb.org/pdb/explore/explore.do?structureId=6gbi" target="_blank"><font face="Monospace">6gbiA</font></a></td><td align="center">0.60</td><td valign="middle" align="center">2.84</td><td valign="middle" align="center">0.200</td><td valign="middle">0.806</td><td><a href="model1_6gbiA.pdb.gz" download>model1_6gbiA.pdb.gz</a></td></tr>
        <tr><td align="center"><script type="text/javascript">jmolRadio("load model1_3bt1U1.pdb.gz; set frank off; select all; spacefill off; wireframe off; select :T; trace 0.35; color trace mediumpurple; select :Q; cartoons; color group; set ambient 40; set spin x 5; set spin y 5; spin off;", "", false);</script></td><td valign="middle" align="center">7</td><td valign="middle"><a href="https://www.rcsb.org/pdb/explore/explore.do?structureId=3bt1" target="_blank"><font face="Monospace">3bt1U1</font></a></td><td align="center">0.60</td><td valign="middle" align="center">2.36</td><td valign="middle" align="center">0.153</td><td valign="middle">0.735</td><td><a href="model1_3bt1U1.pdb.gz" download>model1_3bt1U1.pdb.gz</a></td></tr>
        <tr><td align="center"><script type="text/javascript">jmolRadio("load model1_2l03A.pdb.gz; set frank off; select all; spacefill off; wireframe off; select :T; trace 0.35; color trace mediumpurple; select :Q; cartoons; color group; set ambient 40; set spin x 5; set spin y 5; spin off;", "", false);</script></td><td valign="middle" align="center">8</td><td valign="middle"><a href="https://www.rcsb.org/pdb/explore/explore.do?structureId=2l03" target="_blank"><font face="Monospace">2l03A</font></a></td><td align="center">0.57</td><td valign="middle" align="center">2.57</td><td valign="middle" align="center">0.247</td><td valign="middle">0.725</td><td><a href="model1_2l03A.pdb.gz" download>model1_2l03A.pdb.gz</a></td></tr>
        <tr><td align="center"><script type="text/javascript">jmolRadio("load model1_2qjbD.pdb.gz; set frank off; select all; spacefill off; wireframe off; select :T; trace 0.35; color trace mediumpurple; select :Q; cartoons; color group; set ambient 40; set spin x 5; set spin y 5; spin off;", "", false);</script></td><td valign="middle" align="center">9</td><td valign="middle"><a href="https://www.rcsb.org/pdb/explore/explore.do?structureId=2qjb" target="_blank"><font face="Monospace">2qjbD</font></a></td><td align="center">0.53</td><td valign="middle" align="center">2.55</td><td valign="middle" align="center">0.162</td><td valign="middle">0.694</td><td><a href="model1_2qjbD.pdb.gz" download>model1_2qjbD.pdb.gz</a></td></tr>
        <tr><td align="center"><script type="text/javascript">jmolRadio("load model1_2h7zA.pdb.gz; set frank off; select all; spacefill off; wireframe off; select :T; trace 0.35; color trace mediumpurple; select :Q; cartoons; color group; set ambient 40; set spin x 5; set spin y 5; spin off;", "", false);</script></td><td valign="middle" align="center">10</td><td valign="middle"><a href="https://www.rcsb.org/pdb/explore/explore.do?structureId=2h7z" target="_blank"><font face="Monospace">2h7zA</font></a></td><td align="center">0.53</td><td valign="middle" align="center">3.00</td><td valign="middle" align="center">0.162</td><td valign="middle">0.694</td><td><a href="model1_2h7zA.pdb.gz" download>model1_2h7zA.pdb.gz</a></td></tr>
        </table><br/>

        <table border="0" cellspacing="2" style="font-family:Arial;font-size:12px;background:#F2F2F2;">
            <tr><td valign="top">(a)</td><td align="justify">Query structure is shown in cartoon, while the structural analog is displayed using backbone trace.</td></tr>
            <tr><td valign="top">(b)</td><td align="justify">Ranking of proteins is based on TM-score of the structural alignment between the query structure and known structures in the PDB library.</td></tr>
            <tr><td valign="top">(c)</td><td align="justify">RMSD<sup>a</sup> is the RMSD between residues that are structurally aligned by TM-align.</td></tr>
            <tr><td valign="top">(d)</td><td align="justify">IDEN<sup>a</sup> is the percentage sequence identity in the structurally aligned region.</td></tr>
            <tr><td valign="top">(e)</td><td align="justify">Cov. represents the coverage of the alignment by TM-align and is equal to the number of structurally aligned residues divided by length of the query protein.</td></tr>
        </table>
    </td></tr>
    </table>
</ul>

<div style="background:#6599FF;width:500px;"><b><font face="Arial" 
color="#FFFFFF" size=4>&nbsp;Predicted Gene Ontology (GO) Terms</font></b></div>
<ul>
<table border="0" style="background:#F2F2F2";align="left"><tr><td>

<table border="0" style="background:#F2F2F2";align="left"><tr>
<td align ="center" style="background:#FFFFFF">
    <a href="GOsearchresult_final_MF.svg" target="_blank">
    <img width=500 height=500 border="1" style="border-color:black" alt="GOsearchresult_final_MF.svg" src="GOsearchresult_final_MF.svg">
    </a>
</td>
<td valign="top" align="left">
    <table border="0">
        <tr><td><b>Molecular Function (MF)</b><br></tr></td>
    </table>
    <table border="0" style="font-family:Arial;font-size:13px;">
        <tr><td align="left"><b>GO term</b></td><td align="center"><b>Cscore<sup>GO</sup></b></td><td><b>Name</b></td></tr>
        
        <tr><td align="left"><a href="http://amigo.geneontology.org/amigo/term/GO:0003824" target="_blank">GO:0003824</a></td><td align="center">0.69</td><td>catalytic activity</td></tr>
        <tr><td align="left"><a href="http://amigo.geneontology.org/amigo/term/GO:0016740" target="_blank">GO:0016740</a></td><td align="center">0.64</td><td>transferase activity</td></tr>
        <tr><td align="left"><a href="http://amigo.geneontology.org/amigo/term/GO:0099600" target="_blank">GO:0099600</a></td><td align="center">0.62</td><td>transmembrane receptor activity</td></tr>
        <tr><td align="left"><a href="http://amigo.geneontology.org/amigo/term/GO:0004674" target="_blank">GO:0004674</a></td><td align="center">0.61</td><td>protein serine/threonine kinase activity</td></tr>
        <tr><td align="left"><a href="http://amigo.geneontology.org/amigo/term/GO:0004675" target="_blank">GO:0004675</a></td><td align="center">0.60</td><td>transmembrane receptor protein serine/threonine kinase activity</td></tr>
    </table><br>
    <table border="0" style="font-family:Arial;font-size:13px;">
        <tr><td>Download <a href="GOsearchresult_final_MF.csv" download>full result</a> of the above consensus prediction.</tr></td>
    </table><br>
    <table cellspacing="2" border="0" style="font-family:Arial;font-size:12px;">
        <tr><td valign="top"></td><td align="justify"><b>Click the graph to show a high resolution version.</b></td></tr>
        <tr><td valign="top">(a)</td><td align="justify">Cscore<sup>GO</sup> is the confidence score of predicted GO terms. Cscore<sup>GO</sup> values range in between [0-1]; where a higher value indicates a better confidence in predicting the function using the template.</td></tr>
        <tr><td valign="top">(b)</td><td align="justify">The graph shows the predicted terms within the Gene Ontology hierachy for Molecular Function. Confidently predicted terms are color coded by Cscore<sup>GO</sup>:</td></tr>
        <tr><td valign="top"></td><td align="justify"><table cellspacing="2" border="0" style="border-color:black;font-family:Arial;font-size:12px;"><tr><td style="background-color:magenta;">[0.4,0.5)</td><td style="background-color:cyan;">[0.5,0.6)</td><td style="background-color:green;">[0.6,0.7)</td><td style="background-color:yellow;">[0.7,0.8)</td><td style="background-color:orange;">[0.8,0.9)</td><td style="background-color:red;">[0.9,1.0]</td></tr></table></td></tr>
     </table>
</td>
</tr>
</table>


<table border="0" style="background:#F2F2F2";align="left"><tr>
<td align ="center" style="background:#FFFFFF">
    <a href="GOsearchresult_final_BP.svg" target="_blank">
    <img width=500 height=500 border="1" style="border-color:black" alt="GOsearchresult_final_BP.svg" src="GOsearchresult_final_BP.svg">
    </a>
</td>
<td valign="top" align="left">
    <table border="0">
        <tr><td><b>Biological Process (BP)</b><br></tr></td>
    </table>
    <table border="0" style="font-family:Arial;font-size:13px;">
        <tr><td align="left"><b>GO term</b></td><td align="center"><b>Cscore<sup>GO</sup></b></td><td><b>Name</b></td></tr>
        
        <tr><td align="left"><a href="http://amigo.geneontology.org/amigo/term/GO:0044699" target="_blank">GO:0044699</a></td><td align="center">0.98</td><td>single-organism process</td></tr>
        <tr><td align="left"><a href="http://amigo.geneontology.org/amigo/term/GO:0044767" target="_blank">GO:0044767</a></td><td align="center">0.88</td><td>single-organism developmental process</td></tr>
        <tr><td align="left"><a href="http://amigo.geneontology.org/amigo/term/GO:0009987" target="_blank">GO:0009987</a></td><td align="center">0.76</td><td>cellular process</td></tr>
        <tr><td align="left"><a href="http://amigo.geneontology.org/amigo/term/GO:0065007" target="_blank">GO:0065007</a></td><td align="center">0.73</td><td>biological regulation</td></tr>
        <tr><td align="left"><a href="http://amigo.geneontology.org/amigo/term/GO:0050789" target="_blank">GO:0050789</a></td><td align="center">0.73</td><td>regulation of biological process</td></tr>
        <tr><td align="left"><a href="http://amigo.geneontology.org/amigo/term/GO:0071704" target="_blank">GO:0071704</a></td><td align="center">0.67</td><td>organic substance metabolic process</td></tr>
        <tr><td align="left"><a href="http://amigo.geneontology.org/amigo/term/GO:0051704" target="_blank">GO:0051704</a></td><td align="center">0.63</td><td>multi-organism process</td></tr>
        <tr><td align="left"><a href="http://amigo.geneontology.org/amigo/term/GO:0044260" target="_blank">GO:0044260</a></td><td align="center">0.63</td><td>cellular macromolecule metabolic process</td></tr>
        <tr><td align="left"><a href="http://amigo.geneontology.org/amigo/term/GO:0009405" target="_blank">GO:0009405</a></td><td align="center">0.58</td><td>pathogenesis</td></tr>
        <tr><td align="left"><a href="http://amigo.geneontology.org/amigo/term/GO:0009653" target="_blank">GO:0009653</a></td><td align="center">0.57</td><td>anatomical structure morphogenesis</td></tr>
        <tr><td align="left"><a href="http://amigo.geneontology.org/amigo/term/GO:0051239" target="_blank">GO:0051239</a></td><td align="center">0.56</td><td>regulation of multicellular organismal process</td></tr>
        <tr><td align="left"><a href="http://amigo.geneontology.org/amigo/term/GO:0048518" target="_blank">GO:0048518</a></td><td align="center">0.56</td><td>positive regulation of biological process</td></tr>
        <tr><td align="left"><a href="http://amigo.geneontology.org/amigo/term/GO:2000026" target="_blank">GO:2000026</a></td><td align="center">0.55</td><td>regulation of multicellular organismal development</td></tr>
        <tr><td align="left"><a href="http://amigo.geneontology.org/amigo/term/GO:0051240" target="_blank">GO:0051240</a></td><td align="center">0.52</td><td>positive regulation of multicellular organismal process</td></tr>
        <tr><td align="left"><a href="http://amigo.geneontology.org/amigo/term/GO:0009887" target="_blank">GO:0009887</a></td><td align="center">0.52</td><td>organ morphogenesis</td></tr>
        <tr><td align="left"><a href="http://amigo.geneontology.org/amigo/term/GO:0051094" target="_blank">GO:0051094</a></td><td align="center">0.51</td><td>positive regulation of developmental process</td></tr>
        <tr><td align="left"><a href="http://amigo.geneontology.org/amigo/term/GO:0048856" target="_blank">GO:0048856</a></td><td align="center">0.51</td><td>anatomical structure development</td></tr>
    </table><br>
    <table border="0" style="font-family:Arial;font-size:13px;">
        <tr><td>Download <a href="GOsearchresult_final_BP.csv" download>full result</a> of the above consensus prediction.</tr></td>
    </table><br>
    <table cellspacing="2" border="0" style="font-family:Arial;font-size:12px;">
        <tr><td valign="top"></td><td align="justify"><b>Click the graph to show a high resolution version.</b></td></tr>
        <tr><td valign="top">(a)</td><td align="justify">Cscore<sup>GO</sup> is the confidence score of predicted GO terms. Cscore<sup>GO</sup> values range in between [0-1]; where a higher value indicates a better confidence in predicting the function using the template.</td></tr>
        <tr><td valign="top">(b)</td><td align="justify">The graph shows the predicted terms within the Gene Ontology hierachy for Biological Process. Confidently predicted terms are color coded by Cscore<sup>GO</sup>:</td></tr>
        <tr><td valign="top"></td><td align="justify"><table cellspacing="2" border="0" style="border-color:black;font-family:Arial;font-size:12px;"><tr><td style="background-color:magenta;">[0.4,0.5)</td><td style="background-color:cyan;">[0.5,0.6)</td><td style="background-color:green;">[0.6,0.7)</td><td style="background-color:yellow;">[0.7,0.8)</td><td style="background-color:orange;">[0.8,0.9)</td><td style="background-color:red;">[0.9,1.0]</td></tr></table></td></tr>
     </table>
</td>
</tr>
</table>


<table border="0" style="background:#F2F2F2";align="left"><tr>
<td align ="center" style="background:#FFFFFF">
    <a href="GOsearchresult_final_CC.svg" target="_blank">
    <img width=500 height=500 border="1" style="border-color:black" alt="GOsearchresult_final_CC.svg" src="GOsearchresult_final_CC.svg">
    </a>
</td>
<td valign="top" align="left">
    <table border="0">
        <tr><td><b>Cellular Component (CC)</b><br></tr></td>
    </table>
    <table border="0" style="font-family:Arial;font-size:13px;">
        <tr><td align="left"><b>GO term</b></td><td align="center"><b>Cscore<sup>GO</sup></b></td><td><b>Name</b></td></tr>
        
        <tr><td align="left"><a href="http://amigo.geneontology.org/amigo/term/GO:0044464" target="_blank">GO:0044464</a></td><td align="center">0.89</td><td>cell part</td></tr>
        <tr><td align="left"><a href="http://amigo.geneontology.org/amigo/term/GO:0005576" target="_blank">GO:0005576</a></td><td align="center">0.82</td><td>extracellular region</td></tr>
        <tr><td align="left"><a href="http://amigo.geneontology.org/amigo/term/GO:0044424" target="_blank">GO:0044424</a></td><td align="center">0.71</td><td>intracellular part</td></tr>
        <tr><td align="left"><a href="http://amigo.geneontology.org/amigo/term/GO:0043226" target="_blank">GO:0043226</a></td><td align="center">0.69</td><td>organelle</td></tr>
        <tr><td align="left"><a href="http://amigo.geneontology.org/amigo/term/GO:0043227" target="_blank">GO:0043227</a></td><td align="center">0.65</td><td>membrane-bounded organelle</td></tr>
        <tr><td align="left"><a href="http://amigo.geneontology.org/amigo/term/GO:0043229" target="_blank">GO:0043229</a></td><td align="center">0.62</td><td>intracellular organelle</td></tr>
        <tr><td align="left"><a href="http://amigo.geneontology.org/amigo/term/GO:0016020" target="_blank">GO:0016020</a></td><td align="center">0.61</td><td>membrane</td></tr>
        <tr><td align="left"><a href="http://amigo.geneontology.org/amigo/term/GO:0044444" target="_blank">GO:0044444</a></td><td align="center">0.60</td><td>cytoplasmic part</td></tr>
        <tr><td align="left"><a href="http://amigo.geneontology.org/amigo/term/GO:0043231" target="_blank">GO:0043231</a></td><td align="center">0.56</td><td>intracellular membrane-bounded organelle</td></tr>
    </table><br>
    <table border="0" style="font-family:Arial;font-size:13px;">
        <tr><td>Download <a href="GOsearchresult_final_CC.csv" download>full result</a> of the above consensus prediction.</tr></td>
    </table><br>
    <table cellspacing="2" border="0" style="font-family:Arial;font-size:12px;">
        <tr><td valign="top"></td><td align="justify"><b>Click the graph to show a high resolution version.</b></td></tr>
        <tr><td valign="top">(a)</td><td align="justify">Cscore<sup>GO</sup> is the confidence score of predicted GO terms. Cscore<sup>GO</sup> values range in between [0-1]; where a higher value indicates a better confidence in predicting the function using the template.</td></tr>
        <tr><td valign="top">(b)</td><td align="justify">The graph shows the predicted terms within the Gene Ontology hierachy for Cellular Component. Confidently predicted terms are color coded by Cscore<sup>GO</sup>:</td></tr>
        <tr><td valign="top"></td><td align="justify"><table cellspacing="2" border="0" style="border-color:black;font-family:Arial;font-size:12px;"><tr><td style="background-color:magenta;">[0.4,0.5)</td><td style="background-color:cyan;">[0.5,0.6)</td><td style="background-color:green;">[0.6,0.7)</td><td style="background-color:yellow;">[0.7,0.8)</td><td style="background-color:orange;">[0.8,0.9)</td><td style="background-color:red;">[0.9,1.0]</td></tr></table></td></tr>
     </table>
</td>
</tr>
</table>

</td></tr>
</table>
</ul>

<div style="background:#6599FF;width:500px;"><b><font face="Arial" 
color="#FFFFFF" size=4>&nbsp;Predicted Enzyme Commission (EC) Numbers</font></b></div>
<ul>
    <table border="0" style="background:#F2F2F2";align="left">
        <tr><td align ="center"><script type="text/javascript">
        jmolInitialize("/jmol/");
        jmolSetAppletColor("#000000");
        jmolApplet(500, "load model1.pdb.gz;set frank off;select all;color group;spacefill off;wireframe off;cartoons;;set spin x 10; set spin y 10; spin off;");
        jmolButton("Reset","Reset");
        jmolCheckbox("set spin x 10; set spin y 10; spin on",
            "spin off", "Spin", false);
        jmolCheckbox("set antialiasDisplay true",
            "set antialiasDisplay false", "High quality", false);
        jmolCheckbox("color background white",
            "color background black", "White background", false);
        jmolButton("write PNGJ model.png","Save image");
        </script></td>
        <td valign="top"><b>Top 5 enzyme homologs in PDB</b><br/><br/>
        <table border="0" style="font-family:Arial;font-size:13px;">
        <tr><th>Click<br>to view</th><th>Rank</th><th>Cscore<sup>EC</sup></th><th>PDB<br/>Hit</th><th>TM-score</th><th>RMSD<sup>a</sup></th><th>IDEN<sup>a</sup></th><th>Cov.</th><th>EC Number</th><th>Predicted Active Site Residues</th></tr>
        
        <tr><td align="center"><script type="text/javascript">jmolRadio("select all;wireframe off;spacefill off;label off;;set spin x 5;set spin y 5;spin off;","",false);</script></td><td align="left">1</td><td align="center">0.066</td><td><a href="https://www.rcsb.org/pdb/explore/explore.do?structureId=2h62" target="_blank"><font face="Monospace">2h62C</font></a></td><td align="center">0.515</td><td align="center">2.57</td><td align="center">0.125</td><td align="right">0.673</td><td align="left"><span title=""><a href="http://enzyme.expasy.org/EC/2.7.11.30" target="_blank">2.7.11.30</a></span>&nbsp;</td><td align="left">NA</td></tr>
        <tr><td align="center"><script type="text/javascript">jmolRadio("select all;wireframe off;spacefill off;label off;;set spin x 5;set spin y 5;spin off;","",false);</script></td><td align="left">2</td><td align="center">0.060</td><td><a href="https://www.rcsb.org/pdb/explore/explore.do?structureId=1snz" target="_blank"><font face="Monospace">1snzA</font></a></td><td align="center">0.463</td><td align="center">4.22</td><td align="center">0.043</td><td align="right">0.827</td><td align="left"><span title=""><a href="http://enzyme.expasy.org/EC/5.1.3.3" target="_blank">5.1.3.3</a></span>&nbsp;</td><td align="left">NA</td></tr>
        <tr><td align="center"><script type="text/javascript">jmolRadio("select all;wireframe off;spacefill off;label off;;set spin x 5;set spin y 5;spin off;","",false);</script></td><td align="left">3</td><td align="center">0.060</td><td><a href="https://www.rcsb.org/pdb/explore/explore.do?structureId=2h62" target="_blank"><font face="Monospace">2h62D</font></a></td><td align="center">0.537</td><td align="center">2.51</td><td align="center">0.158</td><td align="right">0.694</td><td align="left"><span title=""><a href="http://enzyme.expasy.org/EC/2.7.11.30" target="_blank">2.7.11.30</a></span>&nbsp;</td><td align="left">NA</td></tr>
        <tr><td align="center"><script type="text/javascript">jmolRadio("select all;wireframe off;spacefill off;label off;;set spin x 5;set spin y 5;spin off;","",false);</script></td><td align="left">4</td><td align="center">0.060</td><td><a href="https://www.rcsb.org/pdb/explore/explore.do?structureId=3ig5" target="_blank"><font face="Monospace">3ig5A</font></a></td><td align="center">0.479</td><td align="center">3.80</td><td align="center">0.032</td><td align="right">0.775</td><td align="left"><span title=""><a href="http://enzyme.expasy.org/EC/6.3.2.2" target="_blank">6.3.2.2</a></span>&nbsp;</td><td align="left">NA</td></tr>
        <tr><td align="center"><script type="text/javascript">jmolRadio("select all;wireframe off;spacefill off;label off;;set spin x 5;set spin y 5;spin off;","",false);</script></td><td align="left">5</td><td align="center">0.060</td><td><a href="https://www.rcsb.org/pdb/explore/explore.do?structureId=1f1h" target="_blank"><font face="Monospace">1f1hA</font></a></td><td align="center">0.455</td><td align="center">4.46</td><td align="center">0.053</td><td align="right">0.837</td><td align="left"><span title=""><a href="http://enzyme.expasy.org/EC/6.3.1.2" target="_blank">6.3.1.2</a></span>&nbsp;</td><td align="left">NA</td></tr>
        </table><br/>

        <table border="0" cellspacing="2" border="0" style="font-family:Arial;font-size:12px;">
            <tr><td valign="top">&nbsp;</td><td align="justify"><b>Click on the radio buttons to visualize predicted active site residues.</b></td></tr>
            <tr><td valign="top">(a)</td><td align="justify">Cscore<sup>EC</sup> is the confidence score for the Enzyme Commission (EC) number prediction. Cscore<sup>EC</sup> values range in between [0-1]; where a higher score indicates a more reliable EC number prediction.</td></tr>
            <tr><td valign="top">(b)</td><td align="justify">TM-score is a measure of  global structural similarity between query and template protein.</td></tr>
            <tr><td valign="bottom">(c)</td><td align="justify">RMSD<sup>a</sup> is the RMSD between residues that are structurally aligned by TM-align.</td></tr>
            <tr><td valign="bottom">(d)</td><td align="justify">IDEN<sup>a</sup> is the percentage sequence identity in the structurally aligned region.</td></tr>
            <tr><td valign="top">(e)</td><td align="justify">Cov. represents the coverage of global structural  alignment and is equal to the number of structurally aligned residues divided by length of the query protein.</td></tr>
        </table>
    </td></tr>
    </table>
</ul>

<div style="background:#6599FF;width:500px;"><b><font face="Arial" 
color="#FFFFFF" size=4>&nbsp;Predicted Ligand Binding Sites</font></b></div>
<ul>
    <table border="0" style="background:#F2F2F2";align="left">
        <tr><td align ="center"><script type="text/javascript">
        jmolInitialize("/jmol/");
        jmolSetAppletColor("#000000");
        jmolApplet(500, "load complex1.pdb.gz;set frank off;select all;color group;spacefill off;wireframe off;cartoons;select hetero;spacefill 95%;set radius 1.0;color cpk;set ambient 40;select 51;wireframe 70;spacefill 100;color cpk;select 51 and *.ca; label %m%R; color label magenta;select 67;wireframe 70;spacefill 100;color cpk;select 67 and *.ca; label %m%R; color label magenta;select 84;wireframe 70;spacefill 100;color cpk;select 84 and *.ca; label %m%R; color label magenta;select 86;wireframe 70;spacefill 100;color cpk;select 86 and *.ca; label %m%R; color label magenta;;set spin x 10; set spin y 10; spin off;");
        jmolButton("Reset","Reset");
        jmolCheckbox("set spin x 10; set spin y 10; spin on",
            "spin off", "Spin", false);
        jmolCheckbox("set antialiasDisplay true",
            "set antialiasDisplay false", "High quality", false);
        jmolCheckbox("color background white",
            "color background black", "White background", false);
        jmolButton("write PNGJ model.png","Save image");
        </script></td>
        <td valign="top"><b>Template proteins with similar binding site:</b><br/><br/>
        <table border="0" style="font-family:Arial;font-size:13px;">
        <tr><th>Click<br>to view</th><th>Rank</th><th>Cscore<sup>LB</sup></th><th>PDB<br/>Hit</th><th>TM-score</th><th>RMSD<sup>a</sup></th><th>IDEN<sup>a</sup></th><th>Cov.</th><th>BS-score</th><th>Lig. Name</th><th>Download<br/>Complex</th><th align="left">Predicted binding site residues</th></tr>
        
        <tr><td align="center"><script type="text/javascript">jmolRadio("load complex1.pdb.gz;set frank off;select all;color group;spacefill off;wireframe off;cartoons;select hetero;spacefill 95%;set radius 1.0;color cpk;set ambient 40;select 51;wireframe 70;spacefill 100;color cpk;select 51 and *.ca; label %m%R; color label magenta;select 67;wireframe 70;spacefill 100;color cpk;select 67 and *.ca; label %m%R; color label magenta;select 84;wireframe 70;spacefill 100;color cpk;select 84 and *.ca; label %m%R; color label magenta;select 86;wireframe 70;spacefill 100;color cpk;select 86 and *.ca; label %m%R; color label magenta;;set spin x 10;set spin y 10;spin off;","",false);</script></td><td align="left">1</td><td align="center">0.02</td><td><a href="https://www.rcsb.org/pdb/explore/explore.do?structureId=2h62" target="_blank"><font face="Monospace">2h622</font></a></td><td align="center">0.515</td><td align="center">2.57</td><td align="center">0.125</td><td align="right">0.673</td><td align="left">0.49</td><td align="left"><font face="Monospace"><a href="http://zhanglab.ccmb.med.umich.edu/BioLiP/sym.cgi?code=III" target="_blank">III</a></font></td><td><a href="complex1.pdb.gz" download>complex1.pdb.gz</a></td><td align="left">51,67,84,86</td></tr>
        <tr><td align="center"><script type="text/javascript">jmolRadio("load complex2.pdb.gz;set frank off;select all;color group;spacefill off;wireframe off;cartoons;select hetero;spacefill 95%;set radius 1.0;color cpk;set ambient 40;select 39;wireframe 70;spacefill 100;color cpk;select 39 and *.ca; label %m%R; color label magenta;select 68;wireframe 70;spacefill 100;color cpk;select 68 and *.ca; label %m%R; color label magenta;select 69;wireframe 70;spacefill 100;color cpk;select 69 and *.ca; label %m%R; color label magenta;select 70;wireframe 70;spacefill 100;color cpk;select 70 and *.ca; label %m%R; color label magenta;;set spin x 10;set spin y 10;spin off;","",false);</script></td><td align="left">2</td><td align="center">0.02</td><td><a href="https://www.rcsb.org/pdb/explore/explore.do?structureId=2h64" target="_blank"><font face="Monospace">2h644</font></a></td><td align="center">0.521</td><td align="center">2.71</td><td align="center">0.122</td><td align="right">0.694</td><td align="left">0.46</td><td align="left"><font face="Monospace"><a href="http://zhanglab.ccmb.med.umich.edu/BioLiP/sym.cgi?code=III" target="_blank">III</a></font></td><td><a href="complex2.pdb.gz" download>complex2.pdb.gz</a></td><td align="left">39,68,69,70</td></tr>
        <tr><td align="center"><script type="text/javascript">jmolRadio("load complex3.pdb.gz;set frank off;select all;color group;spacefill off;wireframe off;cartoons;select hetero;spacefill 95%;set radius 1.0;color cpk;set ambient 40;select 51;wireframe 70;spacefill 100;color cpk;select 51 and *.ca; label %m%R; color label magenta;select 53;wireframe 70;spacefill 100;color cpk;select 53 and *.ca; label %m%R; color label magenta;select 67;wireframe 70;spacefill 100;color cpk;select 67 and *.ca; label %m%R; color label magenta;select 84;wireframe 70;spacefill 100;color cpk;select 84 and *.ca; label %m%R; color label magenta;select 86;wireframe 70;spacefill 100;color cpk;select 86 and *.ca; label %m%R; color label magenta;;set spin x 10;set spin y 10;spin off;","",false);</script></td><td align="left">3</td><td align="center">0.02</td><td><a href="https://www.rcsb.org/pdb/explore/explore.do?structureId=2goo" target="_blank"><font face="Monospace">2goo1</font></a></td><td align="center">0.511</td><td align="center">2.72</td><td align="center">0.123</td><td align="right">0.684</td><td align="left">0.45</td><td align="left"><font face="Monospace"><a href="http://zhanglab.ccmb.med.umich.edu/BioLiP/sym.cgi?code=III" target="_blank">III</a></font></td><td><a href="complex3.pdb.gz" download>complex3.pdb.gz</a></td><td align="left">51,53,67,84,86</td></tr>
        </table><br/>

        <table border="0" cellspacing="2" style="font-family:Arial;font-size:12px;">
            <tr><td valign="top">&nbsp;</td><td align="justify"><b> Click on the radio buttons to visualize predicted binding site and residues.</b></td></tr>
            <tr><td valign="top">(a)</td><td align="justify">Cscore<sup>LB</sup> is the confidence score of predicted binding site. Cscore<sup>LB</sup> values range in between [0-1]; where a higher score indicates a more reliable ligand-binding site prediction. </td></tr>
            <tr><td valign="top">(b)</td><td align="justify">BS-score is a measure of local similarity (sequence & structure) between template binding site and predicted binding site in the query structure. Based on large scale benchmarking analysis, we have observed that a BS-score >1 reflects a significant local match between the predicted and template binding site.
            <tr><td valign="top">(c)</td><td align="justify">TM-score is a measure of  global structural similarity between query and template protein.</td></tr>
            <tr><td valign="top">(d)</td><td align="justify">RMSD<sup>a</sup> the RMSD between residues that are structurally aligned by TM-align.</td></tr>
            <tr><td valign="top">(e)</td><td align="justify">IDEN<sup>a</sup> is the percentage sequence identity in the structurally aligned region.</td></tr>
            <tr><td valign="top">(f)</td><td align="justify">Cov. represents the coverage of global structural alignment and is equal to the number of structurally aligned residues divided by length of the query protein.</td></tr>
        </table>
    </td></tr>
    </table>

</ul>


</font>
<h4 align="center">[Click <a href="result.zip" download>result.zip</a>
    to download all results on this page]</h4>
<hr><br/>
<div style="position:relative;left:5px;">
    <table style="font-family:Arial;font-size:14px;">
    <tr><td colspan=2>References:</td><tr>
    <tr><td valign="top">1.</td><td>Chengxin Zhang, S M Mortuza, Baoji He, Yanting Wang, and Yang Zhang. "Template-based and free modeling of I-TASSER and QUARK pipelines using predicted contact maps in CASP12." Proteins: Structure, Function, and Bioinformatics (2017). doi: 10.1002/prot.25414</td></tr>
    </table>
</div><br>
</body></html>
