<html>
<head><meta http-equiv="content-type" content="text/html; charset=UTF-8">
<title>I-TASSER/COFACTOR NX_Q96A22</title></head>

<script type="text/javascript" src="/jmol/JSmol.min.js"></script>
<script type="text/javascript" src="/jmol/Jmol2.js"></script>
<script type="text/javascript">
function formAction (action, id)
{
    if (action == 'show')
    {
        document.getElementById("optionForm"+id).style.display= "inline";
        document.getElementById("optionGraphic"+id).src="/COFACTOR/img/down_triangle.gif";
        document.getElementById("option"+id).href= "javascript:formAction('hide', "+id+")";
    }
    if (action == 'hide')
    {
        document.getElementById("optionForm"+id).style.display= "none";
        document.getElementById("optionGraphic"+id).src="/COFACTOR/img/right_triangle.gif";
        document.getElementById("option"+id).href= "javascript:formAction('show', "+id+")";
    }
}
</script>
<body>
<table style="display:block;overflow:auto; height:100%; white-space:nowrap; font-size:14px;">
<td>

<font face="Times New Roman">
<font size=4><b><a href="/I-TASSER" target="_top">I-TASSER</a>/<a href="/COFACTOR" target="_top">COFACTOR</a>
results for NX_Q96A22
[<a href="result.zip" download>Download data on this page</a>]
[<a href="." target="_blank">Open in new tab</a>]
</b></font><br></br>

<table><tr><td><div style="background:#6599FF;width:500px;"><b><font face="Arial" 
color="#FFFFFF" size=4>&nbsp;Input Sequence in FASTA format</font></b></div></td>
<td><a id="option1" href="javascript:formAction('hide',1)">
<img id="optionGraphic1" border=0 src="/COFACTOR/img/down_triangle.gif"/>
[Show/Hide]</a></td></tr></table>
<table id="optionForm1"><tr><td><ul>
    <table style="font-family:Monospace;font-size:14px;background:#F2F2F2;">
    <tr><td colspan="2">&gt;NX_Q96A22 (123 residues)<br>
    MGNRVCCGGSWSCPSTFQKKKKTGSQTRRTLKPQPQQLQQNLPKGHETTGHTYERVLQQQ<br>
GSQERSPGLMSEDSNLHYADIQVCSRPHAREVKHVHLENATEYATLRFPQATPRYDSKNG<br>
TLV<br>
    </td></tr>
    </table>
</ul></td></tr></table>

<table><tr><td><div style="background:#6599FF;width:500px;"><b><font face="Arial" 
color="#FFFFFF" size=4>&nbsp;Predicted Secondary Structure</font></b></div></td>
<td><a id="option2" href="javascript:formAction('hide',2)">
<img id="optionGraphic2" border=0 src="/COFACTOR/img/down_triangle.gif"/>
[Show/Hide]</a></td></tr></table>
<table id="optionForm2"><tr><td><ul>
    <table border="0" style="font-family:Monospace;font-size:14px;background:#F2F2F2;">
    <tr><td valign="bottom"><font="Arial"><b>Sequence</b></font></td><td>
    &nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;20&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;40&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;60&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;80&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;100&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;120<br>
&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;|&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;|&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;|&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;|&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;|&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;|<br>
    MGNRVCCGGSWSCPSTFQKKKKTGSQTRRTLKPQPQQLQQNLPKGHETTGHTYERVLQQQGSQERSPGLMSEDSNLHYADIQVCSRPHAREVKHVHLENATEYATLRFPQATPRYDSKNGTLV</td></tr>
    <tr><td><font="Arial"><span title="Predicted Secondary Structure">
    <b>Prediction</b></span></font></td><td>CCCCCCCCCCCCCC<font color="red">HHHHHHH</font>CCCCCCCCCCCCCC<font color="red">HHHHH</font>CCCCCCCCCC<font color="red">HHHHHHH</font>CCCCCCCCCCCCCCCCCCCC<font color="blue">SSSSSSS</font>CCCC<font color="red">HHHHHHHHH</font>CC<font color="red">HHHH</font><font color="blue">SSSS</font>CCCCCCCCCCCCC<font color="blue">SS</font>C</td></tr>
    <tr><td><font="Arial"><span title="Confidence Score of Prediction 
    (0 being least confident, 9 being most confident)">
    <b>Confidence</b></span></font></td><td>997533588878934676521137653332111312232147776777741488873487754457877543466213445774478744544553222123113445888765327897039</td></tr>
    <tr><td><font="Arial"></td><td><font color=red>H</font>:Helix; 
    <font color=blue>S</font>:Strand; C:Coil</td><tr>
    </table>
</ul></td></tr></table>

<table><tr><td><div style="background:#6599FF;width:500px;"><b><font face="Arial" 
color="#FFFFFF" size=4>&nbsp;Predicted Solvent Accessibility</font></b></div></td>
<td><a id="option3" href="javascript:formAction('hide',3)">
<img id="optionGraphic3" border=0 src="/COFACTOR/img/down_triangle.gif"/>
[Show/Hide]</a></td></tr></table>
<table id="optionForm3"><tr><td><ul>
    <table border="0" style="font-family:Monospace;font-size:14px;background:#F2F2F2;">
    <tr><td valign="bottom"><font="Arial"><b>Sequence</b></font></td><td>
    &nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;20&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;40&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;60&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;80&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;100&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;120<br>
&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;|&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;|&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;|&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;|&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;|&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;|<br>
    MGNRVCCGGSWSCPSTFQKKKKTGSQTRRTLKPQPQQLQQNLPKGHETTGHTYERVLQQQGSQERSPGLMSEDSNLHYADIQVCSRPHAREVKHVHLENATEYATLRFPQATPRYDSKNGTLV</td></tr>
    <tr><td><font="Arial"><span title="Solvent Accessibility Prediction">
    <b>Prediction</b></span></font></td><td>763421133415134315545654554454355545545664464472444315412555446654753455554032220211344464414424165134203040443434243752437</td></tr>
    <tr><td><font="Arial"></td><td>Values range from <b>0</b> (buried residue) 
    to <b>8</b> (highly exposed residue)</td><tr>
    </table>
</ul></td></tr></table>

<table><tr><td><div style="background:#6599FF;width:500px;"><b><font face="Arial" 
color="#FFFFFF" size=4>&nbsp;Top 10 threading templates used by I-TASSER</font></b></div></td>
<td><a id="option4" href="javascript:formAction('hide',4)">
<img id="optionGraphic4" border=0 src="/COFACTOR/img/down_triangle.gif"/>
[Show/Hide]</a></td></tr></table>
<table id="optionForm4"><tr><td><ul>
    <table border="0" style="font-family:Monospace;font-size:14px;background:#F2F2F2;">
    <tr><th align="left" valign="top"><font face="Arial">Rank</font></th> <th align="center" valign="top"><font face="Arial">PDB<br/>hit</font></th><th align="center" valign="top"><font face="Arial">ID1</font></th><th align="center" valign="top"><font face="Arial">ID2</font></th><th valign="top"><font face="Arial">Cov</font></th><th valign="top"><font face="Arial">Norm.<br/>Zscore</font></th><th valign="top"><font face="Arial">Threading<br>program</font></th><td>&nbsp;</td><td BGCOLOR="#DEDEDE" style="font-family:Monospace;font-size:13px;">&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;20&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;40&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;60&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;80&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;100&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;120<br>
&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;|&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;|&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;|&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;|&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;|&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;|</td></tr>
    <tr><td colspan="7"></td><th align="right" valign ="bottom" ><font face="Arial">SS<br/>Seq</font></th><td BGCOLOR="#D7D7D7" style="font-family:Monospace;font-size:13px;">CCCCCCCCCCCCCC<font color="red">HHHHHHH</font>CCCCCCCCCCCCCC<font color="red">HHHHH</font>CCCCCCCCCC<font color="red">HHHHHHH</font>CCCCCCCCCCCCCCCCCCCC<font color="blue">SSSSSSS</font>CCCC<font color="red">HHHHHHHHH</font>CC<font color="red">HHHH</font><font color="blue">SSSS</font>CCCCCCCCCCCCC<font color="blue">SS</font>C<br/>MGNRVCCGGSWSCPSTFQKKKKTGSQTRRTLKPQPQQLQQNLPKGHETTGHTYERVLQQQGSQERSPGLMSEDSNLHYADIQVCSRPHAREVKHVHLENATEYATLRFPQATPRYDSKNGTLV</td></tr>

    <tr><td align="center"><font face="Arial">1</font></td><td><a href="https://www.rcsb.org/pdb/explore/explore.do?structureId=6g4w" target=_blank>6g4wr</a></td><td align="center"><font face="Arial"> 0.12</font></td><td><font face="Arial"> 0.11</font></td><td><font face="Arial"> 3.76</font></td><td align="center"><font face="Arial"> 0.78</font></td><td align="center"><font face="Monospace">SPARKSK</font></td><td>&nbsp;</td><td BGCOLOR="#ECECEG" style="font-family:Monospace;font-size:13px;">TH<font color=#15c015>N</font>LLSSHVR-GVG<font color=#15c015>S</font>RGFPLRLQATEV<font color=#f01505>R</font>ICPVEFNPNFVARMIPKV<font color=#c048c0>E</font>W--SAFLEAADNLRLIQVPK<font color=#f09048>G</font>PVEGYEE---------NEEFL<font color=#f01505>R</font>TMH<font color=#15a4a4>H</font>LL<font color=#80a0f0>L</font><font color=#c048c0>E</font>VEVIEG<font color=#15c015>T</font><font color=#80a0f0>L</font>QC<font color=#c0c000>P</font>ESGRMFPISR<font color=#f09048>G</font>IPN</td></tr>
    <tr><td align="center"><font face="Arial">2</font></td><td><a href="https://www.rcsb.org/pdb/explore/explore.do?structureId=5nrl" target=_blank>5nrlU</a></td><td align="center"><font face="Arial"> 0.10</font></td><td><font face="Arial"> 0.10</font></td><td><font face="Arial"> 3.59</font></td><td align="center"><font face="Arial"> 0.29</font></td><td align="center"><font face="Monospace">FFAS3D</font></td><td>&nbsp;</td><td BGCOLOR="#ECECEG" style="font-family:Monospace;font-size:13px;">S<font color=#f09048>G</font>KL<font color=#80a0f0>V</font><font color=#f08080>C</font><font color=#f08080>C</font>NTMHMSW<font color=#15c015>S</font>SVER-HLG<font color=#f09048>G</font>KKHGLNVLRRGISIEKSSDFRQQQKIIEAKAIQVNYS<font color=#15c015>S</font>EVKENSVD<font color=#15c015>S</font>D<font color=#c048c0>D</font>KAKVPPL<font color=#80a0f0>I</font>RIV<font color=#15c015>S</font>GLELSD----TKQKGKKFLVIAY<font color=#c0c000>P</font>FENIAIELPPNEIL</td></tr>
    <tr><td align="center"><font face="Arial">3</font></td><td><a href="https://www.rcsb.org/pdb/explore/explore.do?structureId=4fbd" target=_blank>4fbdA2</a></td><td align="center"><font face="Arial"> 0.12</font></td><td><font face="Arial"> 0.07</font></td><td><font face="Arial"> 2.28</font></td><td align="center"><font face="Arial"> 0.25</font></td><td align="center"><font face="Monospace">HHpred</font></td><td>&nbsp;</td><td BGCOLOR="#ECECEG" style="font-family:Monospace;font-size:13px;">----------------------------------------AGIEFFQGYAPFCRHLYPNFVGALP--------MPAAFL<font color=#c048c0>D</font>LILY<font color=#15c015>S</font><font color=#f01505>R</font>EQ<font color=#80a0f0>A</font>K<font color=#c048c0>E</font>TAAESNT<font color=#15c015>N</font><font color=#80a0f0>A</font>PAWSIIAVA<font color=#15c015>Q</font>NEK----------</td></tr>
    <tr><td align="center"><font face="Arial">4</font></td><td><a href="https://www.rcsb.org/pdb/explore/explore.do?structureId=4hc7" target=_blank>4hc7A</a></td><td align="center"><font face="Arial"> 0.18</font></td><td><font face="Arial"> 0.15</font></td><td><font face="Arial"> 4.77</font></td><td align="center"><font face="Arial"> 0.69</font></td><td align="center"><font face="Monospace">MUSTER</font></td><td>&nbsp;</td><td BGCOLOR="#ECECEG" style="font-family:Monospace;font-size:13px;"><font color=#80a0f0>M</font><font color=#f09048>G</font>-<font color=#f01505>R</font>E<font color=#f08080>C</font><font color=#f08080>C</font><font color=#f09048>G</font>AT-<font color=#15c015>S</font>T<font color=#c0c000>P</font>--LWRRDG<font color=#15c015>T</font><font color=#f09048>G</font>HYLCNACGLYHKMNG<font color=#15c015>Q</font><font color=#15c015>N</font>R<font color=#c0c000>P</font>LIKPKRRLS-----AARRAGTSCANCQTTTTT<font color=#80a0f0>L</font>WRRN-GDPVCNACGLYYKL<font color=#15a4a4>H</font>NI<font color=#15c015>N</font>R--------<font color=#c0c000>P</font>LTMKKEGIQ<font color=#15c015>N</font>----</td></tr>
    <tr><td align="center"><font face="Arial">5</font></td><td><a href="https://www.rcsb.org/pdb/explore/explore.do?structureId=1cjy" target=_blank>1cjyA</a></td><td align="center"><font face="Arial"> 0.12</font></td><td><font face="Arial"> 0.03</font></td><td><font face="Arial"> 1.15</font></td><td align="center"><font face="Arial"> 0.52</font></td><td align="center"><font face="Monospace">CNFpred</font></td><td>&nbsp;</td><td BGCOLOR="#ECECEG" style="font-family:Monospace;font-size:13px;">----------------------------------------------------------------------------<font color=#15a4a4>H</font>KFTVV<font color=#80a0f0>V</font>LRATKVTKGAFGDMDTPDP<font color=#15a4a4>Y</font>VE<font color=#80a0f0>L</font>FIST-------------</td></tr>
    <tr><td align="center"><font face="Arial">6</font></td><td><a href="https://www.rcsb.org/pdb/explore/explore.do?structureId=2pff" target=_blank>2pffB</a></td><td align="center"><font face="Arial"> 0.22</font></td><td><font face="Arial"> 0.20</font></td><td><font face="Arial"> 6.41</font></td><td align="center"><font face="Arial"> 0.57</font></td><td align="center"><font face="Monospace">HHsearch2</font></td><td>&nbsp;</td><td BGCOLOR="#ECECEG" style="font-family:Monospace;font-size:13px;">AK<font color=#15c015>N</font>L<font color=#80a0f0>V</font>V-----<font color=#15c015>S</font>G<font color=#c0c000>P</font>PLTLR<font color=#f01505>K</font>A<font color=#f01505>K</font>AP<font color=#15c015>S</font>GS<font color=#f01505>R</font>IPFSE-RK---LKFSNRFLPVASPFHSH<font color=#80a0f0>L</font>LVPA<font color=#15c015>S</font>DLINKD<font color=#80a0f0>L</font>VKNNVSFNAK<font color=#c048c0>D</font><font color=#80a0f0>I</font><font color=#15c015>Q</font>IL<font color=#15c015>S</font>GSIE<font color=#f01505>R</font>I<font color=#80a0f0>V</font>DCLPW<font color=#c048c0>E</font>TT<font color=#15c015>T</font>QFHI<font color=#80a0f0>L</font>D<font color=#80a0f0>F</font>GPGVLTHRN<font color=#f01505>K</font>D<font color=#f09048>G</font><font color=#15c015>T</font>G<font color=#80a0f0>V</font></td></tr>
    <tr><td align="center"><font face="Arial">7</font></td><td><a href="https://www.rcsb.org/pdb/explore/explore.do?structureId=6n7p" target=_blank>6n7pJ</a></td><td align="center"><font face="Arial"> 0.08</font></td><td><font face="Arial"> 0.07</font></td><td><font face="Arial"> 2.93</font></td><td align="center"><font face="Arial"> <b>1.06</b></font></td><td align="center"><font face="Monospace">NeffPPAS</font></td><td>&nbsp;</td><td BGCOLOR="#ECECEG" style="font-family:Monospace;font-size:13px;">------GSSCLDLFLD<font color=#80a0f0>F</font>VDEQRMYIFAQ<font color=#f01505>R</font>SIAQ<font color=#15c015>Q</font>TLID<font color=#15c015>Q</font>NFEWNDADSDEI<font color=#15c015>T</font>KQNIEKVLENDRKFDKVDK<font color=#c048c0>E</font><font color=#c048c0>D</font>IS<font color=#80a0f0>L</font>IVDGLIKQRNEKIQQKLQNERRILEQKKHYFWLLLQRT<font color=#15a4a4>Y</font>TKTGKPKP</td></tr>
    <tr><td align="center"><font face="Arial">8</font></td><td><a href="https://www.rcsb.org/pdb/explore/explore.do?structureId=2dco" target=_blank>2dcoA</a></td><td align="center"><font face="Arial"> 0.44</font></td><td><font face="Arial"> 0.07</font></td><td><font face="Arial"> 1.90</font></td><td align="center"><font face="Arial"> 0.34</font></td><td align="center"><font face="Monospace">HHsearch</font></td><td>&nbsp;</td><td BGCOLOR="#ECECEG" style="font-family:Monospace;font-size:13px;">S<font color=#f09048>G</font>A<font color=#f01505>R</font>L-APAQ<font color=#80a0f0>W</font><font color=#15c015>S</font><font color=#f08080>C</font>RE<font color=#15c015>T</font><font color=#80a0f0>F</font><font color=#15c015>Q</font>A--------------------------------------------------------------------------------------------------------</td></tr>
    <tr><td align="center"><font face="Arial">9</font></td><td><a href="https://www.rcsb.org/pdb/explore/explore.do?structureId=5o8l" target=_blank>5o8la</a></td><td align="center"><font face="Arial"> 0.11</font></td><td><font face="Arial"> 0.11</font></td><td><font face="Arial"> 3.85</font></td><td align="center"><font face="Arial"> 0.46</font></td><td align="center"><font face="Monospace">SP3</font></td><td>&nbsp;</td><td BGCOLOR="#ECECEG" style="font-family:Monospace;font-size:13px;">ANQHIHILQD<font color=#80a0f0>W</font><font color=#15c015>S</font>PNDVWYNQQVN<font color=#f09048>G</font>NSQLTMDATMQN<font color=#15c015>Q</font><font color=#80a0f0>L</font>LAS<font color=#80a0f0>L</font>TRPITSRDS-MKSFTKDALLVH<font color=#f01505>R</font>TADNSYNQAVPN<font color=#15a4a4>Y</font>SF<font color=#80a0f0>I</font>RAH<font color=#15c015>S</font>EVQTIIA<font color=#f01505>K</font>IISDKHPDL<font color=#15a4a4>Y</font>P<font color=#15c015>T</font>VDKALLAKDSALYDEAFT</td></tr>
    <tr><td align="center"><font face="Arial">10</font></td><td><a href="https://www.rcsb.org/pdb/explore/explore.do?structureId=5tx1" target=_blank>5tx1R</a></td><td align="center"><font face="Arial"> 0.05</font></td><td><font face="Arial"> 0.05</font></td><td><font face="Arial"> 2.32</font></td><td align="center"><font face="Arial"> 0.51</font></td><td align="center"><font face="Monospace">PROSPECTOR2</font></td><td>&nbsp;</td><td BGCOLOR="#ECECEG" style="font-family:Monospace;font-size:13px;"><font color=#80a0f0>M</font>NGTGGAFEGGLFSPYLTTRLPGWAGV<font color=#f01505>R</font>QNVMGSLPANSSTMTYATVGNSSLDSTAAAAAAAAAMTATRLASSYMPSSGSSPSVPSSIIA<font color=#c048c0>E</font>E<font color=#f01505>K</font>LLA<font color=#80a0f0>L</font>LAEL<font color=#c048c0>E</font>ALSRQLAALSELREQQQQQNK</td></tr>


    <tr><td colspan="9" align="left">
        <table cellspacing="2" border="0" width="1000px" style="font-family:Arial;font-size:12px;">
        <tr><td valign="top">(a)</td><td align="justify"><i>ID1</i> is the number of template residues identical to query divided by number of aligned residues.</td></tr>
        <tr><td valign="top">(b)</td><td align="justify"><i>ID2</i> is the number of template residues identical to query divided by query sequence length.</td></tr>
        <tr><td valign="top">(c)</td><td align="justify"><i>Cov</i> is equal the number of aligned template residues divided by query sequence length.</td></tr>
        <tr><td valign="top">(d)</td><td align="justify"><i>Norm. Zscore</i> is the normalized Z-score of the threading alignments. A Normalized Z-score &gt;1 means a good alignment and is highlighted in <b>bold</b>.</td></tr>
        <tr><td valign="top">(e)</td><td align="justify"><i>Download alignment</i> lists the threading program</a> used to identify the template, and provide the 3D structure of aligned regions of threading templates (threading[1-10].pdb.gz).</td></tr>
        <tr><td valign="top">(f)</td><td align="justify">Template residues identical to query sequence are highlighted in <a href=/I-TASSER/color.html target=_blank>color</a>.</td></tr>
        </table>
    </table>
</ul></td></tr></table>


<table><tr><td><div style="background:#6599FF;width:500px;"><b><font face="Arial" 
color="#FFFFFF" size=4>&nbsp;Top 5 final models from I-TASSER</font></b></div></td>
<td><a id="option5" href="javascript:formAction('hide',5)">
<img id="optionGraphic5" border=0 src="/COFACTOR/img/down_triangle.gif"/>
[Show/Hide]</a></td></tr></table>
<table id="optionForm5"><tr><td><ul>
    <table border="0" style="font-size:14px;background:#F2F2F2;">

    <td align ="left">
    <script type="text/javascript">
    jmolInitialize("/jmol/");
    jmolSetAppletColor("#000000");
    jmolApplet(500,"load model1.pdb.gz; set frank off; calculate structure rama;  select all; color group; spacefill off; wireframe off; cartoons;set ambient 40;spin off;");
    jmolButton("Reset","Reset");
    jmolCheckbox("set spin x 10; set spin y 10; spin on",
        "spin off", "Spin", false);
    jmolCheckbox("set antialiasDisplay true",
        "set antialiasDisplay false", "High quality", false);
    jmolCheckbox("color background white",
        "color background black", "White background", false);
    jmolButton("write PNGJ model.png","Save image");
    </script>
    </td>

    <td valign="top">
    <table border="0" style="font-family:Arial;font-size:13px;">
    <tr bgcolor="#DEDEDE" align="center"><td>Click<br>to view</td>
    <td>Rank<sup>a</sup></td><td>Download</td><td>C-score<sup>b</sup></td>
    <td>Estimated TM-score<sup>c</sup></td><td>Estimated RMSD<sup>c</sup></td></tr>

    <tr align="center"><td><script type="text/javascript">
    jmolRadio("load model1.pdb.gz; set frank off; calculate structure rama;  select all; color group; spacefill off; wireframe off; cartoons;set ambient 40;","",true);
    </script></td>
    <td>1</td>
    <td><a href="model1.pdb.gz" download>model1.pdb.gz</a></td>
    <td>-2.88</td>
    <td>0.39&plusmn;0.13</td>
    <td>10.8&plusmn;4.6&Aring;</td></tr>

    
    <tr align="center"><td><script type="text/javascript">
    jmolRadio("load model2.pdb.gz; set frank off; calculate structure rama;  select all; color group; spacefill off; wireframe off; cartoons;set ambient 40;","",false);
    </script></td>
    <td>2</td>
    <td><a href="model2.pdb.gz" download>model2.pdb.gz</a></td>
    <td>-3.24</td></tr>

    <tr align="center"><td><script type="text/javascript">
    jmolRadio("load model3.pdb.gz; set frank off; calculate structure rama;  select all; color group; spacefill off; wireframe off; cartoons;set ambient 40;","",false);
    </script></td>
    <td>3</td>
    <td><a href="model3.pdb.gz" download>model3.pdb.gz</a></td>
    <td>-3.73</td></tr>

    <tr align="center"><td><script type="text/javascript">
    jmolRadio("load model4.pdb.gz; set frank off; calculate structure rama;  select all; color group; spacefill off; wireframe off; cartoons;set ambient 40;","",false);
    </script></td>
    <td>4</td>
    <td><a href="model4.pdb.gz" download>model4.pdb.gz</a></td>
    <td>-4.58</td></tr>

    <tr align="center"><td><script type="text/javascript">
    jmolRadio("load model5.pdb.gz; set frank off; calculate structure rama;  select all; color group; spacefill off; wireframe off; cartoons;set ambient 40;","",false);
    </script></td>
    <td>5</td>
    <td><a href="model5.pdb.gz" download>model5.pdb.gz</a></td>
    <td>-5.06</td></tr>

    </table>
    <table cellspacing="2" border="0" width="600px" style="font-family:Arial;font-size:12px;"><tr></tr><tr></tr>
    <tr><td valign="top">(a)</td><td>
    I-TASSER simulations generate a large ensemble of structural
    conformations, i.e. decoys. These decoys are clustered by
    <a href=/SPICKER target=_blank>SPICKER</a> based on pairwise structure similarity to
    report up to five final models from the five largest clusters. Models are 
    ranked in descending order of cluster size. If the simulations converge
    well, it is possible to have less than 5 models generated, which is
    usually an indication of good model quality.
    </td></tr>
    <tr><td valign="top">(b)</td><td>
    The model confidence is quantitatified by C-score, calculated based on
    significance of threading template alignments and convergence of
    I-TASSER simulations. C-score is typically in the range of [-5, 2],
    with higher C-score signifies higher model confidence.</td></tr>
    <tr><td valign="top">(c)</td><td>
    Model <a href=/TM-score target=_blank>TM-score</a> and RMSD are estimated based on
    C-score and protein length for the first model.
    </td></tr>
    </table>
    </td>
    </table>
</ul></td></tr></table>


<table><tr><td><div style="background:#6599FF;width:500px;"><b><font face="Arial" 
color="#FFFFFF" size=4>&nbsp;Proteins with similar structure</font></b></div></td>
<td><a id="option6" href="javascript:formAction('hide',6)">
<img id="optionGraphic6" border=0 src="/COFACTOR/img/down_triangle.gif"/>
[Show/Hide]</a></td></tr></table>
<table id="optionForm6"><tr><td><ul>
    <table border="0" style="background:#F2F2F2";align="left">
        <tr><td align ="center"><script type="text/javascript">
        jmolInitialize("/jmol/");
        jmolSetAppletColor("#000000");
        jmolApplet(500, "load model1_5wlcLJ.pdb.gz;set frank off;select all;spacefill off;wireframe off;select :T;trace 0.35;color trace mediumpurple;select :Q;cartoons;color group;set ambient 40;");
        jmolButton("Reset","Reset");
        jmolCheckbox("set spin x 10; set spin y 10; spin on",
            "spin off", "Spin", false);
        jmolCheckbox("set antialiasDisplay true",
            "set antialiasDisplay false", "High quality", false);
        jmolCheckbox("color background white",
            "color background black", "White background", false);
        jmolButton("write PNGJ model.png","Save image");
        </script></td>
        <td valign="top"><b>Top 10 structural analogs in PDB (as identified by
        <a href="https://zhanglab.ccmb.med.umich.edu/TM-align" target="_blank">
        TM-align</a>)</b><br/><br/>
        <table border="0" style="font-family:Arial;font-size:13px;">
        <tr><th>Click<br>to view</th><th>Rank</th><th>PDB<br/>Hit</th><th>TM-score</th><th>RMSD<sup>a</sup></th><th>IDEN<sup>a</sup></th><th>Cov.</th><th>Download<br/>Alignment</th></tr>
        
        <tr><td align="center"><script type="text/javascript">jmolRadio("load model1_5wlcLJ.pdb.gz; set frank off; select all; spacefill off; wireframe off; select :T; trace 0.35; color trace mediumpurple; select :Q; cartoons; color group; set ambient 40; set spin x 5; set spin y 5; spin off;", "", false);</script></td><td valign="middle" align="center">1</td><td valign="middle"><a href="https://www.rcsb.org/pdb/explore/explore.do?structureId=5wlc" target="_blank"><font face="Monospace">5wlcLJ</font></a></td><td align="center">0.46</td><td valign="middle" align="center">5.00</td><td valign="middle" align="center">0.065</td><td valign="middle">0.878</td><td><a href="model1_5wlcLJ.pdb.gz" download>model1_5wlcLJ.pdb.gz</a></td></tr>
        <tr><td align="center"><script type="text/javascript">jmolRadio("load model1_4b9yA1.pdb.gz; set frank off; select all; spacefill off; wireframe off; select :T; trace 0.35; color trace mediumpurple; select :Q; cartoons; color group; set ambient 40; set spin x 5; set spin y 5; spin off;", "", false);</script></td><td valign="middle" align="center">2</td><td valign="middle"><a href="https://www.rcsb.org/pdb/explore/explore.do?structureId=4b9y" target="_blank"><font face="Monospace">4b9yA1</font></a></td><td align="center">0.46</td><td valign="middle" align="center">4.93</td><td valign="middle" align="center">0.053</td><td valign="middle">0.837</td><td><a href="model1_4b9yA1.pdb.gz" download>model1_4b9yA1.pdb.gz</a></td></tr>
        <tr><td align="center"><script type="text/javascript">jmolRadio("load model1_2pffA1.pdb.gz; set frank off; select all; spacefill off; wireframe off; select :T; trace 0.35; color trace mediumpurple; select :Q; cartoons; color group; set ambient 40; set spin x 5; set spin y 5; spin off;", "", false);</script></td><td valign="middle" align="center">3</td><td valign="middle"><a href="https://www.rcsb.org/pdb/explore/explore.do?structureId=2pff" target="_blank"><font face="Monospace">2pffA1</font></a></td><td align="center">0.46</td><td valign="middle" align="center">4.95</td><td valign="middle" align="center">0.091</td><td valign="middle">0.829</td><td><a href="model1_2pffA1.pdb.gz" download>model1_2pffA1.pdb.gz</a></td></tr>
        <tr><td align="center"><script type="text/javascript">jmolRadio("load model1_5dkxA1.pdb.gz; set frank off; select all; spacefill off; wireframe off; select :T; trace 0.35; color trace mediumpurple; select :Q; cartoons; color group; set ambient 40; set spin x 5; set spin y 5; spin off;", "", false);</script></td><td valign="middle" align="center">4</td><td valign="middle"><a href="https://www.rcsb.org/pdb/explore/explore.do?structureId=5dkx" target="_blank"><font face="Monospace">5dkxA1</font></a></td><td align="center">0.45</td><td valign="middle" align="center">4.62</td><td valign="middle" align="center">0.027</td><td valign="middle">0.805</td><td><a href="model1_5dkxA1.pdb.gz" download>model1_5dkxA1.pdb.gz</a></td></tr>
        <tr><td align="center"><script type="text/javascript">jmolRadio("load model1_2xvgA.pdb.gz; set frank off; select all; spacefill off; wireframe off; select :T; trace 0.35; color trace mediumpurple; select :Q; cartoons; color group; set ambient 40; set spin x 5; set spin y 5; spin off;", "", false);</script></td><td valign="middle" align="center">5</td><td valign="middle"><a href="https://www.rcsb.org/pdb/explore/explore.do?structureId=2xvg" target="_blank"><font face="Monospace">2xvgA</font></a></td><td align="center">0.45</td><td valign="middle" align="center">5.15</td><td valign="middle" align="center">0.052</td><td valign="middle">0.878</td><td><a href="model1_2xvgA.pdb.gz" download>model1_2xvgA.pdb.gz</a></td></tr>
        <tr><td align="center"><script type="text/javascript">jmolRadio("load model1_1pr9A.pdb.gz; set frank off; select all; spacefill off; wireframe off; select :T; trace 0.35; color trace mediumpurple; select :Q; cartoons; color group; set ambient 40; set spin x 5; set spin y 5; spin off;", "", false);</script></td><td valign="middle" align="center">6</td><td valign="middle"><a href="https://www.rcsb.org/pdb/explore/explore.do?structureId=1pr9" target="_blank"><font face="Monospace">1pr9A</font></a></td><td align="center">0.43</td><td valign="middle" align="center">4.34</td><td valign="middle" align="center">0.071</td><td valign="middle">0.724</td><td><a href="model1_1pr9A.pdb.gz" download>model1_1pr9A.pdb.gz</a></td></tr>
        <tr><td align="center"><script type="text/javascript">jmolRadio("load model1_2et6A1.pdb.gz; set frank off; select all; spacefill off; wireframe off; select :T; trace 0.35; color trace mediumpurple; select :Q; cartoons; color group; set ambient 40; set spin x 5; set spin y 5; spin off;", "", false);</script></td><td valign="middle" align="center">7</td><td valign="middle"><a href="https://www.rcsb.org/pdb/explore/explore.do?structureId=2et6" target="_blank"><font face="Monospace">2et6A1</font></a></td><td align="center">0.43</td><td valign="middle" align="center">4.31</td><td valign="middle" align="center">0.075</td><td valign="middle">0.724</td><td><a href="model1_2et6A1.pdb.gz" download>model1_2et6A1.pdb.gz</a></td></tr>
        <tr><td align="center"><script type="text/javascript">jmolRadio("load model1_1we5A1.pdb.gz; set frank off; select all; spacefill off; wireframe off; select :T; trace 0.35; color trace mediumpurple; select :Q; cartoons; color group; set ambient 40; set spin x 5; set spin y 5; spin off;", "", false);</script></td><td valign="middle" align="center">8</td><td valign="middle"><a href="https://www.rcsb.org/pdb/explore/explore.do?structureId=1we5" target="_blank"><font face="Monospace">1we5A1</font></a></td><td align="center">0.42</td><td valign="middle" align="center">4.96</td><td valign="middle" align="center">0.075</td><td valign="middle">0.780</td><td><a href="model1_1we5A1.pdb.gz" download>model1_1we5A1.pdb.gz</a></td></tr>
        <tr><td align="center"><script type="text/javascript">jmolRadio("load model1_1vl8B.pdb.gz; set frank off; select all; spacefill off; wireframe off; select :T; trace 0.35; color trace mediumpurple; select :Q; cartoons; color group; set ambient 40; set spin x 5; set spin y 5; spin off;", "", false);</script></td><td valign="middle" align="center">9</td><td valign="middle"><a href="https://www.rcsb.org/pdb/explore/explore.do?structureId=1vl8" target="_blank"><font face="Monospace">1vl8B</font></a></td><td align="center">0.42</td><td valign="middle" align="center">4.39</td><td valign="middle" align="center">0.054</td><td valign="middle">0.732</td><td><a href="model1_1vl8B.pdb.gz" download>model1_1vl8B.pdb.gz</a></td></tr>
        <tr><td align="center"><script type="text/javascript">jmolRadio("load model1_1xg5C.pdb.gz; set frank off; select all; spacefill off; wireframe off; select :T; trace 0.35; color trace mediumpurple; select :Q; cartoons; color group; set ambient 40; set spin x 5; set spin y 5; spin off;", "", false);</script></td><td valign="middle" align="center">10</td><td valign="middle"><a href="https://www.rcsb.org/pdb/explore/explore.do?structureId=1xg5" target="_blank"><font face="Monospace">1xg5C</font></a></td><td align="center">0.42</td><td valign="middle" align="center">4.25</td><td valign="middle" align="center">0.039</td><td valign="middle">0.691</td><td><a href="model1_1xg5C.pdb.gz" download>model1_1xg5C.pdb.gz</a></td></tr>
        </table><br/>

        <table border="0" cellspacing="2" style="font-family:Arial;font-size:12px;background:#F2F2F2;">
            <tr><td valign="top">(a)</td><td align="justify">Query structure is shown in cartoon, while the structural analog is displayed using backbone trace.</td></tr>
            <tr><td valign="top">(b)</td><td align="justify">Ranking of proteins is based on TM-score of the structural alignment between the query structure and known structures in the PDB library.</td></tr>
            <tr><td valign="top">(c)</td><td align="justify">RMSD<sup>a</sup> is the RMSD between residues that are structurally aligned by TM-align.</td></tr>
            <tr><td valign="top">(d)</td><td align="justify">IDEN<sup>a</sup> is the percentage sequence identity in the structurally aligned region.</td></tr>
            <tr><td valign="top">(e)</td><td align="justify">Cov. represents the coverage of the alignment by TM-align and is equal to the number of structurally aligned residues divided by length of the query protein.</td></tr>
        </table>
    </td></tr>
    </table>
</ul></td></tr></table>

<table><tr><td><div style="background:#6599FF;width:500px;"><b><font face="Arial" 
color="#FFFFFF" size=4>&nbsp;Predicted Gene Ontology (GO) Terms</font></b></div></td>
<td><a id="option7" href="javascript:formAction('hide',7)">
<img id="optionGraphic7" border=0 src="/COFACTOR/img/down_triangle.gif"/>
[Show/Hide]</a></td></tr></table>
<table id="optionForm7"><tr><td><ul>
<table border="0" style="background:#F2F2F2";align="left"><tr><td>

<table border="0" style="background:#F2F2F2";align="left"><tr>
<td align ="center" style="background:#FFFFFF">
    <a href="GOsearchresult_final_MF.svg" target="_blank">
    <img width=500 height=500 border="1" style="border-color:black" alt="GOsearchresult_final_MF.svg" src="GOsearchresult_final_MF.svg">
    </a>
</td>
<td valign="top" align="left">
    <table border="0">
        <tr><td><b>Molecular Function (MF)</b><br></tr></td>
    </table>
    <table border="0" style="font-family:Arial;font-size:13px;">
        <tr><td align="left"><b>GO term</b></td><td align="center"><b>Cscore<sup>GO</sup></b></td><td><b>Name</b></td></tr>
        
        <tr><td align="left"><a href="http://amigo.geneontology.org/amigo/term/GO:0016798" target="_blank">GO:0016798</a></td><td align="center">0.57</td><td>hydrolase activity, acting on glycosyl bonds</td></tr>
    </table><br>
    <table border="0" style="font-family:Arial;font-size:13px;">
        <tr><td>Download <a href="GOsearchresult_final_MF.csv" download>full result</a> of the above consensus prediction.</tr></td>
    </table><br>
    <table cellspacing="2" border="0" style="font-family:Arial;font-size:12px;">
        <tr><td valign="top"></td><td align="justify"><b>Click the graph to show a high resolution version.</b></td></tr>
        <tr><td valign="top">(a)</td><td align="justify">Cscore<sup>GO</sup> is the confidence score of predicted GO terms. Cscore<sup>GO</sup> values range in between [0-1]; where a higher value indicates a better confidence in predicting the function using the template.</td></tr>
        <tr><td valign="top">(b)</td><td align="justify">The graph shows the predicted terms within the Gene Ontology hierachy for Molecular Function. Confidently predicted terms are color coded by Cscore<sup>GO</sup>:</td></tr>
        <tr><td valign="top"></td><td align="justify"><table cellspacing="2" border="0" style="border-color:black;font-family:Arial;font-size:12px;"><tr><td style="background-color:magenta;">[0.4,0.5)</td><td style="background-color:cyan;">[0.5,0.6)</td><td style="background-color:green;">[0.6,0.7)</td><td style="background-color:yellow;">[0.7,0.8)</td><td style="background-color:orange;">[0.8,0.9)</td><td style="background-color:red;">[0.9,1.0]</td></tr></table></td></tr>
     </table>
</td>
</tr>
</table>


<table border="0" style="background:#F2F2F2";align="left"><tr>
<td align ="center" style="background:#FFFFFF">
    <a href="GOsearchresult_final_BP.svg" target="_blank">
    <img width=500 height=500 border="1" style="border-color:black" alt="GOsearchresult_final_BP.svg" src="GOsearchresult_final_BP.svg">
    </a>
</td>
<td valign="top" align="left">
    <table border="0">
        <tr><td><b>Biological Process (BP)</b><br></tr></td>
    </table>
    <table border="0" style="font-family:Arial;font-size:13px;">
        <tr><td align="left"><b>GO term</b></td><td align="center"><b>Cscore<sup>GO</sup></b></td><td><b>Name</b></td></tr>
        
        <tr><td align="left"><a href="http://amigo.geneontology.org/amigo/term/GO:1904950" target="_blank">GO:1904950</a></td><td align="center">1.00</td><td>negative regulation of establishment of protein localization</td></tr>
        <tr><td align="left"><a href="http://amigo.geneontology.org/amigo/term/GO:1903729" target="_blank">GO:1903729</a></td><td align="center">1.00</td><td>regulation of plasma membrane organization</td></tr>
        <tr><td align="left"><a href="http://amigo.geneontology.org/amigo/term/GO:1903077" target="_blank">GO:1903077</a></td><td align="center">1.00</td><td>negative regulation of protein localization to plasma membrane</td></tr>
        <tr><td align="left"><a href="http://amigo.geneontology.org/amigo/term/GO:0060070" target="_blank">GO:0060070</a></td><td align="center">1.00</td><td>canonical Wnt signaling pathway</td></tr>
        <tr><td align="left"><a href="http://amigo.geneontology.org/amigo/term/GO:0051129" target="_blank">GO:0051129</a></td><td align="center">1.00</td><td>negative regulation of cellular component organization</td></tr>
    </table><br>
    <table border="0" style="font-family:Arial;font-size:13px;">
        <tr><td>Download <a href="GOsearchresult_final_BP.csv" download>full result</a> of the above consensus prediction.</tr></td>
    </table><br>
    <table cellspacing="2" border="0" style="font-family:Arial;font-size:12px;">
        <tr><td valign="top"></td><td align="justify"><b>Click the graph to show a high resolution version.</b></td></tr>
        <tr><td valign="top">(a)</td><td align="justify">Cscore<sup>GO</sup> is the confidence score of predicted GO terms. Cscore<sup>GO</sup> values range in between [0-1]; where a higher value indicates a better confidence in predicting the function using the template.</td></tr>
        <tr><td valign="top">(b)</td><td align="justify">The graph shows the predicted terms within the Gene Ontology hierachy for Biological Process. Confidently predicted terms are color coded by Cscore<sup>GO</sup>:</td></tr>
        <tr><td valign="top"></td><td align="justify"><table cellspacing="2" border="0" style="border-color:black;font-family:Arial;font-size:12px;"><tr><td style="background-color:magenta;">[0.4,0.5)</td><td style="background-color:cyan;">[0.5,0.6)</td><td style="background-color:green;">[0.6,0.7)</td><td style="background-color:yellow;">[0.7,0.8)</td><td style="background-color:orange;">[0.8,0.9)</td><td style="background-color:red;">[0.9,1.0]</td></tr></table></td></tr>
     </table>
</td>
</tr>
</table>


<table border="0" style="background:#F2F2F2";align="left"><tr>
<td align ="center" style="background:#FFFFFF">
    <a href="GOsearchresult_final_CC.svg" target="_blank">
    <img width=500 height=500 border="1" style="border-color:black" alt="GOsearchresult_final_CC.svg" src="GOsearchresult_final_CC.svg">
    </a>
</td>
<td valign="top" align="left">
    <table border="0">
        <tr><td><b>Cellular Component (CC)</b><br></tr></td>
    </table>
    <table border="0" style="font-family:Arial;font-size:13px;">
        <tr><td align="left"><b>GO term</b></td><td align="center"><b>Cscore<sup>GO</sup></b></td><td><b>Name</b></td></tr>
        
        <tr><td align="left"><a href="http://amigo.geneontology.org/amigo/term/GO:0005925" target="_blank">GO:0005925</a></td><td align="center">1.00</td><td>focal adhesion</td></tr>
        <tr><td align="left"><a href="http://amigo.geneontology.org/amigo/term/GO:0070062" target="_blank">GO:0070062</a></td><td align="center">0.78</td><td>extracellular exosome</td></tr>
    </table><br>
    <table border="0" style="font-family:Arial;font-size:13px;">
        <tr><td>Download <a href="GOsearchresult_final_CC.csv" download>full result</a> of the above consensus prediction.</tr></td>
    </table><br>
    <table cellspacing="2" border="0" style="font-family:Arial;font-size:12px;">
        <tr><td valign="top"></td><td align="justify"><b>Click the graph to show a high resolution version.</b></td></tr>
        <tr><td valign="top">(a)</td><td align="justify">Cscore<sup>GO</sup> is the confidence score of predicted GO terms. Cscore<sup>GO</sup> values range in between [0-1]; where a higher value indicates a better confidence in predicting the function using the template.</td></tr>
        <tr><td valign="top">(b)</td><td align="justify">The graph shows the predicted terms within the Gene Ontology hierachy for Cellular Component. Confidently predicted terms are color coded by Cscore<sup>GO</sup>:</td></tr>
        <tr><td valign="top"></td><td align="justify"><table cellspacing="2" border="0" style="border-color:black;font-family:Arial;font-size:12px;"><tr><td style="background-color:magenta;">[0.4,0.5)</td><td style="background-color:cyan;">[0.5,0.6)</td><td style="background-color:green;">[0.6,0.7)</td><td style="background-color:yellow;">[0.7,0.8)</td><td style="background-color:orange;">[0.8,0.9)</td><td style="background-color:red;">[0.9,1.0]</td></tr></table></td></tr>
     </table>
</td>
</tr>
</table>

</td></tr>
</table>
</ul></td></tr></table>

<table><tr><td><div style="background:#6599FF;width:500px;"><b><font face="Arial" 
color="#FFFFFF" size=4>&nbsp;Predicted Enzyme Commission (EC) Numbers</font></b></div></td>
<td><a id="option8" href="javascript:formAction('hide',8)">
<img id="optionGraphic8" border=0 src="/COFACTOR/img/down_triangle.gif"/>
[Show/Hide]</a></td></tr></table>
<table id="optionForm8"><tr><td><ul>
    <table border="0" style="background:#F2F2F2";align="left">
        <tr><td align ="center"><script type="text/javascript">
        jmolInitialize("/jmol/");
        jmolSetAppletColor("#000000");
        jmolApplet(500, "load model1.pdb.gz;set frank off;select all;color group;spacefill off;wireframe off;cartoons;;set spin x 10; set spin y 10; spin off;");
        jmolButton("Reset","Reset");
        jmolCheckbox("set spin x 10; set spin y 10; spin on",
            "spin off", "Spin", false);
        jmolCheckbox("set antialiasDisplay true",
            "set antialiasDisplay false", "High quality", false);
        jmolCheckbox("color background white",
            "color background black", "White background", false);
        jmolButton("write PNGJ model.png","Save image");
        </script></td>
        <td valign="top"><b>Top 5 enzyme homologs in PDB</b><br/><br/>
        <table border="0" style="font-family:Arial;font-size:13px;">
        <tr><th>Click<br>to view</th><th>Rank</th><th>Cscore<sup>EC</sup></th><th>PDB<br/>Hit</th><th>TM-score</th><th>RMSD<sup>a</sup></th><th>IDEN<sup>a</sup></th><th>Cov.</th><th>EC Number</th><th>Predicted Active Site Residues</th></tr>
        
        <tr><td align="center"><script type="text/javascript">jmolRadio("select all;wireframe off;spacefill off;label off;;set spin x 5;set spin y 5;spin off;","",false);</script></td><td align="left">1</td><td align="center">0.060</td><td><a href="https://www.rcsb.org/pdb/explore/explore.do?structureId=1cyd" target="_blank"><font face="Monospace">1cydA</font></a></td><td align="center">0.427</td><td align="center">4.29</td><td align="center">0.058</td><td align="right">0.707</td><td align="left"><span title="Carbonyl reductase (NADPH)."><a href="http://enzyme.expasy.org/EC/1.1.1.184" target="_blank">1.1.1.184</a></span>&nbsp;</td><td align="left">NA</td></tr>
        <tr><td align="center"><script type="text/javascript">jmolRadio("select all;wireframe off;spacefill off;label off;;set spin x 5;set spin y 5;spin off;","",false);</script></td><td align="left">2</td><td align="center">0.060</td><td><a href="https://www.rcsb.org/pdb/explore/explore.do?structureId=1gee" target="_blank"><font face="Monospace">1geeF</font></a></td><td align="center">0.429</td><td align="center">4.13</td><td align="center">0.027</td><td align="right">0.691</td><td align="left"><span title="Glucose 1-dehydrogenase."><a href="http://enzyme.expasy.org/EC/1.1.1.47" target="_blank">1.1.1.47</a></span>&nbsp;</td><td align="left">NA</td></tr>
        <tr><td align="center"><script type="text/javascript">jmolRadio("select all;wireframe off;spacefill off;label off;;set spin x 5;set spin y 5;spin off;","",false);</script></td><td align="left">3</td><td align="center">0.060</td><td><a href="https://www.rcsb.org/pdb/explore/explore.do?structureId=1war" target="_blank"><font face="Monospace">1warA</font></a></td><td align="center">0.434</td><td align="center">4.90</td><td align="center">0.043</td><td align="right">0.846</td><td align="left"><span title="Acid phosphatase."><a href="http://enzyme.expasy.org/EC/3.1.3.2" target="_blank">3.1.3.2</a></span>&nbsp;</td><td align="left">NA</td></tr>
        <tr><td align="center"><script type="text/javascript">jmolRadio("select all;wireframe off;spacefill off;label off;;set spin x 5;set spin y 5;spin off;","",false);</script></td><td align="left">4</td><td align="center">0.060</td><td><a href="https://www.rcsb.org/pdb/explore/explore.do?structureId=1jjf" target="_blank"><font face="Monospace">1jjfA</font></a></td><td align="center">0.428</td><td align="center">4.94</td><td align="center">0.028</td><td align="right">0.797</td><td align="left"><span title="Endo-1,4-beta-xylanase."><a href="http://enzyme.expasy.org/EC/3.2.1.8" target="_blank">3.2.1.8</a></span>&nbsp;</td><td align="left">NA</td></tr>
        <tr><td align="center"><script type="text/javascript">jmolRadio("select all;wireframe off;spacefill off;label off;;set spin x 5;set spin y 5;spin off;","",false);</script></td><td align="left">5</td><td align="center">0.060</td><td><a href="https://www.rcsb.org/pdb/explore/explore.do?structureId=3grk" target="_blank"><font face="Monospace">3grkB</font></a></td><td align="center">0.434</td><td align="center">4.34</td><td align="center">0.049</td><td align="right">0.699</td><td align="left"><span title="Enoyl-[acyl-carrier-protein] reductase (NADH)."><a href="http://enzyme.expasy.org/EC/1.3.1.9" target="_blank">1.3.1.9</a></span>&nbsp;</td><td align="left">NA</td></tr>
        </table><br/>

        <table border="0" cellspacing="2" border="0" style="font-family:Arial;font-size:12px;">
            <tr><td valign="top">&nbsp;</td><td align="justify"><b>Click on the radio buttons to visualize predicted active site residues.</b></td></tr>
            <tr><td valign="top">(a)</td><td align="justify">Cscore<sup>EC</sup> is the confidence score for the Enzyme Commission (EC) number prediction. Cscore<sup>EC</sup> values range in between [0-1]; where a higher score indicates a more reliable EC number prediction.</td></tr>
            <tr><td valign="top">(b)</td><td align="justify">TM-score is a measure of  global structural similarity between query and template protein.</td></tr>
            <tr><td valign="bottom">(c)</td><td align="justify">RMSD<sup>a</sup> is the RMSD between residues that are structurally aligned by TM-align.</td></tr>
            <tr><td valign="bottom">(d)</td><td align="justify">IDEN<sup>a</sup> is the percentage sequence identity in the structurally aligned region.</td></tr>
            <tr><td valign="top">(e)</td><td align="justify">Cov. represents the coverage of global structural  alignment and is equal to the number of structurally aligned residues divided by length of the query protein.</td></tr>
        </table>
    </td></tr>
    </table>
</ul></td></tr></table>

<table><tr><td><div style="background:#6599FF;width:500px;"><b><font face="Arial" 
color="#FFFFFF" size=4>&nbsp;Predicted Ligand Binding Sites</font></b></div></td>
<td><a id="option9" href="javascript:formAction('hide',9)">
<img id="optionGraphic9" border=0 src="/COFACTOR/img/down_triangle.gif"/>
[Show/Hide]</a></td></tr></table>
<table id="optionForm9"><tr><td><ul>
    <table border="0" style="background:#F2F2F2";align="left">
        <tr><td align ="center"><script type="text/javascript">
        jmolInitialize("/jmol/");
        jmolSetAppletColor("#000000");
        jmolApplet(500, "load complex1.pdb.gz;set frank off;select all;color group;spacefill off;wireframe off;cartoons;select hetero;spacefill 95%;set radius 1.0;color cpk;set ambient 40;select 65;wireframe 70;spacefill 100;color cpk;select 65 and *.ca; label %m%R; color label magenta;select 104;wireframe 70;spacefill 100;color cpk;select 104 and *.ca; label %m%R; color label magenta;select 105;wireframe 70;spacefill 100;color cpk;select 105 and *.ca; label %m%R; color label magenta;;set spin x 10; set spin y 10; spin off;");
        jmolButton("Reset","Reset");
        jmolCheckbox("set spin x 10; set spin y 10; spin on",
            "spin off", "Spin", false);
        jmolCheckbox("set antialiasDisplay true",
            "set antialiasDisplay false", "High quality", false);
        jmolCheckbox("color background white",
            "color background black", "White background", false);
        jmolButton("write PNGJ model.png","Save image");
        </script></td>
        <td valign="top"><b>Template proteins with similar binding site:</b><br/><br/>
        <table border="0" style="font-family:Arial;font-size:13px;">
        <tr><th>Click<br>to view</th><th>Rank</th><th>Cscore<sup>LB</sup></th><th>PDB<br/>Hit</th><th>TM-score</th><th>RMSD<sup>a</sup></th><th>IDEN<sup>a</sup></th><th>Cov.</th><th>BS-score</th><th>Lig. Name</th><th>Download<br/>Complex</th><th align="left">Predicted binding site residues</th></tr>
        
        <tr><td align="center"><script type="text/javascript">jmolRadio("load complex1.pdb.gz;set frank off;select all;color group;spacefill off;wireframe off;cartoons;select hetero;spacefill 95%;set radius 1.0;color cpk;set ambient 40;select 65;wireframe 70;spacefill 100;color cpk;select 65 and *.ca; label %m%R; color label magenta;select 104;wireframe 70;spacefill 100;color cpk;select 104 and *.ca; label %m%R; color label magenta;select 105;wireframe 70;spacefill 100;color cpk;select 105 and *.ca; label %m%R; color label magenta;;set spin x 10;set spin y 10;spin off;","",false);</script></td><td align="left">1</td><td align="center">0.01</td><td><a href="https://www.rcsb.org/pdb/explore/explore.do?structureId=1zxl" target="_blank"><font face="Monospace">1zxlB</font></a></td><td align="center">0.429</td><td align="center">4.25</td><td align="center">0.061</td><td align="right">0.707</td><td align="left">0.17</td><td align="left"><font face="Monospace"><a href="http://zhanglab.ccmb.med.umich.edu/BioLiP/sym.cgi?code=JP1" target="_blank">JP1</a></font></td><td><a href="complex1.pdb.gz" download>complex1.pdb.gz</a></td><td align="left">65,104,105</td></tr>
        <tr><td align="center"><script type="text/javascript">jmolRadio("load complex2.pdb.gz;set frank off;select all;color group;spacefill off;wireframe off;cartoons;select hetero;spacefill 95%;set radius 1.0;color cpk;set ambient 40;select 14;wireframe 70;spacefill 100;color cpk;select 14 and *.ca; label %m%R; color label magenta;select 115;wireframe 70;spacefill 100;color cpk;select 115 and *.ca; label %m%R; color label magenta;select 123;wireframe 70;spacefill 100;color cpk;select 123 and *.ca; label %m%R; color label magenta;;set spin x 10;set spin y 10;spin off;","",false);</script></td><td align="left">2</td><td align="center">0.01</td><td><a href="https://www.rcsb.org/pdb/explore/explore.do?structureId=3lt1" target="_blank"><font face="Monospace">3lt1A</font></a></td><td align="center">0.429</td><td align="center">4.24</td><td align="center">0.061</td><td align="right">0.707</td><td align="left">0.19</td><td align="left"><font face="Monospace"><a href="http://zhanglab.ccmb.med.umich.edu/BioLiP/sym.cgi?code=FT2" target="_blank">FT2</a></font></td><td><a href="complex2.pdb.gz" download>complex2.pdb.gz</a></td><td align="left">14,115,123</td></tr>
        <tr><td align="center"><script type="text/javascript">jmolRadio("load complex3.pdb.gz;set frank off;select all;color group;spacefill off;wireframe off;cartoons;select hetero;spacefill 95%;set radius 1.0;color cpk;set ambient 40;select 65;wireframe 70;spacefill 100;color cpk;select 65 and *.ca; label %m%R; color label magenta;select 103;wireframe 70;spacefill 100;color cpk;select 103 and *.ca; label %m%R; color label magenta;select 104;wireframe 70;spacefill 100;color cpk;select 104 and *.ca; label %m%R; color label magenta;;set spin x 10;set spin y 10;spin off;","",false);</script></td><td align="left">3</td><td align="center">0.01</td><td><a href="https://www.rcsb.org/pdb/explore/explore.do?structureId=1zsn" target="_blank"><font face="Monospace">1zsnA</font></a></td><td align="center">0.430</td><td align="center">4.23</td><td align="center">0.061</td><td align="right">0.707</td><td align="left">0.19</td><td align="left"><font face="Monospace"><a href="http://zhanglab.ccmb.med.umich.edu/BioLiP/sym.cgi?code=TN2" target="_blank">TN2</a></font></td><td><a href="complex3.pdb.gz" download>complex3.pdb.gz</a></td><td align="left">65,103,104</td></tr>
        <tr><td align="center"><script type="text/javascript">jmolRadio("load complex4.pdb.gz;set frank off;select all;color group;spacefill off;wireframe off;cartoons;select hetero;spacefill 95%;set radius 1.0;color cpk;set ambient 40;select 105;wireframe 70;spacefill 100;color cpk;select 105 and *.ca; label %m%R; color label magenta;select 116;wireframe 70;spacefill 100;color cpk;select 116 and *.ca; label %m%R; color label magenta;select 117;wireframe 70;spacefill 100;color cpk;select 117 and *.ca; label %m%R; color label magenta;;set spin x 10;set spin y 10;spin off;","",false);</script></td><td align="left">4</td><td align="center">0.01</td><td><a href="https://www.rcsb.org/pdb/explore/explore.do?structureId=1uay" target="_blank"><font face="Monospace">1uayA</font></a></td><td align="center">0.431</td><td align="center">4.62</td><td align="center">0.044</td><td align="right">0.740</td><td align="left">0.35</td><td align="left"><font face="Monospace"><a href="http://zhanglab.ccmb.med.umich.edu/BioLiP/sym.cgi?code=ADN" target="_blank">ADN</a></font></td><td><a href="complex4.pdb.gz" download>complex4.pdb.gz</a></td><td align="left">105,116,117</td></tr>
        <tr><td align="center"><script type="text/javascript">jmolRadio("load complex5.pdb.gz;set frank off;select all;color group;spacefill off;wireframe off;cartoons;select hetero;spacefill 95%;set radius 1.0;color cpk;set ambient 40;select 18;wireframe 70;spacefill 100;color cpk;select 18 and *.ca; label %m%R; color label magenta;select 46;wireframe 70;spacefill 100;color cpk;select 46 and *.ca; label %m%R; color label magenta;select 47;wireframe 70;spacefill 100;color cpk;select 47 and *.ca; label %m%R; color label magenta;select 48;wireframe 70;spacefill 100;color cpk;select 48 and *.ca; label %m%R; color label magenta;select 53;wireframe 70;spacefill 100;color cpk;select 53 and *.ca; label %m%R; color label magenta;;set spin x 10;set spin y 10;spin off;","",false);</script></td><td align="left">5</td><td align="center">0.01</td><td><a href="https://www.rcsb.org/pdb/explore/explore.do?structureId=3zv6" target="_blank"><font face="Monospace">3zv6A</font></a></td><td align="center">0.438</td><td align="center">4.16</td><td align="center">0.037</td><td align="right">0.707</td><td align="left">0.11</td><td align="left"><font face="Monospace"><a href="http://zhanglab.ccmb.med.umich.edu/BioLiP/sym.cgi?code=NAD" target="_blank">NAD</a></font></td><td><a href="complex5.pdb.gz" download>complex5.pdb.gz</a></td><td align="left">18,46,47,48,53</td></tr>
        <tr><td align="center"><script type="text/javascript">jmolRadio("load complex6.pdb.gz;set frank off;select all;color group;spacefill off;wireframe off;cartoons;select hetero;spacefill 95%;set radius 1.0;color cpk;set ambient 40;select 51;wireframe 70;spacefill 100;color cpk;select 51 and *.ca; label %m%R; color label magenta;select 83;wireframe 70;spacefill 100;color cpk;select 83 and *.ca; label %m%R; color label magenta;select 84;wireframe 70;spacefill 100;color cpk;select 84 and *.ca; label %m%R; color label magenta;select 97;wireframe 70;spacefill 100;color cpk;select 97 and *.ca; label %m%R; color label magenta;;set spin x 10;set spin y 10;spin off;","",false);</script></td><td align="left">6</td><td align="center">0.01</td><td><a href="https://www.rcsb.org/pdb/explore/explore.do?structureId=3s55" target="_blank"><font face="Monospace">3s55A</font></a></td><td align="center">0.432</td><td align="center">4.37</td><td align="center">0.052</td><td align="right">0.724</td><td align="left">0.13</td><td align="left"><font face="Monospace"><a href="http://zhanglab.ccmb.med.umich.edu/BioLiP/sym.cgi?code=NAD" target="_blank">NAD</a></font></td><td><a href="complex6.pdb.gz" download>complex6.pdb.gz</a></td><td align="left">51,83,84,97</td></tr>
        </table><br/>

        <table border="0" cellspacing="2" style="font-family:Arial;font-size:12px;">
            <tr><td valign="top">&nbsp;</td><td align="justify"><b> Click on the radio buttons to visualize predicted binding site and residues.</b></td></tr>
            <tr><td valign="top">(a)</td><td align="justify">Cscore<sup>LB</sup> is the confidence score of predicted binding site. Cscore<sup>LB</sup> values range in between [0-1]; where a higher score indicates a more reliable ligand-binding site prediction. </td></tr>
            <tr><td valign="top">(b)</td><td align="justify">BS-score is a measure of local similarity (sequence &amp; structure) between template binding site and predicted binding site in the query structure. Based on large scale benchmarking analysis, we have observed that a BS-score &gt;1 reflects a significant local match between the predicted and template binding site.
            <tr><td valign="top">(c)</td><td align="justify">TM-score is a measure of  global structural similarity between query and template protein.</td></tr>
            <tr><td valign="top">(d)</td><td align="justify">RMSD<sup>a</sup> the RMSD between residues that are structurally aligned by TM-align.</td></tr>
            <tr><td valign="top">(e)</td><td align="justify">IDEN<sup>a</sup> is the percentage sequence identity in the structurally aligned region.</td></tr>
            <tr><td valign="top">(f)</td><td align="justify">Cov. represents the coverage of global structural alignment and is equal to the number of structurally aligned residues divided by length of the query protein.</td></tr>
        </table>
    </td></tr>
    </table>
</ul></td></tr></table>


<hr>
<div style="position:relative;left:5px;">
    <table style="font-family:Arial;font-size:14px;">
    <tr><td colspan=2>References:
    	<a id="option10" href="javascript:formAction('hide',10)">
	<img id="optionGraphic10" border=0 src="/COFACTOR/img/down_triangle.gif"/>
	[Show/Hide]</a>
    </td><tr>
    <tr><td>
	<table id="optionForm10">
	<tr><td valign="top">1.</td><td>Chengxin Zhang, Xiaoqiong Wei, Gilbert S. Omenn, Yang Zhang. "Structure and Protein Interaction-based Gene Ontology Annotations Reveal Likely Functions of Uncharacterized Proteins on Human Chromosome 17". Journal of Proteome Research, 17: 4186-4196 (2018).</td></tr>
	</table>
    </td></tr>
    </table>
</div><br>
</td>
</table>
</body></html>
