<html>
<head><meta http-equiv="content-type" content="text/html; charset=UTF-8">
<title>C-I-TASSER CIT290</title></head>

<script type="text/javascript" src="/jmol/JSmol.min.js"></script>
<script type="text/javascript" src="/jmol/Jmol2.js"></script>

<body>
<font face="Times New Roman">
[<a href="/C-I-TASSER">Back to server</a>]
<br>
<h1 align="center">C-I-TASSER results for CIT290</h1>
<h4 align="center">[Click <a href="result.zip" download>result.zip</a>
    to download all results on this page]</h4>
<font size="4" face="Arial">

<div style="background:#6599FF;width:500px;"><b><font face="Arial" 
color="#FFFFFF" size=4>&nbsp;Input Sequence in FASTA format</font></b></div>
<ul>
    <table style="font-family:Monospace;font-size:14px;background:#F2F2F2;">
    <tr><td colspan="2">&gt;NX_Q6UWY5 (402 residues)<br>
    MMVALRGASALLVLFLAAFLPPPQCTQDPAMVHYIYQRFRVLEQGLEKCTQATRAYIQEF<br>
QEFSKNISVMLGRCQTYTSEYKSAVGNLALRVERAQREIDYIQYLREADECIESEDKTLA<br>
EMLLQEAEEEKKIRTLLNASCDNMLMGIKSLKIVKKMMDTHGSWMKDAVYNSPKVYLLIG<br>
SRNNTVWEFANIRAFMEDNTKPAPRKQILTLSWQGTGQVIYKGFLFFHNQATSNEIIKYN<br>
LQKRTVEDRMLLPGGVGRALVYQHSPSTYIDLAVDEHGLWAIHSGPGTHSHLVLTKIEPG<br>
TLGVEHSWDTPCRSQDAEASFLLCGVLYVVYSTGGQGPHRITCIYDPLGTISEEDLPNLF<br>
FPKRPRSHSMIHYNPRDKQLYAWNEGNQIIYKLQTKRKLPLK<br>
    </td></tr>
    </table><br>
</ul>

<div style="background:#6599FF;width:500px;"><b><font face="Arial" 
color="#FFFFFF" size=4>&nbsp;Predicted Secondary Structure</font></b></div>
<ul>
    <table border="0" style="font-family:Monospace;font-size:14px;background:#F2F2F2;">
    <tr><td valign="bottom"><font="Arial"><b>Sequence</b></font></td><td>
    &nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;20&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;40&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;60&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;80&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;100&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;120&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;140&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;160&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;180&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;200&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;220&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;240&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;260&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;280&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;300&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;320&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;340&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;360&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;380&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;400<br>
&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;|&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;|&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;|&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;|&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;|&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;|&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;|&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;|&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;|&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;|&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;|&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;|&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;|&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;|&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;|&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;|&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;|&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;|&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;|&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;|<br>
    MMVALRGASALLVLFLAAFLPPPQCTQDPAMVHYIYQRFRVLEQGLEKCTQATRAYIQEFQEFSKNISVMLGRCQTYTSEYKSAVGNLALRVERAQREIDYIQYLREADECIESEDKTLAEMLLQEAEEEKKIRTLLNASCDNMLMGIKSLKIVKKMMDTHGSWMKDAVYNSPKVYLLIGSRNNTVWEFANIRAFMEDNTKPAPRKQILTLSWQGTGQVIYKGFLFFHNQATSNEIIKYNLQKRTVEDRMLLPGGVGRALVYQHSPSTYIDLAVDEHGLWAIHSGPGTHSHLVLTKIEPGTLGVEHSWDTPCRSQDAEASFLLCGVLYVVYSTGGQGPHRITCIYDPLGTISEEDLPNLFFPKRPRSHSMIHYNPRDKQLYAWNEGNQIIYKLQTKRKLPLK</td></tr>
    <tr><td><font="Arial"><span title="Predicted Secondary Structure">
    <b>Prediction</b></span></font></td><td>CCCCCCC<font color="red">HHHHHHHHHHHH</font>CCC<font color="red">HHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHH</font>CCCCCCC<font color="blue">SS</font>CCC<font color="red">HHHHHHHHHHHH</font>CCCCCCCCCCCCCCCC<font color="blue">SS</font>C<font color="blue">SS</font>CCC<font color="blue">SSSSS</font>CCCCCC<font color="blue">SSS</font>CCCCCCCC<font color="blue">SSSSSS</font>CCCCC<font color="blue">SSSSS</font>CC<font color="red">HHHHH</font>CCCCCCCC<font color="blue">SSSS</font>CCCCCCCCC<font color="blue">SSSS</font>CCC<font color="blue">SSSSS</font>CCCCCC<font color="blue">SSSSS</font>CCCCC<font color="blue">SSSSS</font>CCCCCCCCCCCCCCCCCCC<font color="blue">SSSSS</font>CCCC<font color="blue">SSSSSS</font>CCCCCCC<font color="blue">SSSSSS</font>CCCCC<font color="blue">SSSSSSS</font>CCCCC<font color="red">HHHHHHHHHH</font>C<font color="blue">SSSSSSS</font>CCCCCCC<font color="blue">SSSSSSS</font>CCCCCCCCCCCC<font color="blue">SS</font>CCCCCCCC<font color="blue">S</font>CC<font color="blue">SS</font>CCCCC<font color="blue">SSSSSS</font>CC<font color="blue">SSSSSSSSS</font>CCCCCCC</td></tr>
    <tr><td><font="Arial"><span title="Confidence Score of Prediction 
    (0 being least confident, 9 being most confident)">
    <b>Confidence</b></span></font></td><td>964212225788888864047725678999999999999999999999988779999999999999999999999999999999753899999999878887503655411113057788887642034567776655678965702158769820167445121489999982899834549869996469998625456675588546640267549986817897048988089998678807767516742258665336887617888638862999824678993899974767650599997367840588887750859998624889861899999789985333432112157655431001086557388984883899998846778999</td></tr>
    <tr><td><font="Arial"></td><td><font color=red>H</font>:Helix; 
    <font color=blue>S</font>:Strand; C:Coil</td><tr>
    </table>
</ul>

<div style="background:#6599FF;width:500px;"><b><font face="Arial" 
color="#FFFFFF" size=4>&nbsp;Predicted Solvent Accessibility</font></b></div>
<ul>
    <table border="0" style="font-family:Monospace;font-size:14px;background:#F2F2F2;">
    <tr><td valign="bottom"><font="Arial"><b>Sequence</b></font></td><td>
    &nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;20&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;40&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;60&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;80&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;100&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;120&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;140&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;160&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;180&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;200&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;220&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;240&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;260&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;280&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;300&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;320&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;340&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;360&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;380&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;400<br>
&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;|&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;|&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;|&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;|&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;|&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;|&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;|&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;|&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;|&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;|&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;|&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;|&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;|&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;|&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;|&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;|&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;|&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;|&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;|&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;|<br>
    MMVALRGASALLVLFLAAFLPPPQCTQDPAMVHYIYQRFRVLEQGLEKCTQATRAYIQEFQEFSKNISVMLGRCQTYTSEYKSAVGNLALRVERAQREIDYIQYLREADECIESEDKTLAEMLLQEAEEEKKIRTLLNASCDNMLMGIKSLKIVKKMMDTHGSWMKDAVYNSPKVYLLIGSRNNTVWEFANIRAFMEDNTKPAPRKQILTLSWQGTGQVIYKGFLFFHNQATSNEIIKYNLQKRTVEDRMLLPGGVGRALVYQHSPSTYIDLAVDEHGLWAIHSGPGTHSHLVLTKIEPGTLGVEHSWDTPCRSQDAEASFLLCGVLYVVYSTGGQGPHRITCIYDPLGTISEEDLPNLFFPKRPRSHSMIHYNPRDKQLYAWNEGNQIIYKLQTKRKLPLK</td></tr>
    <tr><td><font="Arial"><span title="Solvent Accessibility Prediction">
    <b>Prediction</b></span></font></td><td>613103300000000010102233334133104302520440453054136424522530451154046105505533540353044144315504531530543455442142525414542454456645145344430311033044033234254310000001137442000011043210100221330254344444342312221100000002000001234444200001044542533421422233211032233100000000300000000453521000010236404134204041435300000000000000102446542201000002443444330403041323311102012412200001211001020415563558</td></tr>
    <tr><td><font="Arial"></td><td>Values range from <b>0</b> (buried residue) 
    to <b>8</b> (highly exposed residue)</td><tr>
    </table>
</ul>

<div style="background:#6599FF;width:500px;"><b><font face="Arial" 
color="#FFFFFF" size=4>&nbsp;Predicted contact map used in C-I-TASSER simulation</font></b></div>
<ul>
    <table border="0" style="font-family:Monospace;font-size:14px;background:#F2F2F2;">
    <tr><td><a href=contact.png><img src=contact.png width=1000></a></td></tr>
    <tr><td>
    <table cellspacing="2" border="0" width="1000px" style="font-family:Arial;font-size:12px;">
    <tr><td>C-I-TASSER simulation is guided by consensus contact map
    (upper left triangle) derived based on confidence scores of
    <a href=/TripletRes>TripletRes</a> (lower right triangle),
    <a href=/ResTriplet>ResTriplet</a>, <a href=/ResPRE>ResPRE</a>, and 
    <a href=/NeBcon>NeBcon</a>.
    In the contact map, the axes mark the residue indexes along the sequence,
    while each dot represents a residue pair with predicted contact.
    No dot is close to the diagonal, because C-I-TASSER does not consider
    contacts for residue pairs separated by &lt;6 residues.
    </td></tr>
    </table>
    </td></tr>
    </table>
</ul>

<div style="background:#6599FF;width:500px;"><b><font face="Arial" 
color="#FFFFFF" size=4>&nbsp;Top 10 threading templates used by C-I-TASSER</font></b></div>
<ul>
    <table border="0" style="font-family:Monospace;font-size:14px;background:#F2F2F2;">
    <tr><th align="left" valign="top"><font face="Arial">Rank</font></th> <th align="center" valign="top"><font face="Arial">PDB<br/>hit</font></th><th align="center" valign="top"><font face="Arial">ID1</font></th><th align="center" valign="top"><font face="Arial">ID2</font></th><th valign="top"><font face="Arial">Cov</font></th><th valign="top"><font face="Arial">Norm.<br/>Zscore</font></th><th valign="top"><font face="Arial">Download</br>alignment</font></th><td>&nbsp;</td><td BGCOLOR="#DEDEDE" style="font-family:Monospace;font-size:13px;">&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;20&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;40&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;60&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;80&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;100&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;120&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;140&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;160&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;180&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;200&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;220&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;240&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;260&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;280&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;300&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;320&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;340&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;360&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;380&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;400<br>
&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;|&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;|&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;|&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;|&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;|&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;|&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;|&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;|&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;|&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;|&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;|&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;|&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;|&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;|&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;|&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;|&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;|&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;|&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;|&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;|</td></tr>
    <tr><td colspan="7"></td><th align="right" valign ="bottom" ><font face="Arial">SS<br/>Seq</font></th><td BGCOLOR="#D7D7D7" style="font-family:Monospace;font-size:13px;">CCCCCCC<font color="red">HHHHHHHHHHHH</font>CCC<font color="red">HHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHH</font>CCCCCCC<font color="blue">SS</font>CCC<font color="red">HHHHHHHHHHHH</font>CCCCCCCCCCCCCCCC<font color="blue">SS</font>C<font color="blue">SS</font>CCC<font color="blue">SSSSS</font>CCCCCC<font color="blue">SSS</font>CCCCCCCC<font color="blue">SSSSSS</font>CCCCC<font color="blue">SSSSS</font>CC<font color="red">HHHHH</font>CCCCCCCC<font color="blue">SSSS</font>CCCCCCCCC<font color="blue">SSSS</font>CCC<font color="blue">SSSSS</font>CCCCCC<font color="blue">SSSSS</font>CCCCC<font color="blue">SSSSS</font>CCCCCCCCCCCCCCCCCCC<font color="blue">SSSSS</font>CCCC<font color="blue">SSSSSS</font>CCCCCCC<font color="blue">SSSSSS</font>CCCCC<font color="blue">SSSSSSS</font>CCCCC<font color="red">HHHHHHHHHH</font>C<font color="blue">SSSSSSS</font>CCCCCCC<font color="blue">SSSSSSS</font>CCCCCCCCCCCC<font color="blue">SS</font>CCCCCCCC<font color="blue">S</font>CC<font color="blue">SS</font>CCCCC<font color="blue">SSSSSS</font>CC<font color="blue">SSSSSSSSS</font>CCCCCCC<br/>MMVALRGASALLVLFLAAFLPPPQCTQDPAMVHYIYQRFRVLEQGLEKCTQATRAYIQEFQEFSKNISVMLGRCQTYTSEYKSAVGNLALRVERAQREIDYIQYLREADECIESEDKTLAEMLLQEAEEEKKIRTLLNASCDNMLMGIKSLKIVKKMMDTHGSWMKDAVYNSPKVYLLIGSRNNTVWEFANIRAFMEDNTKPAPRKQILTLSWQGTGQVIYKGFLFFHNQATSNEIIKYNLQKRTVEDRMLLPGGVGRALVYQHSPSTYIDLAVDEHGLWAIHSGPGTHSHLVLTKIEPGTLGVEHSWDTPCRSQDAEASFLLCGVLYVVYSTGGQGPHRITCIYDPLGTISEEDLPNLFFPKRPRSHSMIHYNPRDKQLYAWNEGNQIIYKLQTKRKLPLK</td></tr>

    <tr><td align="center"><font face="Arial">1</font></td><td><a href="https://www.rcsb.org/pdb/explore/explore.do?structureId=5amo" target=_blank>5amoA</a></td><td align="center"><font face="Arial"> 0.31</font></td><td><font face="Arial"> 0.19</font></td><td><font face="Arial"> 5.79</font></td><td align="center"><font face="Arial"> <b>2.68</b></font></td><td align="center"><font face="Monospace"><a href="threading1.pdb.gz" download>SPARKS-K</a></font></td><td>&nbsp;</td><td BGCOLOR="#ECECEG" style="font-family:Monospace;font-size:13px;">-----------------------------------------------------------------------------------------------------------------------------VSNL<font color=#c048c0>E</font>ERL<font color=#f01505>R</font>ACMQKLACGK<font color=#80a0f0>L</font>T<font color=#f09048>G</font><font color=#80a0f0>I</font>SDPVT<font color=#80a0f0>V</font><font color=#f01505>K</font>TSGSRF<font color=#f09048>G</font><font color=#15c015>S</font><font color=#80a0f0>W</font><font color=#80a0f0>M</font>T<font color=#c048c0>D</font>PLEGDNR<font color=#80a0f0>V</font>WYMD<font color=#f09048>G</font>YH<font color=#15c015>N</font><font color=#15c015>N</font>R<font color=#80a0f0>V</font>R<font color=#c048c0>E</font>YKSMVD<font color=#80a0f0>F</font><font color=#80a0f0>M</font>NTDNF---TSHR<font color=#80a0f0>L</font>PHP<font color=#80a0f0>W</font>S<font color=#f09048>G</font><font color=#15c015>T</font><font color=#f09048>G</font><font color=#15c015>Q</font><font color=#80a0f0>V</font>V<font color=#15a4a4>Y</font>N<font color=#f09048>G</font>SIY<font color=#80a0f0>F</font>NKF-Q<font color=#15c015>S</font>HI<font color=#80a0f0>I</font><font color=#80a0f0>I</font>RFD<font color=#80a0f0>L</font>KTE<font color=#15c015>T</font>ILKTRS<font color=#80a0f0>L</font>DY-------------SD<font color=#80a0f0>I</font><font color=#c048c0>D</font><font color=#80a0f0>L</font>M<font color=#80a0f0>V</font><font color=#c048c0>D</font><font color=#c048c0>E</font>N<font color=#f09048>G</font><font color=#80a0f0>L</font><font color=#80a0f0>W</font><font color=#80a0f0>A</font>VYATNQNAGNI<font color=#80a0f0>V</font>IS<font color=#f01505>K</font>LD<font color=#c0c000>P</font>VS<font color=#80a0f0>L</font>QILQT<font color=#80a0f0>W</font>N<font color=#15c015>T</font>SYPKRS<font color=#80a0f0>A</font>GEA<font color=#80a0f0>F</font>II<font color=#f08080>C</font><font color=#f09048>G</font>T<font color=#80a0f0>L</font><font color=#15a4a4>Y</font><font color=#80a0f0>V</font>TNGYS<font color=#f09048>G</font>G--TKVHYA<font color=#15a4a4>Y</font>QTNASTY<font color=#c048c0>E</font>Y--IDIP<font color=#80a0f0>F</font>QNKYSHI<font color=#15c015>S</font><font color=#80a0f0>M</font>LD<font color=#15a4a4>Y</font><font color=#15c015>N</font><font color=#c0c000>P</font>K<font color=#c048c0>D</font>RA<font color=#80a0f0>L</font><font color=#15a4a4>Y</font><font color=#80a0f0>A</font><font color=#80a0f0>W</font><font color=#15c015>N</font>N<font color=#f09048>G</font>H<font color=#15c015>Q</font>TL<font color=#15a4a4>Y</font>NVTLF------</td></tr>
    <tr><td align="center"><font face="Arial">2</font></td><td><a href="https://www.rcsb.org/pdb/explore/explore.do?structureId=4d7c" target=_blank>4d7cA</a></td><td align="center"><font face="Arial"> 0.24</font></td><td><font face="Arial"> 0.15</font></td><td><font face="Arial"> 4.57</font></td><td align="center"><font face="Arial"> <b>2.92</b></font></td><td align="center"><font face="Monospace"><a href="threading2.pdb.gz" download>FFAS-3D</a></font></td><td>&nbsp;</td><td BGCOLOR="#ECECEG" style="font-family:Monospace;font-size:13px;">---------------------------------------------------------------------------------------------------------------------------------------------EP<font color=#80a0f0>M</font>ITS<font color=#80a0f0>I</font>GNPAQ<font color=#80a0f0>V</font>L<font color=#f01505>K</font>VKE<font color=#15c015>T</font>F<font color=#f09048>G</font>T<font color=#80a0f0>W</font>LRESANR<font color=#15c015>S</font>DRIWVTEHFSGIM<font color=#80a0f0>V</font>K<font color=#c048c0>E</font><font color=#80a0f0>F</font>EDLP<font color=#80a0f0>A</font>LLNSS----FTLLH<font color=#80a0f0>L</font>PHYFH<font color=#f09048>G</font>C<font color=#f09048>G</font>HAV<font color=#15a4a4>Y</font>NNS<font color=#80a0f0>L</font>YY<font color=#15a4a4>H</font>-KGG<font color=#15c015>S</font><font color=#15c015>N</font>T<font color=#80a0f0>I</font>VRFEFG<font color=#f01505>K</font>E<font color=#15c015>T</font>PQTLK<font color=#80a0f0>L</font>EDALYFDRKYLFANSK<font color=#15c015>T</font><font color=#15a4a4>Y</font>FNI<font color=#80a0f0>A</font><font color=#80a0f0>V</font><font color=#c048c0>D</font><font color=#c048c0>E</font>K<font color=#f09048>G</font><font color=#80a0f0>L</font><font color=#80a0f0>W</font>I<font color=#80a0f0>I</font>YASSVDG<font color=#15c015>S</font>SILVAQLDER<font color=#15c015>T</font>FS<font color=#80a0f0>V</font>LRHIN<font color=#15c015>T</font>TYPKSK<font color=#80a0f0>A</font>GNA<font color=#80a0f0>F</font>IAQ<font color=#f09048>G</font>I<font color=#80a0f0>L</font><font color=#15a4a4>Y</font><font color=#80a0f0>V</font>TDT----KDT<font color=#f01505>R</font>V<font color=#15c015>T</font>FAF<font color=#c048c0>D</font>L<font color=#80a0f0>L</font>RG--KQINA<font color=#15c015>N</font>FGLRMSQSVLA<font color=#80a0f0>M</font>LS<font color=#15a4a4>Y</font><font color=#15c015>N</font>M<font color=#f01505>R</font><font color=#c048c0>D</font>QH<font color=#80a0f0>L</font><font color=#15a4a4>Y</font>S<font color=#80a0f0>W</font>ED<font color=#f09048>G</font>HLML<font color=#15a4a4>Y</font>PVHFSS-----</td></tr>
    <tr><td align="center"><font face="Arial">3</font></td><td><a href="https://www.rcsb.org/pdb/explore/explore.do?structureId=5afb" target=_blank>5afbA</a></td><td align="center"><font face="Arial"> 0.29</font></td><td><font face="Arial"> 0.20</font></td><td><font face="Arial"> 6.05</font></td><td align="center"><font face="Arial"> <b>8.12</b></font></td><td align="center"><font face="Monospace"><a href="threading3.pdb.gz" download>HHpred</a></font></td><td>&nbsp;</td><td BGCOLOR="#ECECEG" style="font-family:Monospace;font-size:13px;">-------------------------------------------------------------------------------------------------------PDVFP<font color=#c048c0>D</font>P<font color=#f08080>C</font>PGTY-<font color=#f01505>K</font>Y<font color=#80a0f0>L</font>EVQYECVPYK-----VEKVFL<font color=#f08080>C</font>PGL<font color=#80a0f0>L</font>K<font color=#f09048>G</font>VYQSEHLFESDHQS<font color=#f09048>G</font>A<font color=#80a0f0>W</font>C<font color=#f01505>K</font><font color=#c048c0>D</font>PLQA<font color=#15c015>S</font>D<font color=#f01505>K</font>I<font color=#15a4a4>Y</font>YMPWT<font color=#f01505>R</font>TD<font color=#15c015>T</font>LT<font color=#c048c0>E</font>YSSKDD<font color=#80a0f0>F</font>IAGR---PTTTYK<font color=#80a0f0>L</font>PHRVD<font color=#f09048>G</font><font color=#15c015>T</font><font color=#f09048>G</font>F<font color=#80a0f0>V</font>V<font color=#15a4a4>Y</font>D<font color=#f09048>G</font>A<font color=#80a0f0>L</font><font color=#80a0f0>F</font><font color=#80a0f0>F</font>NKER<font color=#15c015>T</font>-RN<font color=#80a0f0>I</font>V<font color=#f01505>K</font>FD<font color=#80a0f0>L</font>RT<font color=#f01505>R</font>IKSGEAIIAN-ANDTSP<font color=#15a4a4>Y</font>RWGGKSD<font color=#80a0f0>I</font><font color=#c048c0>D</font><font color=#80a0f0>L</font><font color=#80a0f0>A</font><font color=#80a0f0>V</font><font color=#c048c0>D</font><font color=#c048c0>E</font>N<font color=#f09048>G</font><font color=#80a0f0>L</font><font color=#80a0f0>W</font>V<font color=#80a0f0>I</font>YATEQNNGKI<font color=#80a0f0>V</font>ISQLN<font color=#c0c000>P</font>Y<font color=#15c015>T</font><font color=#80a0f0>L</font>RI<font color=#c048c0>E</font>GT<font color=#80a0f0>W</font><font color=#c048c0>D</font><font color=#15c015>T</font>AYDKRS<font color=#80a0f0>A</font>SNA<font color=#80a0f0>F</font>MI<font color=#f08080>C</font><font color=#f09048>G</font>I<font color=#80a0f0>L</font><font color=#15a4a4>Y</font><font color=#80a0f0>V</font><font color=#80a0f0>V</font>K<font color=#15c015>S</font>------NK<font color=#80a0f0>I</font>DY<font color=#80a0f0>I</font><font color=#15a4a4>Y</font>NTDQ--<font color=#15c015>S</font>KDS<font color=#80a0f0>L</font>VDVP<font color=#80a0f0>F</font><font color=#c0c000>P</font>NSYQYIAAVD<font color=#15a4a4>Y</font><font color=#15c015>N</font><font color=#c0c000>P</font><font color=#f01505>R</font><font color=#c048c0>D</font>NL<font color=#80a0f0>L</font><font color=#15a4a4>Y</font>V<font color=#80a0f0>W</font><font color=#15c015>N</font>NYHVVK<font color=#15a4a4>Y</font>S<font color=#80a0f0>L</font>DF-------</td></tr>
    <tr><td align="center"><font face="Arial">4</font></td><td><a href="https://www.rcsb.org/pdb/explore/explore.do?structureId=5afb" target=_blank>5afbA2</a></td><td align="center"><font face="Arial"> 0.30</font></td><td><font face="Arial"> 0.18</font></td><td><font face="Arial"> 5.59</font></td><td align="center"><font face="Arial"> <b>2.13</b></font></td><td align="center"><font face="Monospace"><a href="threading4.pdb.gz" download>MUSTER</a></font></td><td>&nbsp;</td><td BGCOLOR="#ECECEG" style="font-family:Monospace;font-size:13px;">-------------------------------------------------------------------------------------------------------------------------------------VEQKVFL<font color=#f08080>C</font>PGL<font color=#80a0f0>L</font>K<font color=#f09048>G</font>VYQSEHLFESDHQS<font color=#f09048>G</font>A<font color=#80a0f0>W</font>C<font color=#f01505>K</font><font color=#c048c0>D</font>PLQA<font color=#15c015>S</font>D<font color=#f01505>K</font>I<font color=#15a4a4>Y</font>YMPWT<font color=#f01505>R</font>TD<font color=#15c015>T</font>LT<font color=#c048c0>E</font>YSSKDD<font color=#80a0f0>F</font>IAGRP---TTTYK<font color=#80a0f0>L</font>PHRVD<font color=#f09048>G</font><font color=#15c015>T</font><font color=#f09048>G</font>F<font color=#80a0f0>V</font>V<font color=#15a4a4>Y</font>D<font color=#f09048>G</font>A<font color=#80a0f0>L</font><font color=#80a0f0>F</font><font color=#80a0f0>F</font>NK-ERTRN<font color=#80a0f0>I</font>V<font color=#f01505>K</font>FD<font color=#80a0f0>L</font>RT<font color=#f01505>R</font>IKSGEAIIANNYHDTSP<font color=#15a4a4>Y</font>RWGGKSD<font color=#80a0f0>I</font><font color=#c048c0>D</font><font color=#80a0f0>L</font><font color=#80a0f0>A</font><font color=#80a0f0>V</font><font color=#c048c0>D</font><font color=#c048c0>E</font>N<font color=#f09048>G</font><font color=#80a0f0>L</font><font color=#80a0f0>W</font>V<font color=#80a0f0>I</font>YATEQNNGKI<font color=#80a0f0>V</font>ISQLN<font color=#c0c000>P</font>Y<font color=#15c015>T</font><font color=#80a0f0>L</font>RI<font color=#c048c0>E</font>GT<font color=#80a0f0>W</font><font color=#c048c0>D</font><font color=#15c015>T</font>AYDKRS<font color=#80a0f0>A</font>SNA<font color=#80a0f0>F</font>MI<font color=#f08080>C</font><font color=#f09048>G</font>I<font color=#80a0f0>L</font><font color=#15a4a4>Y</font><font color=#80a0f0>V</font><font color=#80a0f0>V</font>K<font color=#15c015>S</font>------NK<font color=#80a0f0>I</font>DY<font color=#80a0f0>I</font><font color=#15a4a4>Y</font>NTDQS--KDS<font color=#80a0f0>L</font>VDVP<font color=#80a0f0>F</font><font color=#c0c000>P</font>NSYQYIAAVD<font color=#15a4a4>Y</font><font color=#15c015>N</font><font color=#c0c000>P</font><font color=#f01505>R</font><font color=#c048c0>D</font>NL<font color=#80a0f0>L</font><font color=#15a4a4>Y</font>V<font color=#80a0f0>W</font><font color=#15c015>N</font>NYHVVK<font color=#15a4a4>Y</font>S<font color=#80a0f0>L</font>DF-------</td></tr>
    <tr><td align="center"><font face="Arial">5</font></td><td><a href="https://www.rcsb.org/pdb/explore/explore.do?structureId=4xat" target=_blank>4xatA</a></td><td align="center"><font face="Arial"> 0.30</font></td><td><font face="Arial"> 0.19</font></td><td><font face="Arial"> 5.65</font></td><td align="center"><font face="Arial"> <b>2.20</b></font></td><td align="center"><font face="Monospace"><a href="threading5.pdb.gz" download>CNFpred</a></font></td><td>&nbsp;</td><td BGCOLOR="#ECECEG" style="font-family:Monospace;font-size:13px;">-----------------------------------------------------------------------------------------------------------------------------------------------K<font color=#80a0f0>L</font>T<font color=#f09048>G</font><font color=#80a0f0>I</font>SDPVT<font color=#80a0f0>V</font><font color=#f01505>K</font>TSGSRF<font color=#f09048>G</font><font color=#15c015>S</font><font color=#80a0f0>W</font><font color=#80a0f0>M</font>T<font color=#c048c0>D</font>PLAGDNR<font color=#80a0f0>V</font>WYMD<font color=#f09048>G</font>YN<font color=#15c015>N</font>RF<font color=#80a0f0>V</font>R<font color=#c048c0>E</font>YKSMVD<font color=#80a0f0>F</font><font color=#80a0f0>M</font>NTD---NFTSHR<font color=#80a0f0>L</font>PHP<font color=#80a0f0>W</font>S<font color=#f09048>G</font><font color=#15c015>T</font><font color=#f09048>G</font><font color=#15c015>Q</font><font color=#80a0f0>V</font>V<font color=#15a4a4>Y</font>N<font color=#f09048>G</font>SIY<font color=#80a0f0>F</font>NKFQ-<font color=#15c015>S</font>HI<font color=#80a0f0>I</font><font color=#80a0f0>I</font>RFD<font color=#80a0f0>L</font>KTE<font color=#15c015>T</font>ILKTRS<font color=#80a0f0>L</font>DYAGYNMYH<font color=#15a4a4>Y</font>AWGGHSD<font color=#80a0f0>I</font><font color=#c048c0>D</font><font color=#80a0f0>L</font>M<font color=#80a0f0>V</font><font color=#c048c0>D</font><font color=#c048c0>E</font>S<font color=#f09048>G</font><font color=#80a0f0>L</font><font color=#80a0f0>W</font><font color=#80a0f0>A</font>VYATNQNAGNI<font color=#80a0f0>V</font>VSRLD<font color=#c0c000>P</font>VS<font color=#80a0f0>L</font>QTLQT<font color=#80a0f0>W</font>N<font color=#15c015>T</font>SYPKRS<font color=#80a0f0>A</font>GEA<font color=#80a0f0>F</font>II<font color=#f08080>C</font><font color=#f09048>G</font>T<font color=#80a0f0>L</font><font color=#15a4a4>Y</font><font color=#80a0f0>V</font>TNGYS--<font color=#f09048>G</font>GTKVHYA<font color=#15a4a4>Y</font>QTNASTY<font color=#c048c0>E</font>--YIDIP<font color=#80a0f0>F</font>QNKYSHI<font color=#15c015>S</font><font color=#80a0f0>M</font>LD<font color=#15a4a4>Y</font><font color=#15c015>N</font><font color=#c0c000>P</font>K<font color=#c048c0>D</font>RA<font color=#80a0f0>L</font><font color=#15a4a4>Y</font><font color=#80a0f0>A</font><font color=#80a0f0>W</font><font color=#15c015>N</font>N<font color=#f09048>G</font>H<font color=#15c015>Q</font><font color=#80a0f0>I</font>L<font color=#15a4a4>Y</font>NVTLFH-----</td></tr>
    <tr><td align="center"><font face="Arial">6</font></td><td><a href="https://www.rcsb.org/pdb/explore/explore.do?structureId=5afb" target=_blank>5afbA2</a></td><td align="center"><font face="Arial"> 0.31</font></td><td><font face="Arial"> 0.19</font></td><td><font face="Arial"> 5.79</font></td><td align="center"><font face="Arial"> <b>3.04</b></font></td><td align="center"><font face="Monospace"><a href="threading6.pdb.gz" download>HHsearch-2</a></font></td><td>&nbsp;</td><td BGCOLOR="#ECECEG" style="font-family:Monospace;font-size:13px;">-------------------------------------------------------------------------------------------------------------------------------V<font color=#c048c0>E</font>----QK--VFL<font color=#f08080>C</font>PGL<font color=#80a0f0>L</font>K<font color=#f09048>G</font>VYQSEHLFESDHQS<font color=#f09048>G</font>A<font color=#80a0f0>W</font>C<font color=#f01505>K</font><font color=#c048c0>D</font>PLQA<font color=#15c015>S</font>D<font color=#f01505>K</font>I<font color=#15a4a4>Y</font>YMPWT<font color=#f01505>R</font>TD<font color=#15c015>T</font>LT<font color=#c048c0>E</font>YSSKDD<font color=#80a0f0>F</font>IAGR---PTTTYK<font color=#80a0f0>L</font>PHRVD<font color=#f09048>G</font><font color=#15c015>T</font><font color=#f09048>G</font>F<font color=#80a0f0>V</font>V<font color=#15a4a4>Y</font>D<font color=#f09048>G</font>A<font color=#80a0f0>L</font><font color=#80a0f0>F</font><font color=#80a0f0>F</font>NKER<font color=#15c015>T</font>-RN<font color=#80a0f0>I</font>V<font color=#f01505>K</font>FD<font color=#80a0f0>L</font>RT<font color=#f01505>R</font>IKSGEAIIAN-ANYTSP<font color=#15a4a4>Y</font>RWGGKSD<font color=#80a0f0>I</font><font color=#c048c0>D</font><font color=#80a0f0>L</font><font color=#80a0f0>A</font><font color=#80a0f0>V</font><font color=#c048c0>D</font><font color=#c048c0>E</font>N<font color=#f09048>G</font><font color=#80a0f0>L</font><font color=#80a0f0>W</font>V<font color=#80a0f0>I</font>YATEQNNGKI<font color=#80a0f0>V</font>ISQLN<font color=#c0c000>P</font>Y<font color=#15c015>T</font><font color=#80a0f0>L</font>RI<font color=#c048c0>E</font>GT<font color=#80a0f0>W</font><font color=#c048c0>D</font><font color=#15c015>T</font>AYDKRS<font color=#80a0f0>A</font>SNA<font color=#80a0f0>F</font>MI<font color=#f08080>C</font><font color=#f09048>G</font>I<font color=#80a0f0>L</font><font color=#15a4a4>Y</font><font color=#80a0f0>V</font><font color=#80a0f0>V</font>K<font color=#15c015>S</font>------NK<font color=#80a0f0>I</font>DY<font color=#80a0f0>I</font><font color=#15a4a4>Y</font>NTDQ--<font color=#15c015>S</font>KDS<font color=#80a0f0>L</font>VDVP<font color=#80a0f0>F</font><font color=#c0c000>P</font>NSYQYIAAVD<font color=#15a4a4>Y</font><font color=#15c015>N</font><font color=#c0c000>P</font><font color=#f01505>R</font><font color=#c048c0>D</font>NL<font color=#80a0f0>L</font><font color=#15a4a4>Y</font>V<font color=#80a0f0>W</font><font color=#15c015>N</font>NYHVVK<font color=#15a4a4>Y</font>S<font color=#80a0f0>L</font>DF-------</td></tr>
    <tr><td align="center"><font face="Arial">7</font></td><td><a href="https://www.rcsb.org/pdb/explore/explore.do?structureId=5afb" target=_blank>5afbA2</a></td><td align="center"><font face="Arial"> 0.30</font></td><td><font face="Arial"> 0.18</font></td><td><font face="Arial"> 5.59</font></td><td align="center"><font face="Arial"> <b>3.15</b></font></td><td align="center"><font face="Monospace"><a href="threading7.pdb.gz" download>Neff-PPAS</a></font></td><td>&nbsp;</td><td BGCOLOR="#ECECEG" style="font-family:Monospace;font-size:13px;">-------------------------------------------------------------------------------------------------------------------------------------VEQKVFL<font color=#f08080>C</font>PGL<font color=#80a0f0>L</font>K<font color=#f09048>G</font>VYQSEHLFESDHQS<font color=#f09048>G</font>A<font color=#80a0f0>W</font>C<font color=#f01505>K</font><font color=#c048c0>D</font>PLQA<font color=#15c015>S</font>D<font color=#f01505>K</font>I<font color=#15a4a4>Y</font>YMPWY<font color=#f01505>R</font>TD<font color=#15c015>T</font>LT<font color=#c048c0>E</font>YSSKDD<font color=#80a0f0>F</font>IAGR---PTTTYK<font color=#80a0f0>L</font>PHRVD<font color=#f09048>G</font><font color=#15c015>T</font><font color=#f09048>G</font>F<font color=#80a0f0>V</font>V<font color=#15a4a4>Y</font>D<font color=#f09048>G</font>A<font color=#80a0f0>L</font><font color=#80a0f0>F</font><font color=#80a0f0>F</font>NKE-RTRN<font color=#80a0f0>I</font>V<font color=#f01505>K</font>FD<font color=#80a0f0>L</font>RT<font color=#f01505>R</font>IKSGEAIIAANYHDTSP<font color=#15a4a4>Y</font>RWGGKSD<font color=#80a0f0>I</font><font color=#c048c0>D</font><font color=#80a0f0>L</font><font color=#80a0f0>A</font><font color=#80a0f0>V</font><font color=#c048c0>D</font><font color=#c048c0>E</font>N<font color=#f09048>G</font><font color=#80a0f0>L</font><font color=#80a0f0>W</font>V<font color=#80a0f0>I</font>YATEQNNGKI<font color=#80a0f0>V</font>ISQLN<font color=#c0c000>P</font>Y<font color=#15c015>T</font><font color=#80a0f0>L</font>RI<font color=#c048c0>E</font>GT<font color=#80a0f0>W</font><font color=#c048c0>D</font><font color=#15c015>T</font>AYDKRS<font color=#80a0f0>A</font>SNA<font color=#80a0f0>F</font>MI<font color=#f08080>C</font><font color=#f09048>G</font>I<font color=#80a0f0>L</font><font color=#15a4a4>Y</font><font color=#80a0f0>V</font><font color=#80a0f0>V</font>K<font color=#15c015>S</font>------NK<font color=#80a0f0>I</font>DY<font color=#80a0f0>I</font><font color=#15a4a4>Y</font>NTDQS--KDS<font color=#80a0f0>L</font>VDVP<font color=#80a0f0>F</font><font color=#c0c000>P</font>NSYQYIAAVD<font color=#15a4a4>Y</font><font color=#15c015>N</font><font color=#c0c000>P</font><font color=#f01505>R</font><font color=#c048c0>D</font>NL<font color=#80a0f0>L</font><font color=#15a4a4>Y</font>V<font color=#80a0f0>W</font><font color=#15c015>N</font>NYHVVK<font color=#15a4a4>Y</font>S<font color=#80a0f0>L</font>DF-------</td></tr>
    <tr><td align="center"><font face="Arial">8</font></td><td><a href="https://www.rcsb.org/pdb/explore/explore.do?structureId=5afb" target=_blank>5afbA2</a></td><td align="center"><font face="Arial"> 0.31</font></td><td><font face="Arial"> 0.19</font></td><td><font face="Arial"> 5.79</font></td><td align="center"><font face="Arial"> <b>4.73</b></font></td><td align="center"><font face="Monospace"><a href="threading8.pdb.gz" download>HHsearch</a></font></td><td>&nbsp;</td><td BGCOLOR="#ECECEG" style="font-family:Monospace;font-size:13px;">--------------------------------------------------------------------------------------------------------------------------------V<font color=#c048c0>E</font>Q-----KVFL<font color=#f08080>C</font>PGL<font color=#80a0f0>L</font>K<font color=#f09048>G</font>VYQSEHLFESDHQS<font color=#f09048>G</font>A<font color=#80a0f0>W</font>C<font color=#f01505>K</font><font color=#c048c0>D</font>PLQA<font color=#15c015>S</font>D<font color=#f01505>K</font>I<font color=#15a4a4>Y</font>YMPWT<font color=#f01505>R</font>TD<font color=#15c015>T</font>LT<font color=#c048c0>E</font>YSSKDD<font color=#80a0f0>F</font>IAGRP---TTTYK<font color=#80a0f0>L</font>PHRVD<font color=#f09048>G</font><font color=#15c015>T</font><font color=#f09048>G</font>F<font color=#80a0f0>V</font>V<font color=#15a4a4>Y</font>D<font color=#f09048>G</font>A<font color=#80a0f0>L</font><font color=#80a0f0>F</font><font color=#80a0f0>F</font>NKER<font color=#15c015>T</font>R-N<font color=#80a0f0>I</font>V<font color=#f01505>K</font>FD<font color=#80a0f0>L</font>RT<font color=#f01505>R</font>IKSGEAIIANNYHDTSP<font color=#15a4a4>Y</font>RWGGKSD<font color=#80a0f0>I</font><font color=#c048c0>D</font><font color=#80a0f0>L</font><font color=#80a0f0>A</font><font color=#80a0f0>V</font><font color=#c048c0>D</font><font color=#c048c0>E</font>N<font color=#f09048>G</font><font color=#80a0f0>L</font><font color=#80a0f0>W</font>V<font color=#80a0f0>I</font>YATEQNNGKI<font color=#80a0f0>V</font>ISQLN<font color=#c0c000>P</font>Y<font color=#15c015>T</font><font color=#80a0f0>L</font>RI<font color=#c048c0>E</font>GT<font color=#80a0f0>W</font><font color=#c048c0>D</font><font color=#15c015>T</font>AYDKRS<font color=#80a0f0>A</font>SNA<font color=#80a0f0>F</font>MI<font color=#f08080>C</font><font color=#f09048>G</font>I<font color=#80a0f0>L</font><font color=#15a4a4>Y</font><font color=#80a0f0>V</font><font color=#80a0f0>V</font>K<font color=#15c015>S</font>------NK<font color=#80a0f0>I</font>DY<font color=#80a0f0>I</font><font color=#15a4a4>Y</font>NTD--Q<font color=#15c015>S</font>KDS<font color=#80a0f0>L</font>VDVP<font color=#80a0f0>F</font><font color=#c0c000>P</font>NSYQYIAAVD<font color=#15a4a4>Y</font><font color=#15c015>N</font><font color=#c0c000>P</font><font color=#f01505>R</font><font color=#c048c0>D</font>NL<font color=#80a0f0>L</font><font color=#15a4a4>Y</font>V<font color=#80a0f0>W</font><font color=#15c015>N</font>NYHVVK<font color=#15a4a4>Y</font>S<font color=#80a0f0>L</font>DF-------</td></tr>
    <tr><td align="center"><font face="Arial">9</font></td><td><a href="https://www.rcsb.org/pdb/explore/explore.do?structureId=5amo" target=_blank>5amoA</a></td><td align="center"><font face="Arial"> 0.32</font></td><td><font face="Arial"> 0.20</font></td><td><font face="Arial"> 5.93</font></td><td align="center"><font face="Arial"> <b>2.67</b></font></td><td align="center"><font face="Monospace"><a href="threading9.pdb.gz" download>PROSPECTOR2</a></font></td><td>&nbsp;</td><td BGCOLOR="#ECECEG" style="font-family:Monospace;font-size:13px;">V--------------------------------------SN<font color=#80a0f0>L</font><font color=#c048c0>E</font>ER<font color=#80a0f0>L</font>RA<font color=#f08080>C</font>M<font color=#15c015>Q</font>KL---------------------------------------------------------------------------------------ACGK<font color=#80a0f0>L</font>T<font color=#f09048>G</font><font color=#80a0f0>I</font>SDPVT<font color=#80a0f0>V</font><font color=#f01505>K</font>TSGSRF<font color=#f09048>G</font><font color=#15c015>S</font><font color=#80a0f0>W</font><font color=#80a0f0>M</font>T<font color=#c048c0>D</font>PLAPDNR<font color=#80a0f0>V</font>WYMD<font color=#f09048>G</font>YN<font color=#15c015>N</font>RF<font color=#80a0f0>V</font>R<font color=#c048c0>E</font>YKSMVD<font color=#80a0f0>F</font><font color=#80a0f0>M</font>NT---DNFTSHR<font color=#80a0f0>L</font>PHP<font color=#80a0f0>W</font>S<font color=#f09048>G</font><font color=#15c015>T</font><font color=#f09048>G</font><font color=#15c015>Q</font><font color=#80a0f0>V</font>V<font color=#15a4a4>Y</font>N<font color=#f09048>G</font>SIY<font color=#80a0f0>F</font>NK-FQ<font color=#15c015>S</font>HI<font color=#80a0f0>I</font><font color=#80a0f0>I</font>RFD<font color=#80a0f0>L</font>KTE<font color=#15c015>T</font>ILKTRS<font color=#80a0f0>L</font>D-------------YSD<font color=#80a0f0>I</font><font color=#c048c0>D</font><font color=#80a0f0>L</font>M<font color=#80a0f0>V</font><font color=#c048c0>D</font><font color=#c048c0>E</font>N<font color=#f09048>G</font><font color=#80a0f0>L</font><font color=#80a0f0>W</font><font color=#80a0f0>A</font>VYATNQNAGNI<font color=#80a0f0>V</font>IS<font color=#f01505>K</font>LD<font color=#c0c000>P</font>VS<font color=#80a0f0>L</font>QILQT<font color=#80a0f0>W</font>N<font color=#15c015>T</font>SYPKRS<font color=#80a0f0>A</font>GEA<font color=#80a0f0>F</font>II<font color=#f08080>C</font><font color=#f09048>G</font>T<font color=#80a0f0>L</font><font color=#15a4a4>Y</font><font color=#80a0f0>V</font>T--N<font color=#f09048>G</font>YS<font color=#f09048>G</font>GTKVHYA<font color=#15a4a4>Y</font>QTNAS--TYEYIDIP<font color=#80a0f0>F</font>QNKYSHI<font color=#15c015>S</font><font color=#80a0f0>M</font>LD<font color=#15a4a4>Y</font><font color=#15c015>N</font><font color=#c0c000>P</font>K<font color=#c048c0>D</font>RA<font color=#80a0f0>L</font><font color=#15a4a4>Y</font><font color=#80a0f0>A</font><font color=#80a0f0>W</font><font color=#15c015>N</font>N<font color=#f09048>G</font>H<font color=#15c015>Q</font>TL<font color=#15a4a4>Y</font>NVTL------F</td></tr>
    <tr><td align="center"><font face="Arial">10</font></td><td><a href="https://www.rcsb.org/pdb/explore/explore.do?structureId=4wxq" target=_blank>4wxqA</a></td><td align="center"><font face="Arial"> 0.31</font></td><td><font face="Arial"> 0.19</font></td><td><font face="Arial"> 5.86</font></td><td align="center"><font face="Arial"> <b>4.00</b></font></td><td align="center"><font face="Monospace"><a href="threading10.pdb.gz" download>SAM</a></font></td><td>&nbsp;</td><td BGCOLOR="#ECECEG" style="font-family:Monospace;font-size:13px;">--------------------------------------------------------------------------------------------------------------------------------------------GCGE<font color=#80a0f0>L</font>VWVGEPLTLRTITGKY<font color=#f09048>G</font>V<font color=#80a0f0>W</font><font color=#80a0f0>M</font>R<font color=#c048c0>D</font>PKPTY<font color=#c0c000>P</font>YRIDTV<font color=#f09048>G</font>TDVRQ<font color=#80a0f0>V</font>F<font color=#c048c0>E</font>YDL<font color=#80a0f0>I</font>SQ<font color=#80a0f0>F</font><font color=#80a0f0>M</font>QGYP---SKVH<font color=#80a0f0>I</font><font color=#80a0f0>L</font>PRPLES<font color=#15c015>T</font><font color=#f09048>G</font>A<font color=#80a0f0>V</font>V<font color=#15a4a4>Y</font>S<font color=#f09048>G</font>S<font color=#80a0f0>L</font>Y<font color=#80a0f0>F</font>Q-G<font color=#80a0f0>A</font>E<font color=#15c015>S</font>RTV<font color=#80a0f0>I</font>R<font color=#15a4a4>Y</font>E<font color=#80a0f0>L</font>NTE<font color=#15c015>T</font><font color=#80a0f0>V</font>KAEKEI<font color=#c0c000>P</font><font color=#f09048>G</font>AYHGQFP<font color=#15a4a4>Y</font>SWGGY<font color=#15c015>T</font>D<font color=#80a0f0>I</font><font color=#c048c0>D</font><font color=#80a0f0>L</font><font color=#80a0f0>A</font><font color=#80a0f0>V</font><font color=#c048c0>D</font><font color=#c048c0>E</font>A<font color=#f09048>G</font><font color=#80a0f0>L</font><font color=#80a0f0>W</font>V<font color=#80a0f0>I</font>Y<font color=#15c015>S</font>TDEAKGAI<font color=#80a0f0>V</font><font color=#80a0f0>L</font>S<font color=#f01505>K</font>LN<font color=#c0c000>P</font>EN<font color=#80a0f0>L</font>EL<font color=#c048c0>E</font>QT<font color=#80a0f0>W</font>E<font color=#15c015>T</font>NI<font color=#f01505>R</font>K<font color=#15c015>Q</font>SVANA<font color=#80a0f0>F</font>II<font color=#f08080>C</font><font color=#f09048>G</font>T<font color=#80a0f0>L</font><font color=#15a4a4>Y</font>T<font color=#80a0f0>V</font>S<font color=#15c015>S</font>YTSA-DATVNFA<font color=#15a4a4>Y</font><font color=#c048c0>D</font>TGTG<font color=#80a0f0>I</font>--SKTLTIP<font color=#80a0f0>F</font>KN<font color=#f01505>R</font>YKYS<font color=#15c015>S</font><font color=#80a0f0>M</font><font color=#80a0f0>I</font>D<font color=#15a4a4>Y</font><font color=#15c015>N</font><font color=#c0c000>P</font>LE<font color=#f01505>K</font>K<font color=#80a0f0>L</font>F<font color=#80a0f0>A</font><font color=#80a0f0>W</font>DNL<font color=#15c015>N</font>MVT<font color=#15a4a4>Y</font>DIKL-------</td></tr>


    <tr><td colspan="9" align="left">
        <table cellspacing="2" border="0" width="1000px" style="font-family:Arial;font-size:12px;">
        <tr><td valign="top">(a)</td><td align="justify"><i>ID1</i> is the number of template residues identical to query divided by number of aligned residues.</td></tr>
        <tr><td valign="top">(b)</td><td align="justify"><i>ID2</i> is the number of template residues identical to query divided by query sequence length.</td></tr>
        <tr><td valign="top">(c)</td><td align="justify"><i>Cov</i> is equal the number of aligned template residues divided by query sequence length.</td></tr>
        <tr><td valign="top">(d)</td><td align="justify"><i>Norm. Zscore</i> is the normalized Z-score of the threading alignments. A Normalized Z-score &gt;1 means a good alignment and is highlighted in <b>bold</b>.</td></tr>
        <tr><td valign="top">(e)</td><td align="justify"><i>Download alignment</i> lists the threading program</a> used to identify the template, and provide the 3D structure of aligned regions of threading templates (threading[1-10].pdb.gz).</td></tr>
        <tr><td valign="top">(f)</td><td align="justify">Template residues identical to query sequence are highlighted in <a href=/C-I-TASSER/color.html target=_blank>color</a>.</td></tr>
        </table>
    </table>
</ul>


<div style="background:#6599FF;width:500px;"><b><font face="Arial" 
color="#FFFFFF" size=4>&nbsp;Top 5 final models from C-I-TASSER</font></b></div>
<font size=2 face="Times New Roman"><br>
</font>
<ul>
    <table border="0" style="font-size:14px;background:#F2F2F2;">

    <td align ="left">
    <script type="text/javascript">
    jmolInitialize("/jmol/");
    jmolSetAppletColor("#000000");
    jmolApplet(500,"load model1.pdb.gz; set frank off; calculate structure rama;  select all; color group; spacefill off; wireframe off; cartoons;set ambient 40;spin off;");
    jmolButton("Reset","Reset");
    jmolCheckbox("set spin x 10; set spin y 10; spin on",
        "spin off", "Spin", false);
    jmolCheckbox("set antialiasDisplay true",
        "set antialiasDisplay false", "High quality", false);
    jmolCheckbox("color background white",
        "color background black", "White background", false);
    jmolButton("write PNGJ model.png","Save image");
    </script>
    </td>

    <td valign="top">
    <table border="0" style="font-family:Arial;font-size:13px;">
    <tr bgcolor="#DEDEDE" align="center"><td>Click<br>to view</td>
    <td>Rank<sup>a</sup></td><td>Download</td><td>C-score<sup>b</sup></td>
    <td>Estimated TM-score<sup>c</sup></td><td>Estimated RMSD<sup>c</sup></td></tr>

    <tr align="center"><td><script type="text/javascript">
    jmolRadio("load model1.pdb.gz; set frank off; calculate structure rama;  select all; color group; spacefill off; wireframe off; cartoons;set ambient 40;","",true);
    </script></td>
    <td>1</td>
    <td><a href="model1.pdb.gz" download>model1.pdb.gz</a></td>
    <td>1.31</td>
    <td>0.89&plusmn;0.06</td>
    <td>4.1&plusmn;2.8&Aring;</td></tr>

    
    <tr align="center"><td><script type="text/javascript">
    jmolRadio("load model2.pdb.gz; set frank off; calculate structure rama;  select all; color group; spacefill off; wireframe off; cartoons;set ambient 40;","",false);
    </script></td>
    <td>2</td>
    <td><a href="model2.pdb.gz" download>model2.pdb.gz</a></td>
    <td>0.18</td></tr>

    <tr align="center"><td><script type="text/javascript">
    jmolRadio("load model3.pdb.gz; set frank off; calculate structure rama;  select all; color group; spacefill off; wireframe off; cartoons;set ambient 40;","",false);
    </script></td>
    <td>3</td>
    <td><a href="model3.pdb.gz" download>model3.pdb.gz</a></td>
    <td>0.18</td></tr>

    <tr align="center"><td><script type="text/javascript">
    jmolRadio("load model4.pdb.gz; set frank off; calculate structure rama;  select all; color group; spacefill off; wireframe off; cartoons;set ambient 40;","",false);
    </script></td>
    <td>4</td>
    <td><a href="model4.pdb.gz" download>model4.pdb.gz</a></td>
    <td>-2.17</td></tr>

    <tr align="center"><td><script type="text/javascript">
    jmolRadio("load model5.pdb.gz; set frank off; calculate structure rama;  select all; color group; spacefill off; wireframe off; cartoons;set ambient 40;","",false);
    </script></td>
    <td>5</td>
    <td><a href="model5.pdb.gz" download>model5.pdb.gz</a></td>
    <td>-2.89</td></tr>

    </table>
    <table cellspacing="2" border="0" width="600px" style="font-family:Arial;font-size:12px;"><tr></tr><tr></tr>
    <tr><td valign="top">(a)</td><td>
    C-I-TASSER simulations generate a large ensemble of structural
    conformations, i.e. decoys. These decoys are clustered by
    <a href=/SPICKER target=_blank>SPICKER</a> based on pairwise structure similarity to
    report up to five final models from the five largest clusters. Models are 
    ranked in descending order of cluster size. If the simulations converge
    well, it is possible to have less than 5 models generated, which is
    usually an indication of good model quality.
    </td></tr>
    <tr><td valign="top">(b)</td><td>
    The model confidence is quantitatified by C-score, calculated based on
    significance of threading template alignments and convergence of
    C-I-TASSER simulations. C-score is typically in the range of [-5, 2],
    with higher C-score signifies higher model confidence.</td></tr>
    <tr><td valign="top">(c)</td><td>
    Model <a href=/TM-score target=_blank>TM-score</a> and RMSD are estimated based on
    C-score and protein length for the first model.
    </td></tr>
    </table>
    </td>
    </table>
</ul>


<div style="background:#6599FF;width:500px;"><b><font face="Arial" 
color="#FFFFFF" size=4>&nbsp;Proteins with similar structure</font></b></div>
<ul>
    <table border="0" style="background:#F2F2F2";align="left">
        <tr><td align ="center"><script type="text/javascript">
        jmolInitialize("/jmol/");
        jmolSetAppletColor("#000000");
        jmolApplet(500, "load model1_5afbA.pdb.gz;set frank off;select all;spacefill off;wireframe off;select :T;trace 0.35;color trace mediumpurple;select :Q;cartoons;color group;set ambient 40;");
        jmolButton("Reset","Reset");
        jmolCheckbox("set spin x 10; set spin y 10; spin on",
            "spin off", "Spin", false);
        jmolCheckbox("set antialiasDisplay true",
            "set antialiasDisplay false", "High quality", false);
        jmolCheckbox("color background white",
            "color background black", "White background", false);
        jmolButton("write PNGJ model.png","Save image");
        </script></td>
        <td valign="top"><b>Top 10 structural analogs in PDB (as identified by
        <a href="https://zhanglab.ccmb.med.umich.edu/TM-align" target="_blank">
        TM-align</a>)</b><br/><br/>
        <table border="0" style="font-family:Arial;font-size:13px;">
        <tr><th>Click<br>to view</th><th>Rank</th><th>PDB<br/>Hit</th><th>TM-score</th><th>RMSD<sup>a</sup></th><th>IDEN<sup>a</sup></th><th>Cov.</th><th>Download<br/>Alignment</th></tr>
        
        <tr><td align="center"><script type="text/javascript">jmolRadio("load model1_5afbA.pdb.gz; set frank off; select all; spacefill off; wireframe off; select :T; trace 0.35; color trace mediumpurple; select :Q; cartoons; color group; set ambient 40; set spin x 5; set spin y 5; spin off;", "", false);</script></td><td valign="middle" align="center">1</td><td valign="middle"><a href="https://www.rcsb.org/pdb/explore/explore.do?structureId=5afb" target="_blank"><font face="Monospace">5afbA</font></a></td><td align="center">0.65</td><td valign="middle" align="center">2.44</td><td valign="middle" align="center">0.263</td><td valign="middle">0.692</td><td><a href="model1_5afbA.pdb.gz" download>model1_5afbA.pdb.gz</a></td></tr>
        <tr><td align="center"><script type="text/javascript">jmolRadio("load model1_4wxqA.pdb.gz; set frank off; select all; spacefill off; wireframe off; select :T; trace 0.35; color trace mediumpurple; select :Q; cartoons; color group; set ambient 40; set spin x 5; set spin y 5; spin off;", "", false);</script></td><td valign="middle" align="center">2</td><td valign="middle"><a href="https://www.rcsb.org/pdb/explore/explore.do?structureId=4wxq" target="_blank"><font face="Monospace">4wxqA</font></a></td><td align="center">0.61</td><td valign="middle" align="center">1.16</td><td valign="middle" align="center">0.297</td><td valign="middle">0.619</td><td><a href="model1_4wxqA.pdb.gz" download>model1_4wxqA.pdb.gz</a></td></tr>
        <tr><td align="center"><script type="text/javascript">jmolRadio("load model1_5amoA.pdb.gz; set frank off; select all; spacefill off; wireframe off; select :T; trace 0.35; color trace mediumpurple; select :Q; cartoons; color group; set ambient 40; set spin x 5; set spin y 5; spin off;", "", false);</script></td><td valign="middle" align="center">3</td><td valign="middle"><a href="https://www.rcsb.org/pdb/explore/explore.do?structureId=5amo" target="_blank"><font face="Monospace">5amoA</font></a></td><td align="center">0.60</td><td valign="middle" align="center">1.44</td><td valign="middle" align="center">0.296</td><td valign="middle">0.622</td><td><a href="model1_5amoA.pdb.gz" download>model1_5amoA.pdb.gz</a></td></tr>
        <tr><td align="center"><script type="text/javascript">jmolRadio("load model1_4d77A.pdb.gz; set frank off; select all; spacefill off; wireframe off; select :T; trace 0.35; color trace mediumpurple; select :Q; cartoons; color group; set ambient 40; set spin x 5; set spin y 5; spin off;", "", false);</script></td><td valign="middle" align="center">4</td><td valign="middle"><a href="https://www.rcsb.org/pdb/explore/explore.do?structureId=4d77" target="_blank"><font face="Monospace">4d77A</font></a></td><td align="center">0.58</td><td valign="middle" align="center">1.48</td><td valign="middle" align="center">0.244</td><td valign="middle">0.602</td><td><a href="model1_4d77A.pdb.gz" download>model1_4d77A.pdb.gz</a></td></tr>
        <tr><td align="center"><script type="text/javascript">jmolRadio("load model1_5wbiA.pdb.gz; set frank off; select all; spacefill off; wireframe off; select :T; trace 0.35; color trace mediumpurple; select :Q; cartoons; color group; set ambient 40; set spin x 5; set spin y 5; spin off;", "", false);</script></td><td valign="middle" align="center">5</td><td valign="middle"><a href="https://www.rcsb.org/pdb/explore/explore.do?structureId=5wbi" target="_blank"><font face="Monospace">5wbiA</font></a></td><td align="center">0.51</td><td valign="middle" align="center">5.19</td><td valign="middle" align="center">0.071</td><td valign="middle">0.699</td><td><a href="model1_5wbiA.pdb.gz" download>model1_5wbiA.pdb.gz</a></td></tr>
        <tr><td align="center"><script type="text/javascript">jmolRadio("load model1_3bykA.pdb.gz; set frank off; select all; spacefill off; wireframe off; select :T; trace 0.35; color trace mediumpurple; select :Q; cartoons; color group; set ambient 40; set spin x 5; set spin y 5; spin off;", "", false);</script></td><td valign="middle" align="center">6</td><td valign="middle"><a href="https://www.rcsb.org/pdb/explore/explore.do?structureId=3byk" target="_blank"><font face="Monospace">3bykA</font></a></td><td align="center">0.49</td><td valign="middle" align="center">4.16</td><td valign="middle" align="center">0.071</td><td valign="middle">0.617</td><td><a href="model1_3bykA.pdb.gz" download>model1_3bykA.pdb.gz</a></td></tr>
        <tr><td align="center"><script type="text/javascript">jmolRadio("load model1_6bcuW.pdb.gz; set frank off; select all; spacefill off; wireframe off; select :T; trace 0.35; color trace mediumpurple; select :Q; cartoons; color group; set ambient 40; set spin x 5; set spin y 5; spin off;", "", false);</script></td><td valign="middle" align="center">7</td><td valign="middle"><a href="https://www.rcsb.org/pdb/explore/explore.do?structureId=6bcu" target="_blank"><font face="Monospace">6bcuW</font></a></td><td align="center">0.49</td><td valign="middle" align="center">5.61</td><td valign="middle" align="center">0.067</td><td valign="middle">0.697</td><td><a href="model1_6bcuW.pdb.gz" download>model1_6bcuW.pdb.gz</a></td></tr>
        <tr><td align="center"><script type="text/javascript">jmolRadio("load model1_3vsrA.pdb.gz; set frank off; select all; spacefill off; wireframe off; select :T; trace 0.35; color trace mediumpurple; select :Q; cartoons; color group; set ambient 40; set spin x 5; set spin y 5; spin off;", "", false);</script></td><td valign="middle" align="center">8</td><td valign="middle"><a href="https://www.rcsb.org/pdb/explore/explore.do?structureId=3vsr" target="_blank"><font face="Monospace">3vsrA</font></a></td><td align="center">0.49</td><td valign="middle" align="center">4.46</td><td valign="middle" align="center">0.081</td><td valign="middle">0.619</td><td><a href="model1_3vsrA.pdb.gz" download>model1_3vsrA.pdb.gz</a></td></tr>
        <tr><td align="center"><script type="text/javascript">jmolRadio("load model1_2yfrA.pdb.gz; set frank off; select all; spacefill off; wireframe off; select :T; trace 0.35; color trace mediumpurple; select :Q; cartoons; color group; set ambient 40; set spin x 5; set spin y 5; spin off;", "", false);</script></td><td valign="middle" align="center">9</td><td valign="middle"><a href="https://www.rcsb.org/pdb/explore/explore.do?structureId=2yfr" target="_blank"><font face="Monospace">2yfrA</font></a></td><td align="center">0.43</td><td valign="middle" align="center">4.57</td><td valign="middle" align="center">0.040</td><td valign="middle">0.550</td><td><a href="model1_2yfrA.pdb.gz" download>model1_2yfrA.pdb.gz</a></td></tr>
        <tr><td align="center"><script type="text/javascript">jmolRadio("load model1_3lrvA2.pdb.gz; set frank off; select all; spacefill off; wireframe off; select :T; trace 0.35; color trace mediumpurple; select :Q; cartoons; color group; set ambient 40; set spin x 5; set spin y 5; spin off;", "", false);</script></td><td valign="middle" align="center">10</td><td valign="middle"><a href="https://www.rcsb.org/pdb/explore/explore.do?structureId=3lrv" target="_blank"><font face="Monospace">3lrvA2</font></a></td><td align="center">0.33</td><td valign="middle" align="center">4.35</td><td valign="middle" align="center">0.090</td><td valign="middle">0.430</td><td><a href="model1_3lrvA2.pdb.gz" download>model1_3lrvA2.pdb.gz</a></td></tr>
        </table><br/>

        <table border="0" cellspacing="2" style="font-family:Arial;font-size:12px;background:#F2F2F2;">
            <tr><td valign="top">(a)</td><td align="justify">Query structure is shown in cartoon, while the structural analog is displayed using backbone trace.</td></tr>
            <tr><td valign="top">(b)</td><td align="justify">Ranking of proteins is based on TM-score of the structural alignment between the query structure and known structures in the PDB library.</td></tr>
            <tr><td valign="top">(c)</td><td align="justify">RMSD<sup>a</sup> is the RMSD between residues that are structurally aligned by TM-align.</td></tr>
            <tr><td valign="top">(d)</td><td align="justify">IDEN<sup>a</sup> is the percentage sequence identity in the structurally aligned region.</td></tr>
            <tr><td valign="top">(e)</td><td align="justify">Cov. represents the coverage of the alignment by TM-align and is equal to the number of structurally aligned residues divided by length of the query protein.</td></tr>
        </table>
    </td></tr>
    </table>
</ul>

<div style="background:#6599FF;width:500px;"><b><font face="Arial" 
color="#FFFFFF" size=4>&nbsp;Predicted Gene Ontology (GO) Terms</font></b></div>
<ul>
<table border="0" style="background:#F2F2F2";align="left"><tr><td>

<table border="0" style="background:#F2F2F2";align="left"><tr>
<td align ="center" style="background:#FFFFFF">
    <a href="GOsearchresult_final_MF.svg" target="_blank">
    <img width=500 height=500 border="1" style="border-color:black" alt="GOsearchresult_final_MF.svg" src="GOsearchresult_final_MF.svg">
    </a>
</td>
<td valign="top" align="left">
    <table border="0">
        <tr><td><b>Molecular Function (MF)</b><br></tr></td>
    </table>
    <table border="0" style="font-family:Arial;font-size:13px;">
        <tr><td align="left"><b>GO term</b></td><td align="center"><b>Cscore<sup>GO</sup></b></td><td><b>Name</b></td></tr>
        
        <tr><td align="left"><a href="http://amigo.geneontology.org/amigo/term/GO:0043167" target="_blank">GO:0043167</a></td><td align="center">0.47</td><td>ion binding</td></tr>
        <tr><td align="left"><a href="http://amigo.geneontology.org/amigo/term/GO:0046872" target="_blank">GO:0046872</a></td><td align="center">0.45</td><td>metal ion binding</td></tr>
        <tr><td align="left"><a href="http://amigo.geneontology.org/amigo/term/GO:0004872" target="_blank">GO:0004872</a></td><td align="center">0.45</td><td>receptor activity</td></tr>
        <tr><td align="left"><a href="http://amigo.geneontology.org/amigo/term/GO:0042277" target="_blank">GO:0042277</a></td><td align="center">0.44</td><td>peptide binding</td></tr>
        <tr><td align="left"><a href="http://amigo.geneontology.org/amigo/term/GO:0004888" target="_blank">GO:0004888</a></td><td align="center">0.42</td><td>transmembrane signaling receptor activity</td></tr>
        <tr><td align="left"><a href="http://amigo.geneontology.org/amigo/term/GO:0004930" target="_blank">GO:0004930</a></td><td align="center">0.38</td><td>G-protein coupled receptor activity</td></tr>
        <tr><td align="left"><a href="http://amigo.geneontology.org/amigo/term/GO:0005509" target="_blank">GO:0005509</a></td><td align="center">0.37</td><td>calcium ion binding</td></tr>
        <tr><td align="left"><a href="http://amigo.geneontology.org/amigo/term/GO:0008092" target="_blank">GO:0008092</a></td><td align="center">0.36</td><td>cytoskeletal protein binding</td></tr>
        <tr><td align="left"><a href="http://amigo.geneontology.org/amigo/term/GO:0005102" target="_blank">GO:0005102</a></td><td align="center">0.33</td><td>receptor binding</td></tr>
        <tr><td align="left"><a href="http://amigo.geneontology.org/amigo/term/GO:0001664" target="_blank">GO:0001664</a></td><td align="center">0.32</td><td>G-protein coupled receptor binding</td></tr>
    </table><br>
    <table border="0" style="font-family:Arial;font-size:13px;">
        <tr><td>Download <a href="GOsearchresult_final_MF.csv" download>full result</a> of the above consensus prediction.</tr></td>
    </table><br>
    <table cellspacing="2" border="0" style="font-family:Arial;font-size:12px;">
        <tr><td valign="top"></td><td align="justify"><b>Click the graph to show a high resolution version.</b></td></tr>
        <tr><td valign="top">(a)</td><td align="justify">Cscore<sup>GO</sup> is the confidence score of predicted GO terms. Cscore<sup>GO</sup> values range in between [0-1]; where a higher value indicates a better confidence in predicting the function using the template.</td></tr>
        <tr><td valign="top">(b)</td><td align="justify">The graph shows the predicted terms within the Gene Ontology hierachy for Molecular Function. Confidently predicted terms are color coded by Cscore<sup>GO</sup>:</td></tr>
        <tr><td valign="top"></td><td align="justify"><table cellspacing="2" border="0" style="border-color:black;font-family:Arial;font-size:12px;"><tr><td style="background-color:magenta;">[0.4,0.5)</td><td style="background-color:cyan;">[0.5,0.6)</td><td style="background-color:green;">[0.6,0.7)</td><td style="background-color:yellow;">[0.7,0.8)</td><td style="background-color:orange;">[0.8,0.9)</td><td style="background-color:red;">[0.9,1.0]</td></tr></table></td></tr>
     </table>
</td>
</tr>
</table>


<table border="0" style="background:#F2F2F2";align="left"><tr>
<td align ="center" style="background:#FFFFFF">
    <a href="GOsearchresult_final_BP.svg" target="_blank">
    <img width=500 height=500 border="1" style="border-color:black" alt="GOsearchresult_final_BP.svg" src="GOsearchresult_final_BP.svg">
    </a>
</td>
<td valign="top" align="left">
    <table border="0">
        <tr><td><b>Biological Process (BP)</b><br></tr></td>
    </table>
    <table border="0" style="font-family:Arial;font-size:13px;">
        <tr><td align="left"><b>GO term</b></td><td align="center"><b>Cscore<sup>GO</sup></b></td><td><b>Name</b></td></tr>
        
        <tr><td align="left"><a href="http://amigo.geneontology.org/amigo/term/GO:0009987" target="_blank">GO:0009987</a></td><td align="center">0.96</td><td>cellular process</td></tr>
        <tr><td align="left"><a href="http://amigo.geneontology.org/amigo/term/GO:0044699" target="_blank">GO:0044699</a></td><td align="center">0.94</td><td>single-organism process</td></tr>
        <tr><td align="left"><a href="http://amigo.geneontology.org/amigo/term/GO:0044763" target="_blank">GO:0044763</a></td><td align="center">0.90</td><td>single-organism cellular process</td></tr>
        <tr><td align="left"><a href="http://amigo.geneontology.org/amigo/term/GO:0065007" target="_blank">GO:0065007</a></td><td align="center">0.87</td><td>biological regulation</td></tr>
        <tr><td align="left"><a href="http://amigo.geneontology.org/amigo/term/GO:0050789" target="_blank">GO:0050789</a></td><td align="center">0.86</td><td>regulation of biological process</td></tr>
        <tr><td align="left"><a href="http://amigo.geneontology.org/amigo/term/GO:0050794" target="_blank">GO:0050794</a></td><td align="center">0.85</td><td>regulation of cellular process</td></tr>
        <tr><td align="left"><a href="http://amigo.geneontology.org/amigo/term/GO:0016043" target="_blank">GO:0016043</a></td><td align="center">0.83</td><td>cellular component organization</td></tr>
        <tr><td align="left"><a href="http://amigo.geneontology.org/amigo/term/GO:0044767" target="_blank">GO:0044767</a></td><td align="center">0.82</td><td>single-organism developmental process</td></tr>
        <tr><td align="left"><a href="http://amigo.geneontology.org/amigo/term/GO:0048518" target="_blank">GO:0048518</a></td><td align="center">0.79</td><td>positive regulation of biological process</td></tr>
        <tr><td align="left"><a href="http://amigo.geneontology.org/amigo/term/GO:0050793" target="_blank">GO:0050793</a></td><td align="center">0.77</td><td>regulation of developmental process</td></tr>
        <tr><td align="left"><a href="http://amigo.geneontology.org/amigo/term/GO:0051128" target="_blank">GO:0051128</a></td><td align="center">0.74</td><td>regulation of cellular component organization</td></tr>
        <tr><td align="left"><a href="http://amigo.geneontology.org/amigo/term/GO:0019222" target="_blank">GO:0019222</a></td><td align="center">0.73</td><td>regulation of metabolic process</td></tr>
        <tr><td align="left"><a href="http://amigo.geneontology.org/amigo/term/GO:0060255" target="_blank">GO:0060255</a></td><td align="center">0.72</td><td>regulation of macromolecule metabolic process</td></tr>
        <tr><td align="left"><a href="http://amigo.geneontology.org/amigo/term/GO:0048519" target="_blank">GO:0048519</a></td><td align="center">0.72</td><td>negative regulation of biological process</td></tr>
        <tr><td align="left"><a href="http://amigo.geneontology.org/amigo/term/GO:0048522" target="_blank">GO:0048522</a></td><td align="center">0.71</td><td>positive regulation of cellular process</td></tr>
        <tr><td align="left"><a href="http://amigo.geneontology.org/amigo/term/GO:0045595" target="_blank">GO:0045595</a></td><td align="center">0.71</td><td>regulation of cell differentiation</td></tr>
        <tr><td align="left"><a href="http://amigo.geneontology.org/amigo/term/GO:0022604" target="_blank">GO:0022604</a></td><td align="center">0.69</td><td>regulation of cell morphogenesis</td></tr>
        <tr><td align="left"><a href="http://amigo.geneontology.org/amigo/term/GO:0051239" target="_blank">GO:0051239</a></td><td align="center">0.68</td><td>regulation of multicellular organismal process</td></tr>
        <tr><td align="left"><a href="http://amigo.geneontology.org/amigo/term/GO:0051094" target="_blank">GO:0051094</a></td><td align="center">0.68</td><td>positive regulation of developmental process</td></tr>
        <tr><td align="left"><a href="http://amigo.geneontology.org/amigo/term/GO:0010769" target="_blank">GO:0010769</a></td><td align="center">0.68</td><td>regulation of cell morphogenesis involved in differentiation</td></tr>
        <tr><td align="left"><a href="http://amigo.geneontology.org/amigo/term/GO:2000026" target="_blank">GO:2000026</a></td><td align="center">0.67</td><td>regulation of multicellular organismal development</td></tr>
        <tr><td align="left"><a href="http://amigo.geneontology.org/amigo/term/GO:0009653" target="_blank">GO:0009653</a></td><td align="center">0.67</td><td>anatomical structure morphogenesis</td></tr>
        <tr><td align="left"><a href="http://amigo.geneontology.org/amigo/term/GO:0048869" target="_blank">GO:0048869</a></td><td align="center">0.65</td><td>cellular developmental process</td></tr>
        <tr><td align="left"><a href="http://amigo.geneontology.org/amigo/term/GO:0010468" target="_blank">GO:0010468</a></td><td align="center">0.65</td><td>regulation of gene expression</td></tr>
        <tr><td align="left"><a href="http://amigo.geneontology.org/amigo/term/GO:0009893" target="_blank">GO:0009893</a></td><td align="center">0.65</td><td>positive regulation of metabolic process</td></tr>
        <tr><td align="left"><a href="http://amigo.geneontology.org/amigo/term/GO:0051130" target="_blank">GO:0051130</a></td><td align="center">0.64</td><td>positive regulation of cellular component organization</td></tr>
        <tr><td align="left"><a href="http://amigo.geneontology.org/amigo/term/GO:0010604" target="_blank">GO:0010604</a></td><td align="center">0.64</td><td>positive regulation of macromolecule metabolic process</td></tr>
        <tr><td align="left"><a href="http://amigo.geneontology.org/amigo/term/GO:0048646" target="_blank">GO:0048646</a></td><td align="center">0.62</td><td>anatomical structure formation involved in morphogenesis</td></tr>
        <tr><td align="left"><a href="http://amigo.geneontology.org/amigo/term/GO:0010605" target="_blank">GO:0010605</a></td><td align="center">0.62</td><td>negative regulation of macromolecule metabolic process</td></tr>
        <tr><td align="left"><a href="http://amigo.geneontology.org/amigo/term/GO:0031323" target="_blank">GO:0031323</a></td><td align="center">0.61</td><td>regulation of cellular metabolic process</td></tr>
        <tr><td align="left"><a href="http://amigo.geneontology.org/amigo/term/GO:0010717" target="_blank">GO:0010717</a></td><td align="center">0.61</td><td>regulation of epithelial to mesenchymal transition</td></tr>
        <tr><td align="left"><a href="http://amigo.geneontology.org/amigo/term/GO:0003188" target="_blank">GO:0003188</a></td><td align="center">0.61</td><td>heart valve formation</td></tr>
        <tr><td align="left"><a href="http://amigo.geneontology.org/amigo/term/GO:0080090" target="_blank">GO:0080090</a></td><td align="center">0.60</td><td>regulation of primary metabolic process</td></tr>
        <tr><td align="left"><a href="http://amigo.geneontology.org/amigo/term/GO:0045597" target="_blank">GO:0045597</a></td><td align="center">0.59</td><td>positive regulation of cell differentiation</td></tr>
        <tr><td align="left"><a href="http://amigo.geneontology.org/amigo/term/GO:0008152" target="_blank">GO:0008152</a></td><td align="center">0.59</td><td>metabolic process</td></tr>
        <tr><td align="left"><a href="http://amigo.geneontology.org/amigo/term/GO:0048523" target="_blank">GO:0048523</a></td><td align="center">0.58</td><td>negative regulation of cellular process</td></tr>
        <tr><td align="left"><a href="http://amigo.geneontology.org/amigo/term/GO:0051246" target="_blank">GO:0051246</a></td><td align="center">0.57</td><td>regulation of protein metabolic process</td></tr>
        <tr><td align="left"><a href="http://amigo.geneontology.org/amigo/term/GO:0071704" target="_blank">GO:0071704</a></td><td align="center">0.56</td><td>organic substance metabolic process</td></tr>
        <tr><td align="left"><a href="http://amigo.geneontology.org/amigo/term/GO:0010770" target="_blank">GO:0010770</a></td><td align="center">0.56</td><td>positive regulation of cell morphogenesis involved in differentiation</td></tr>
        <tr><td align="left"><a href="http://amigo.geneontology.org/amigo/term/GO:0051240" target="_blank">GO:0051240</a></td><td align="center">0.55</td><td>positive regulation of multicellular organismal process</td></tr>
        <tr><td align="left"><a href="http://amigo.geneontology.org/amigo/term/GO:0044238" target="_blank">GO:0044238</a></td><td align="center">0.55</td><td>primary metabolic process</td></tr>
        <tr><td align="left"><a href="http://amigo.geneontology.org/amigo/term/GO:0032879" target="_blank">GO:0032879</a></td><td align="center">0.55</td><td>regulation of localization</td></tr>
        <tr><td align="left"><a href="http://amigo.geneontology.org/amigo/term/GO:0051960" target="_blank">GO:0051960</a></td><td align="center">0.54</td><td>regulation of nervous system development</td></tr>
        <tr><td align="left"><a href="http://amigo.geneontology.org/amigo/term/GO:0044237" target="_blank">GO:0044237</a></td><td align="center">0.54</td><td>cellular metabolic process</td></tr>
        <tr><td align="left"><a href="http://amigo.geneontology.org/amigo/term/GO:0030334" target="_blank">GO:0030334</a></td><td align="center">0.54</td><td>regulation of cell migration</td></tr>
        <tr><td align="left"><a href="http://amigo.geneontology.org/amigo/term/GO:0061024" target="_blank">GO:0061024</a></td><td align="center">0.53</td><td>membrane organization</td></tr>
        <tr><td align="left"><a href="http://amigo.geneontology.org/amigo/term/GO:0048856" target="_blank">GO:0048856</a></td><td align="center">0.53</td><td>anatomical structure development</td></tr>
        <tr><td align="left"><a href="http://amigo.geneontology.org/amigo/term/GO:0043170" target="_blank">GO:0043170</a></td><td align="center">0.53</td><td>macromolecule metabolic process</td></tr>
        <tr><td align="left"><a href="http://amigo.geneontology.org/amigo/term/GO:0022607" target="_blank">GO:0022607</a></td><td align="center">0.53</td><td>cellular component assembly</td></tr>
        <tr><td align="left"><a href="http://amigo.geneontology.org/amigo/term/GO:0051171" target="_blank">GO:0051171</a></td><td align="center">0.52</td><td>regulation of nitrogen compound metabolic process</td></tr>
        <tr><td align="left"><a href="http://amigo.geneontology.org/amigo/term/GO:0045162" target="_blank">GO:0045162</a></td><td align="center">0.52</td><td>clustering of voltage-gated sodium channels</td></tr>
        <tr><td align="left"><a href="http://amigo.geneontology.org/amigo/term/GO:0034641" target="_blank">GO:0034641</a></td><td align="center">0.51</td><td>cellular nitrogen compound metabolic process</td></tr>
        <tr><td align="left"><a href="http://amigo.geneontology.org/amigo/term/GO:0061387" target="_blank">GO:0061387</a></td><td align="center">0.50</td><td>regulation of extent of cell growth</td></tr>
        <tr><td align="left"><a href="http://amigo.geneontology.org/amigo/term/GO:0050770" target="_blank">GO:0050770</a></td><td align="center">0.50</td><td>regulation of axonogenesis</td></tr>
        <tr><td align="left"><a href="http://amigo.geneontology.org/amigo/term/GO:0048638" target="_blank">GO:0048638</a></td><td align="center">0.50</td><td>regulation of developmental growth</td></tr>
        <tr><td align="left"><a href="http://amigo.geneontology.org/amigo/term/GO:0043933" target="_blank">GO:0043933</a></td><td align="center">0.50</td><td>macromolecular complex subunit organization</td></tr>
        <tr><td align="left"><a href="http://amigo.geneontology.org/amigo/term/GO:0019538" target="_blank">GO:0019538</a></td><td align="center">0.50</td><td>protein metabolic process</td></tr>
    </table><br>
    <table border="0" style="font-family:Arial;font-size:13px;">
        <tr><td>Download <a href="GOsearchresult_final_BP.csv" download>full result</a> of the above consensus prediction.</tr></td>
    </table><br>
    <table cellspacing="2" border="0" style="font-family:Arial;font-size:12px;">
        <tr><td valign="top"></td><td align="justify"><b>Click the graph to show a high resolution version.</b></td></tr>
        <tr><td valign="top">(a)</td><td align="justify">Cscore<sup>GO</sup> is the confidence score of predicted GO terms. Cscore<sup>GO</sup> values range in between [0-1]; where a higher value indicates a better confidence in predicting the function using the template.</td></tr>
        <tr><td valign="top">(b)</td><td align="justify">The graph shows the predicted terms within the Gene Ontology hierachy for Biological Process. Confidently predicted terms are color coded by Cscore<sup>GO</sup>:</td></tr>
        <tr><td valign="top"></td><td align="justify"><table cellspacing="2" border="0" style="border-color:black;font-family:Arial;font-size:12px;"><tr><td style="background-color:magenta;">[0.4,0.5)</td><td style="background-color:cyan;">[0.5,0.6)</td><td style="background-color:green;">[0.6,0.7)</td><td style="background-color:yellow;">[0.7,0.8)</td><td style="background-color:orange;">[0.8,0.9)</td><td style="background-color:red;">[0.9,1.0]</td></tr></table></td></tr>
     </table>
</td>
</tr>
</table>


<table border="0" style="background:#F2F2F2";align="left"><tr>
<td align ="center" style="background:#FFFFFF">
    <a href="GOsearchresult_final_CC.svg" target="_blank">
    <img width=500 height=500 border="1" style="border-color:black" alt="GOsearchresult_final_CC.svg" src="GOsearchresult_final_CC.svg">
    </a>
</td>
<td valign="top" align="left">
    <table border="0">
        <tr><td><b>Cellular Component (CC)</b><br></tr></td>
    </table>
    <table border="0" style="font-family:Arial;font-size:13px;">
        <tr><td align="left"><b>GO term</b></td><td align="center"><b>Cscore<sup>GO</sup></b></td><td><b>Name</b></td></tr>
        
        <tr><td align="left"><a href="http://amigo.geneontology.org/amigo/term/GO:0044464" target="_blank">GO:0044464</a></td><td align="center">0.97</td><td>cell part</td></tr>
        <tr><td align="left"><a href="http://amigo.geneontology.org/amigo/term/GO:0044424" target="_blank">GO:0044424</a></td><td align="center">0.90</td><td>intracellular part</td></tr>
        <tr><td align="left"><a href="http://amigo.geneontology.org/amigo/term/GO:0044444" target="_blank">GO:0044444</a></td><td align="center">0.76</td><td>cytoplasmic part</td></tr>
        <tr><td align="left"><a href="http://amigo.geneontology.org/amigo/term/GO:0043226" target="_blank">GO:0043226</a></td><td align="center">0.76</td><td>organelle</td></tr>
        <tr><td align="left"><a href="http://amigo.geneontology.org/amigo/term/GO:0043229" target="_blank">GO:0043229</a></td><td align="center">0.72</td><td>intracellular organelle</td></tr>
        <tr><td align="left"><a href="http://amigo.geneontology.org/amigo/term/GO:0016020" target="_blank">GO:0016020</a></td><td align="center">0.72</td><td>membrane</td></tr>
        <tr><td align="left"><a href="http://amigo.geneontology.org/amigo/term/GO:0043227" target="_blank">GO:0043227</a></td><td align="center">0.70</td><td>membrane-bounded organelle</td></tr>
        <tr><td align="left"><a href="http://amigo.geneontology.org/amigo/term/GO:0043231" target="_blank">GO:0043231</a></td><td align="center">0.64</td><td>intracellular membrane-bounded organelle</td></tr>
        <tr><td align="left"><a href="http://amigo.geneontology.org/amigo/term/GO:0044422" target="_blank">GO:0044422</a></td><td align="center">0.62</td><td>organelle part</td></tr>
        <tr><td align="left"><a href="http://amigo.geneontology.org/amigo/term/GO:0044446" target="_blank">GO:0044446</a></td><td align="center">0.60</td><td>intracellular organelle part</td></tr>
        <tr><td align="left"><a href="http://amigo.geneontology.org/amigo/term/GO:0044421" target="_blank">GO:0044421</a></td><td align="center">0.53</td><td>extracellular region part</td></tr>
        <tr><td align="left"><a href="http://amigo.geneontology.org/amigo/term/GO:0097458" target="_blank">GO:0097458</a></td><td align="center">0.50</td><td>neuron part</td></tr>
    </table><br>
    <table border="0" style="font-family:Arial;font-size:13px;">
        <tr><td>Download <a href="GOsearchresult_final_CC.csv" download>full result</a> of the above consensus prediction.</tr></td>
    </table><br>
    <table cellspacing="2" border="0" style="font-family:Arial;font-size:12px;">
        <tr><td valign="top"></td><td align="justify"><b>Click the graph to show a high resolution version.</b></td></tr>
        <tr><td valign="top">(a)</td><td align="justify">Cscore<sup>GO</sup> is the confidence score of predicted GO terms. Cscore<sup>GO</sup> values range in between [0-1]; where a higher value indicates a better confidence in predicting the function using the template.</td></tr>
        <tr><td valign="top">(b)</td><td align="justify">The graph shows the predicted terms within the Gene Ontology hierachy for Cellular Component. Confidently predicted terms are color coded by Cscore<sup>GO</sup>:</td></tr>
        <tr><td valign="top"></td><td align="justify"><table cellspacing="2" border="0" style="border-color:black;font-family:Arial;font-size:12px;"><tr><td style="background-color:magenta;">[0.4,0.5)</td><td style="background-color:cyan;">[0.5,0.6)</td><td style="background-color:green;">[0.6,0.7)</td><td style="background-color:yellow;">[0.7,0.8)</td><td style="background-color:orange;">[0.8,0.9)</td><td style="background-color:red;">[0.9,1.0]</td></tr></table></td></tr>
     </table>
</td>
</tr>
</table>

</td></tr>
</table>
</ul>

<div style="background:#6599FF;width:500px;"><b><font face="Arial" 
color="#FFFFFF" size=4>&nbsp;Predicted Enzyme Commission (EC) Numbers</font></b></div>
<ul>
    <table border="0" style="background:#F2F2F2";align="left">
        <tr><td align ="center"><script type="text/javascript">
        jmolInitialize("/jmol/");
        jmolSetAppletColor("#000000");
        jmolApplet(500, "load model1.pdb.gz;set frank off;select all;color group;spacefill off;wireframe off;cartoons;;set spin x 10; set spin y 10; spin off;");
        jmolButton("Reset","Reset");
        jmolCheckbox("set spin x 10; set spin y 10; spin on",
            "spin off", "Spin", false);
        jmolCheckbox("set antialiasDisplay true",
            "set antialiasDisplay false", "High quality", false);
        jmolCheckbox("color background white",
            "color background black", "White background", false);
        jmolButton("write PNGJ model.png","Save image");
        </script></td>
        <td valign="top"><b>Top 5 enzyme homologs in PDB</b><br/><br/>
        <table border="0" style="font-family:Arial;font-size:13px;">
        <tr><th>Click<br>to view</th><th>Rank</th><th>Cscore<sup>EC</sup></th><th>PDB<br/>Hit</th><th>TM-score</th><th>RMSD<sup>a</sup></th><th>IDEN<sup>a</sup></th><th>Cov.</th><th>EC Number</th><th>Predicted Active Site Residues</th></tr>
        
        <tr><td align="center"><script type="text/javascript">jmolRadio("select all;wireframe off;spacefill off;label off;;set spin x 5;set spin y 5;spin off;","",false);</script></td><td align="left">1</td><td align="center">0.060</td><td><a href="https://www.rcsb.org/pdb/explore/explore.do?structureId=1vcu" target="_blank"><font face="Monospace">1vcuB</font></a></td><td align="center">0.452</td><td align="center">4.30</td><td align="center">0.029</td><td align="right">0.572</td><td align="left"><span title=""><a href="http://enzyme.expasy.org/EC/3.2.1.18" target="_blank">3.2.1.18</a></span>&nbsp;</td><td align="left">NA</td></tr>
        <tr><td align="center"><script type="text/javascript">jmolRadio("select all;wireframe off;spacefill off;label off;;set spin x 5;set spin y 5;spin off;","",false);</script></td><td align="left">2</td><td align="center">0.060</td><td><a href="https://www.rcsb.org/pdb/explore/explore.do?structureId=2aey" target="_blank"><font face="Monospace">2aeyA</font></a></td><td align="center">0.459</td><td align="center">4.07</td><td align="center">0.062</td><td align="right">0.575</td><td align="left"><span title=""><a href="http://enzyme.expasy.org/EC/3.2.1.80" target="_blank">3.2.1.80</a></span>&nbsp;</td><td align="left">NA</td></tr>
        <tr><td align="center"><script type="text/javascript">jmolRadio("select all;wireframe off;spacefill off;label off;select 318;wireframe 70;spacefill 100;color cpk;select 318 and *.ca; label %m%R; color label magenta;;set spin x 5;set spin y 5;spin off;","",false);</script></td><td align="left">3</td><td align="center">0.060</td><td><a href="https://www.rcsb.org/pdb/explore/explore.do?structureId=2ac1" target="_blank"><font face="Monospace">2ac1A</font></a></td><td align="center">0.461</td><td align="center">4.12</td><td align="center">0.083</td><td align="right">0.577</td><td align="left"><span title=""><a href="http://enzyme.expasy.org/EC/3.2.1.26" target="_blank">3.2.1.26</a></span>&nbsp;</td><td align="left">318</td></tr>
        <tr><td align="center"><script type="text/javascript">jmolRadio("select all;wireframe off;spacefill off;label off;;set spin x 5;set spin y 5;spin off;","",false);</script></td><td align="left">4</td><td align="center">0.060</td><td><a href="https://www.rcsb.org/pdb/explore/explore.do?structureId=1st8" target="_blank"><font face="Monospace">1st8A</font></a></td><td align="center">0.459</td><td align="center">4.14</td><td align="center">0.066</td><td align="right">0.577</td><td align="left"><span title=""><a href="http://enzyme.expasy.org/EC/3.2.1.80" target="_blank">3.2.1.80</a></span>&nbsp;</td><td align="left">NA</td></tr>
        <tr><td align="center"><script type="text/javascript">jmolRadio("select all;wireframe off;spacefill off;label off;;set spin x 5;set spin y 5;spin off;","",false);</script></td><td align="left">5</td><td align="center">0.060</td><td><a href="https://www.rcsb.org/pdb/explore/explore.do?structureId=2vsm" target="_blank"><font face="Monospace">2vsmA</font></a></td><td align="center">0.459</td><td align="center">4.74</td><td align="center">0.078</td><td align="right">0.602</td><td align="left"><span title=""><a href="http://enzyme.expasy.org/EC/3.2.1.18" target="_blank">3.2.1.18</a></span>&nbsp;</td><td align="left">NA</td></tr>
        </table><br/>

        <table border="0" cellspacing="2" border="0" style="font-family:Arial;font-size:12px;">
            <tr><td valign="top">&nbsp;</td><td align="justify"><b>Click on the radio buttons to visualize predicted active site residues.</b></td></tr>
            <tr><td valign="top">(a)</td><td align="justify">Cscore<sup>EC</sup> is the confidence score for the Enzyme Commission (EC) number prediction. Cscore<sup>EC</sup> values range in between [0-1]; where a higher score indicates a more reliable EC number prediction.</td></tr>
            <tr><td valign="top">(b)</td><td align="justify">TM-score is a measure of  global structural similarity between query and template protein.</td></tr>
            <tr><td valign="bottom">(c)</td><td align="justify">RMSD<sup>a</sup> is the RMSD between residues that are structurally aligned by TM-align.</td></tr>
            <tr><td valign="bottom">(d)</td><td align="justify">IDEN<sup>a</sup> is the percentage sequence identity in the structurally aligned region.</td></tr>
            <tr><td valign="top">(e)</td><td align="justify">Cov. represents the coverage of global structural  alignment and is equal to the number of structurally aligned residues divided by length of the query protein.</td></tr>
        </table>
    </td></tr>
    </table>
</ul>

<div style="background:#6599FF;width:500px;"><b><font face="Arial" 
color="#FFFFFF" size=4>&nbsp;Predicted Ligand Binding Sites</font></b></div>
<ul>
    <table border="0" style="background:#F2F2F2";align="left">
        <tr><td align ="center"><script type="text/javascript">
        jmolInitialize("/jmol/");
        jmolSetAppletColor("#000000");
        jmolApplet(500, "load complex1.pdb.gz;set frank off;select all;color group;spacefill off;wireframe off;cartoons;select hetero;spacefill 95%;set radius 1.0;color cpk;set ambient 40;select 214;wireframe 70;spacefill 100;color cpk;select 214 and *.ca; label %m%R; color label magenta;select 215;wireframe 70;spacefill 100;color cpk;select 215 and *.ca; label %m%R; color label magenta;select 216;wireframe 70;spacefill 100;color cpk;select 216 and *.ca; label %m%R; color label magenta;select 217;wireframe 70;spacefill 100;color cpk;select 217 and *.ca; label %m%R; color label magenta;select 219;wireframe 70;spacefill 100;color cpk;select 219 and *.ca; label %m%R; color label magenta;select 227;wireframe 70;spacefill 100;color cpk;select 227 and *.ca; label %m%R; color label magenta;;set spin x 10; set spin y 10; spin off;");
        jmolButton("Reset","Reset");
        jmolCheckbox("set spin x 10; set spin y 10; spin on",
            "spin off", "Spin", false);
        jmolCheckbox("set antialiasDisplay true",
            "set antialiasDisplay false", "High quality", false);
        jmolCheckbox("color background white",
            "color background black", "White background", false);
        jmolButton("write PNGJ model.png","Save image");
        </script></td>
        <td valign="top"><b>Template proteins with similar binding site:</b><br/><br/>
        <table border="0" style="font-family:Arial;font-size:13px;">
        <tr><th>Click<br>to view</th><th>Rank</th><th>Cscore<sup>LB</sup></th><th>PDB<br/>Hit</th><th>TM-score</th><th>RMSD<sup>a</sup></th><th>IDEN<sup>a</sup></th><th>Cov.</th><th>BS-score</th><th>Lig. Name</th><th>Download<br/>Complex</th><th align="left">Predicted binding site residues</th></tr>
        
        <tr><td align="center"><script type="text/javascript">jmolRadio("load complex1.pdb.gz;set frank off;select all;color group;spacefill off;wireframe off;cartoons;select hetero;spacefill 95%;set radius 1.0;color cpk;set ambient 40;select 214;wireframe 70;spacefill 100;color cpk;select 214 and *.ca; label %m%R; color label magenta;select 215;wireframe 70;spacefill 100;color cpk;select 215 and *.ca; label %m%R; color label magenta;select 216;wireframe 70;spacefill 100;color cpk;select 216 and *.ca; label %m%R; color label magenta;select 217;wireframe 70;spacefill 100;color cpk;select 217 and *.ca; label %m%R; color label magenta;select 219;wireframe 70;spacefill 100;color cpk;select 219 and *.ca; label %m%R; color label magenta;select 227;wireframe 70;spacefill 100;color cpk;select 227 and *.ca; label %m%R; color label magenta;;set spin x 10;set spin y 10;spin off;","",false);</script></td><td align="left">1</td><td align="center">0.01</td><td><a href="https://www.rcsb.org/pdb/explore/explore.do?structureId=1v0f" target="_blank"><font face="Monospace">1v0fE</font></a></td><td align="center">0.467</td><td align="center">4.55</td><td align="center">0.068</td><td align="right">0.600</td><td align="left">0.68</td><td align="left"><font face="Monospace"><a href="http://zhanglab.ccmb.med.umich.edu/BioLiP/sym.cgi?code=SIA" target="_blank">SIA</a></font></td><td><a href="complex1.pdb.gz" download>complex1.pdb.gz</a></td><td align="left">214,215,216,217,219,227</td></tr>
        <tr><td align="center"><script type="text/javascript">jmolRadio("load complex2.pdb.gz;set frank off;select all;color group;spacefill off;wireframe off;cartoons;select hetero;spacefill 95%;set radius 1.0;color cpk;set ambient 40;select 162;wireframe 70;spacefill 100;color cpk;select 162 and *.ca; label %m%R; color label magenta;select 164;wireframe 70;spacefill 100;color cpk;select 164 and *.ca; label %m%R; color label magenta;select 216;wireframe 70;spacefill 100;color cpk;select 216 and *.ca; label %m%R; color label magenta;select 217;wireframe 70;spacefill 100;color cpk;select 217 and *.ca; label %m%R; color label magenta;;set spin x 10;set spin y 10;spin off;","",false);</script></td><td align="left">2</td><td align="center">0.01</td><td><a href="https://www.rcsb.org/pdb/explore/explore.do?structureId=2ht5" target="_blank"><font face="Monospace">2ht5A</font></a></td><td align="center">0.483</td><td align="center">4.43</td><td align="center">0.070</td><td align="right">0.629</td><td align="left">0.44</td><td align="left"><font face="Monospace"><a href="http://zhanglab.ccmb.med.umich.edu/BioLiP/sym.cgi?code=CA" target="_blank">CA</a></font></td><td><a href="complex2.pdb.gz" download>complex2.pdb.gz</a></td><td align="left">162,164,216,217</td></tr>
        <tr><td align="center"><script type="text/javascript">jmolRadio("load complex3.pdb.gz;set frank off;select all;color group;spacefill off;wireframe off;cartoons;select hetero;spacefill 95%;set radius 1.0;color cpk;set ambient 40;select 221;wireframe 70;spacefill 100;color cpk;select 221 and *.ca; label %m%R; color label magenta;select 222;wireframe 70;spacefill 100;color cpk;select 222 and *.ca; label %m%R; color label magenta;select 223;wireframe 70;spacefill 100;color cpk;select 223 and *.ca; label %m%R; color label magenta;select 225;wireframe 70;spacefill 100;color cpk;select 225 and *.ca; label %m%R; color label magenta;;set spin x 10;set spin y 10;spin off;","",false);</script></td><td align="left">3</td><td align="center">0.01</td><td><a href="https://www.rcsb.org/pdb/explore/explore.do?structureId=1v0f" target="_blank"><font face="Monospace">1v0fD</font></a></td><td align="center">0.467</td><td align="center">4.56</td><td align="center">0.074</td><td align="right">0.600</td><td align="left">0.51</td><td align="left"><font face="Monospace"><a href="http://zhanglab.ccmb.med.umich.edu/BioLiP/sym.cgi?code=SIA" target="_blank">SIA</a></font></td><td><a href="complex3.pdb.gz" download>complex3.pdb.gz</a></td><td align="left">221,222,223,225</td></tr>
        <tr><td align="center"><script type="text/javascript">jmolRadio("load complex4.pdb.gz;set frank off;select all;color group;spacefill off;wireframe off;cartoons;select hetero;spacefill 95%;set radius 1.0;color cpk;set ambient 40;select 162;wireframe 70;spacefill 100;color cpk;select 162 and *.ca; label %m%R; color label magenta;select 182;wireframe 70;spacefill 100;color cpk;select 182 and *.ca; label %m%R; color label magenta;select 216;wireframe 70;spacefill 100;color cpk;select 216 and *.ca; label %m%R; color label magenta;select 270;wireframe 70;spacefill 100;color cpk;select 270 and *.ca; label %m%R; color label magenta;select 271;wireframe 70;spacefill 100;color cpk;select 271 and *.ca; label %m%R; color label magenta;;set spin x 10;set spin y 10;spin off;","",false);</script></td><td align="left">4</td><td align="center">0.01</td><td><a href="https://www.rcsb.org/pdb/explore/explore.do?structureId=3pij" target="_blank"><font face="Monospace">3pijB</font></a></td><td align="center">0.471</td><td align="center">4.06</td><td align="center">0.054</td><td align="right">0.585</td><td align="left">0.48</td><td align="left"><font face="Monospace"><a href="http://zhanglab.ccmb.med.umich.edu/BioLiP/sym.cgi?code=FRU" target="_blank">FRU</a></font></td><td><a href="complex4.pdb.gz" download>complex4.pdb.gz</a></td><td align="left">162,182,216,270,271</td></tr>
        <tr><td align="center"><script type="text/javascript">jmolRadio("load complex5.pdb.gz;set frank off;select all;color group;spacefill off;wireframe off;cartoons;select hetero;spacefill 95%;set radius 1.0;color cpk;set ambient 40;select 219;wireframe 70;spacefill 100;color cpk;select 219 and *.ca; label %m%R; color label magenta;select 278;wireframe 70;spacefill 100;color cpk;select 278 and *.ca; label %m%R; color label magenta;select 279;wireframe 70;spacefill 100;color cpk;select 279 and *.ca; label %m%R; color label magenta;;set spin x 10;set spin y 10;spin off;","",false);</script></td><td align="left">5</td><td align="center">0.01</td><td><a href="https://www.rcsb.org/pdb/explore/explore.do?structureId=2vwd" target="_blank"><font face="Monospace">2vwdA</font></a></td><td align="center">0.455</td><td align="center">4.71</td><td align="center">0.078</td><td align="right">0.600</td><td align="left">0.45</td><td align="left"><font face="Monospace"><a href="http://zhanglab.ccmb.med.umich.edu/BioLiP/sym.cgi?code=GBL" target="_blank">GBL</a></font></td><td><a href="complex5.pdb.gz" download>complex5.pdb.gz</a></td><td align="left">219,278,279</td></tr>
        </table><br/>

        <table border="0" cellspacing="2" style="font-family:Arial;font-size:12px;">
            <tr><td valign="top">&nbsp;</td><td align="justify"><b> Click on the radio buttons to visualize predicted binding site and residues.</b></td></tr>
            <tr><td valign="top">(a)</td><td align="justify">Cscore<sup>LB</sup> is the confidence score of predicted binding site. Cscore<sup>LB</sup> values range in between [0-1]; where a higher score indicates a more reliable ligand-binding site prediction. </td></tr>
            <tr><td valign="top">(b)</td><td align="justify">BS-score is a measure of local similarity (sequence & structure) between template binding site and predicted binding site in the query structure. Based on large scale benchmarking analysis, we have observed that a BS-score >1 reflects a significant local match between the predicted and template binding site.
            <tr><td valign="top">(c)</td><td align="justify">TM-score is a measure of  global structural similarity between query and template protein.</td></tr>
            <tr><td valign="top">(d)</td><td align="justify">RMSD<sup>a</sup> the RMSD between residues that are structurally aligned by TM-align.</td></tr>
            <tr><td valign="top">(e)</td><td align="justify">IDEN<sup>a</sup> is the percentage sequence identity in the structurally aligned region.</td></tr>
            <tr><td valign="top">(f)</td><td align="justify">Cov. represents the coverage of global structural alignment and is equal to the number of structurally aligned residues divided by length of the query protein.</td></tr>
        </table>
    </td></tr>
    </table>

</ul>


</font>
<h4 align="center">[Click <a href="result.zip" download>result.zip</a>
    to download all results on this page]</h4>
<hr><br/>
<div style="position:relative;left:5px;">
    <table style="font-family:Arial;font-size:14px;">
    <tr><td colspan=2>References:</td><tr>
    <tr><td valign="top">1.</td><td>Chengxin Zhang, S M Mortuza, Baoji He, Yanting Wang, and Yang Zhang. "Template-based and free modeling of I-TASSER and QUARK pipelines using predicted contact maps in CASP12." Proteins: Structure, Function, and Bioinformatics (2017). doi: 10.1002/prot.25414</td></tr>
    </table>
</div><br>
</body></html>
